<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2016).():. doi:10.1105/Tpc.15.01057; PMID:27169463</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The Effects of High Steady State Auxin Levels on Root Cell Elongation in Brachypodium.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>David Pacheco-Villalobos, Sara M D&#195;&#173;az-Moreno, Alja van der Schuren, Takayuki Tamaki, Yeon Hee Kang, Bojan Gujas, Ondrej Novak, Nina Jaspert, Zhenni Li, Sebastian Wolf, Claudia Oecking, Karin Ljung, Vincent Bulone, Christian S Hardtke</b>
      </span>
      <br/>
      <span>Affiliation: University of Lausanne CITY: Lausanne POSTAL_CODE: CH-1015 Switzerland [CH] christian.hardtke@unil.ch.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The long-standing Acid Growth Theory of Plant Cell Elongation posits that auxin promotes cell elongation by stimulating cell wall acidification and thus expansin action. To date, the paucity of pertinent genetic materials has precluded thorough analysis of the importance of this concept in roots. The recent isolation of mutants of the model grass species Brachypodium distachyon with dramatically enhanced root cell elongation due to increased cellular auxin levels has allowed us to address this question. We found that the primary transcriptomic effect associated with elevated steady state auxin concentration in elongating root cells is up-regulation of cell wall remodeling factors, notably expansins, while plant hormone signaling pathways maintain remarkable homeostasis. These changes are specifically accompanied by reduced cell wall arabinogalactan complexity but not by increased proton excretion. On the contrary, we observed a tendency for decreased rather than increased proton extrusion from root elongation zones with higher cellular auxin levels. Moreover, similar to Brachypodium distachyon, root cell elongation is, in general, robustly buffered against external pH fluctuation in Arabidopsis thaliana. However, forced acidification through artificial proton pump activation inhibits root cell elongation. Thus, the interplay between auxin, proton pump activation and expansin action may be more flexible in roots than in shoots.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/27169463">http://www.ncbi.nlm.nih.gov/pubmed/27169463</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2016).():. doi:10.1105/Tpc.16.01057; PMID:27152020</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The Effects of High Steady State Auxin Levels on Root Cell Elongation in Brachypodium.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>David Pacheco-Villalobos, Sara M D&#195;&#173;az-Moreno, Alja van der Schuren, Takayuki Tamaki, Yeon Hee Kang, Bojan Gujas, Ondrej Novak, Nina Jaspert, Zhenni Li, Sebastian Wolf, Claudia Oecking, Karin Ljung, Vincent Bulone, Christian S Hardtke</b>
      </span>
      <br/>
      <span>Affiliation: University of Lausanne CITY: Lausanne POSTAL_CODE: CH-1015 Switzerland [CH] christian.hardtke@unil.ch.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The long-standing Acid Growth Theory of Plant Cell Elongation posits that auxin promotes cell elongation by stimulating cell wall acidification and thus expansin action. To date, the paucity of pertinent genetic materials has precluded thorough analysis of the importance of this concept in roots. The recent isolation of mutants of the model grass species Brachypodium distachyon with dramatically enhanced root cell elongation due to increased cellular auxin levels has allowed us to address this question. We found that the primary transcriptomic effect associated with elevated steady state auxin concentration in elongating root cells is up-regulation of cell wall remodeling factors, notably expansins, while plant hormone signaling pathways maintain remarkable homeostasis. These changes are specifically accompanied by reduced cell wall arabinogalactan complexity but not by increased proton excretion. On the contrary, we observed a tendency for decreased rather than increased proton extrusion from root elongation zones with higher cellular auxin levels. Moreover, similar to Brachypodium distachyon, root cell elongation is, in general, robustly buffered against external pH fluctuation in Arabidopsis thaliana. However, forced acidification through artificial proton pump activation inhibits root cell elongation. Thus, the interplay between auxin, proton pump activation and expansin action may be more flexible in roots than in shoots.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/27152020">http://www.ncbi.nlm.nih.gov/pubmed/27152020</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2015).27(1):71-85. doi:10.1105/Tpc.114.133991; PMID:25634987</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genome-wide analysis of alternative splicing landscapes modulated during plant-virus interactions in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kranthi K Mandadi, Karen-Beth G Scholthof</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Pathology and Microbiology; Texas A&amp;M University; College Station; Texas 77843 kbgs@tamu.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>In eukaryotes, alternative splicing (AS) promotes transcriptome and proteome diversity. The extent of genome-wide AS changes occurring during a plant-microbe interaction is largely unknown. Here, using high-throughput, paired-end RNA sequencing, we generated an isoform-level spliceome map of Brachypodium distachyon infected with Panicum mosaic virus and its satellite virus. Overall, we detected &#226;&#136;&#188;44,443 transcripts in B. distachyon, &#226;&#136;&#188;30% more than those annotated in the reference genome. Expression of &#226;&#136;&#188;28,900 transcripts was &#226;&#137;&#165;2 fragments per kilobase of transcript per million mapped fragments, and &#226;&#136;&#188;42% of multi-exonic genes were alternatively spliced. Comparative analysis of AS patterns in B. distachyon, rice (Oryza sativa), maize (Zea mays), sorghum (Sorghum bicolor), Arabidopsis thaliana, potato (Solanum tuberosum), Medicago truncatula, and poplar (Populus trichocarpa) revealed conserved ratios of the AS types between monocots and dicots. Virus infection quantitatively altered AS events in Brachypodium with little effect on the AS ratios. We discovered AS events for &gt;100 immune-related genes encoding receptor-like kinases, NB-LRR resistance proteins, transcription factors, RNA silencing, and splicing-associated proteins. Cloning and molecular characterization of SCL33, a serine/arginine-rich splicing factor, identified multiple novel intron-retaining splice variants that are developmentally regulated and modulated during virus infection. B. distachyon SCL33 splicing patterns are also strikingly conserved compared with a distant Arabidopsis SCL33 ortholog. This analysis provides new insights into AS landscapes conserved among monocots and dicots and uncovered AS events in plant defense-related genes.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25634987">http://www.ncbi.nlm.nih.gov/pubmed/25634987</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2015).27(1):7. doi:10.1105/Tpc.115.00039; PMID:25634986</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Sick as a&#226;&#128;&#166; grass? Viral infection causes massive changes in alternative splicing in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jennifer Mach</b>
      </span>
      <br/>
      <span>Affiliation: Science Editor jmach@aspb.org.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/></p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25634986">http://www.ncbi.nlm.nih.gov/pubmed/25634986</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2014).26(3):1094-104. doi:10.1105/Tpc.113.121830; PMID:24642944</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A role for barley CRYPTOCHROME1 in light regulation of grain dormancy and germination.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jose M Barrero, A Bruce Downie, Qian Xu, Frank Gubler</b>
      </span>
      <br/>
      <span>Affiliation: CSIRO Plant Industry; Canberra ACT 2601; Australia.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>It is well known that abscisic acid (ABA) plays a central role in the regulation of seed dormancy and that transcriptional regulation of genes encoding ABA biosynthetic and degradation enzymes is responsible for determining ABA content. However, little is known about the upstream signaling pathways impinging on transcription to ultimately regulate ABA content or how environmental signals (e.g., light and cold) might direct such expression in grains. Our previous studies indicated that light is a key environmental signal inhibiting germination in dormant grains of barley (Hordeum vulgare), wheat (Triticum aestivum), and Brachypodium distachyon and that this effect attenuates as after-ripening progresses further. We found that the blue component of the light spectrum inhibits completion of germination in barley by inducing the expression of the ABA biosynthetic gene 9-cis-epoxycarotenoid dioxygenase and dampening expression of ABA 8'-hydroxylase, thus increasing ABA content in the grain. We have now created barley transgenic lines downregulating the genes encoding the blue light receptors CRYTOCHROME (CRY1) and CRY2. Our results demonstrate that CRY1 is the key receptor perceiving and transducing the blue light signal in dormant grains.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24642944">http://www.ncbi.nlm.nih.gov/pubmed/24642944</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2013).25(11):4283. doi:10.1105/Tpc.113.251112; PMID:24285793</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Small RNAs and the big decisions: MicroRNA regulation of photoperiodic flowering in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jennifer Mach</b>
      </span>
      <br/>
      <span>Affiliation: Science Editor.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/></p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24285793">http://www.ncbi.nlm.nih.gov/pubmed/24285793</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2013).25(11):4363-77. doi:10.1105/Tpc.113.118620; PMID:24285787</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Regulation of FLOWERING LOCUS T by a microRNA in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Liang Wu, Dongfeng Liu, Jiajie Wu, Rongzhi Zhang, Zhengrui Qin, Danmei Liu, Aili Li, Daolin Fu, Wenxue Zhai, Long Mao</b>
      </span>
      <br/>
      <span>Affiliation: National Key Facility for Crop Gene Resources and Genetic Improvement; Institute of Crop Science; Chinese Academy of Agricultural Sciences; Beijing 100081; China.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The highly conserved florigen gene FLOWERING LOCUS T (FT) functions at the core of the flowering pathways. Extensive studies have examined the transcriptional regulation of FT; however, other layers of FT regulation remain unclear. Here, we identified miR5200 a Pooideae-specific microRNA that is expressed in leaves and targets Brachypodium distachyon FT orthologs for mRNA cleavage. miR5200 was abundantly expressed in plants grown under short-day (SD) conditions but was dramatically repressed in plants transferred to long-day (LD) conditions. We also found that the epigenetic chromatin status, specifically the levels of histone methylation marks, at miR5200 precursor loci changed in response to daylength. Moreover, artificial interruption of miR5200 activity by target mimicry in B. distachyon altered flowering time in SD but not in LD conditions, suggesting that miR5200 functions in photoperiod-mediated flowering time regulation. Together, these findings illustrate a posttranscriptional regulation mechanism of FT and provide insights into understanding of the multiple concerted pathways for flowering time control in plants.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24285787">http://www.ncbi.nlm.nih.gov/pubmed/24285787</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2013).25(10):3685-98. doi:10.1105/Tpc.113.114553; PMID:24104565</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Reticulate evolution of the rye genome.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Mihaela M Martis, Ruonan Zhou, Grit Haseneyer, Thomas Schmutzer, Jan Vr&#195;&#161;na, Marie Kubal&#195;&#161;kov&#195;&#161;, Susanne K&#195;&#182;nig, Karl G Kugler, Uwe Scholz, Bernd Hackauf, Viktor Korzun, Chris-Carolin Sch&#195;&#182;n, Jaroslav Dolezel, Eva Bauer, Klaus F X Mayer, Nils Stein</b>
      </span>
      <br/>
      <span>Affiliation: Helmholtz Center Munich; German Research Centre for Environmental Health; Munich Information Center for Protein Sequences/IBIS; Institute of Bioinformatics and Systems Biology; 85764 Neuherberg; Germany.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Rye (Secale cereale) is closely related to wheat (Triticum aestivum) and barley (Hordeum vulgare). Due to its large genome (~8 Gb) and its regional importance, genome analysis of rye has lagged behind other cereals. Here, we established a virtual linear gene order model (genome zipper) comprising 22,426 or 72% of the detected set of 31,008 rye genes. This was achieved by high-throughput transcript mapping, chromosome survey sequencing, and integration of conserved synteny information of three sequenced model grass genomes (Brachypodium distachyon, rice [Oryza sativa], and sorghum [Sorghum bicolor]). This enabled a genome-wide high-density comparative analysis of rye/barley/model grass genome synteny. Seventeen conserved syntenic linkage blocks making up the rye and barley genomes were defined in comparison to model grass genomes. Six major translocations shaped the modern rye genome in comparison to a putative Triticeae ancestral genome. Strikingly dissimilar conserved syntenic gene content, gene sequence diversity signatures, and phylogenetic networks were found for individual rye syntenic blocks. This indicates that introgressive hybridizations (diploid or polyploidy hybrid speciation) and/or a series of whole-genome or chromosome duplications played a role in rye speciation and genome evolution.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24104565">http://www.ncbi.nlm.nih.gov/pubmed/24104565</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Review</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2013).25(5):1489-505. doi:10.1105/Tpc.113.111658; PMID:23709626</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Plant immune responses against viruses: how does a virus cause disease?</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kranthi K Mandadi, Karen-Beth G Scholthof</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Pathology and Microbiology; Texas A&amp;M University; College Station; Texas 77843; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plants respond to pathogens using elaborate networks of genetic interactions. Recently, significant progress has been made in understanding RNA silencing and how viruses counter this apparently ubiquitous antiviral defense. In addition, plants also induce hypersensitive and systemic acquired resistance responses, which together limit the virus to infected cells and impart resistance to the noninfected tissues. Molecular processes such as the ubiquitin proteasome system and DNA methylation are also critical to antiviral defenses. Here, we provide a summary and update of advances in plant antiviral immune responses, beyond RNA silencing mechanisms-advances that went relatively unnoticed in the realm of RNA silencing and nonviral immune responses. We also document the rise of Brachypodium and Setaria species as model grasses to study antiviral responses in Poaceae, aspects that have been relatively understudied, despite grasses being the primary source of our calories, as well as animal feed, forage, recreation, and biofuel needs in the 21st century. Finally, we outline critical gaps, future prospects, and considerations central to studying plant antiviral immunity. To promote an integrated model of plant immunity, we discuss analogous viral and nonviral immune concepts and propose working definitions of viral effectors, effector-triggered immunity, and viral pathogen-triggered immunity.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23709626">http://www.ncbi.nlm.nih.gov/pubmed/23709626</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2013).496(7443):87-90. doi:10.1038/Nature11997; PMID:23535596</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Draft genome of the wheat A-genome progenitor Triticum urartu.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Hong-Qing Ling, Shancen Zhao, Dongcheng Liu, Junyi Wang, Hua Sun, Chi Zhang, Huajie Fan, Dong Li, Lingli Dong, Yong Tao, Chuan Gao, Huilan Wu, Yiwen Li, Yan Cui, Xiaosen Guo, Shusong Zheng, Biao Wang, Kang Yu, Qinsi Liang, Wenlong Yang, Xueyuan Lou, Jie Chen, Mingji Feng, Jianbo Jian, Xiaofei Zhang, Guangbin Luo, Ying Jiang, Junjie Liu, Zhaobao Wang, Yuhui Sha, Bairu Zhang, Huajun Wu, Dingzhong Tang, Qianhua Shen, Pengya Xue, Shenhao Zou, Xiujie Wang, Xin Liu, Famin Wang, Yanping Yang, Xueli An, Zhenying Dong, Kunpu Zhang, Xiangqi Zhang, Ming-Cheng Luo, Jan Dvorak, Yiping Tong, Jian Wang, Huanming Yang, Zhensheng Li, Daowen Wang, Aimin Zhang, Jun Wang</b>
      </span>
      <br/>
      <span>Affiliation: State Key Laboratory of Plant Cell and Chromosome Engineering; Institute of Genetics and Developmental Biology; Chinese Academy of Sciences; Beijing 100101; China.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Bread wheat (Triticum aestivum, AABBDD) is one of the most widely cultivated and consumed food crops in the world. However, the complex polyploid nature of its genome makes genetic and functional analyses extremely challenging. The A&#226;&#128;&#137;genome, as a basic genome of bread wheat and other polyploid wheats, for example, T. turgidum (AABB), T. timopheevii (AAGG) and T. zhukovskyi (AAGGA(m)A(m)), is central to wheat evolution, domestication and genetic improvement. The progenitor species of the A&#226;&#128;&#137;genome is the diploid wild einkorn wheat T.&#226;&#128;&#137;urartu, which resembles cultivated wheat more extensively than do Aegilops speltoides (the ancestor of the B genome) and Ae. tauschii (the donor of the D genome), especially in the morphology and development of spike and seed. Here we present the generation, assembly and analysis of a whole-genome shotgun draft sequence of the T.&#226;&#128;&#137;urartu genome. We identified protein-coding gene models, performed genome structure analyses and assessed its utility for analysing agronomically important genes and for developing molecular markers. Our T.&#226;&#128;&#137;urartu genome assembly provides a diploid reference for analysis of polyploid wheat genomes and is a valuable resource for the genetic improvement of wheat.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v496/n7443/Nature11997.html">http://www.nature.com/journal/v496/n7443/Nature11997.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2012).491(7426):705-10. doi:10.1038/Nature11650; PMID:23192148</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Analysis of the bread wheat genome using whole-genome shotgun sequencing.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Rachel Brenchley, Manuel Spannagl, Matthias Pfeifer, Gary L A Barker, Rosalinda D'Amore, Alexandra M Allen, Neil McKenzie, Melissa Kramer, Arnaud Kerhornou, Dan Bolser, Suzanne Kay, Darren Waite, Martin Trick, Ian Bancroft, Yong Gu, Naxin Huo, Ming-Cheng Luo, Sunish Sehgal, Bikram Gill, Sharyar Kianian, Olin Anderson, Paul Kersey, Jan Dvorak, W Richard McCombie, Anthony Hall, Klaus F X Mayer, Keith J Edwards, Michael W Bevan, Neil Hall</b>
      </span>
      <br/>
      <span>Affiliation: Centre for Genome Research; University of Liverpool; Liverpool L69 7ZB; UK.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Bread wheat (Triticum aestivum) is a globally important crop, accounting for 20 per cent of the calories consumed by humans. Major efforts are underway worldwide to increase wheat production by extending genetic diversity and analysing key traits, and genomic resources can accelerate progress. But so far the very large size and polyploid complexity of the bread wheat genome have been substantial barriers to genome analysis. Here we report the sequencing of its large, 17-gigabase-pair, hexaploid genome using 454&#226;&#128;&#137;pyrosequencing, and comparison of this with the sequences of diploid ancestral and progenitor genomes. We identified between 94,000 and 96,000 genes, and assigned two-thirds to the three component genomes (A, B and D) of hexaploid wheat. High-resolution synteny maps identified many small disruptions to conserved gene order. We show that the hexaploid genome is highly dynamic, with significant loss of gene family members on polyploidization and domestication, and an abundance of gene fragments. Several classes of genes involved in energy harvesting, metabolism and growth are among expanded gene families that could be associated with crop productivity. Our analyses, coupled with the identification of extensive genetic variation, provide a resource for accelerating gene discovery and improving this major crop.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v491/n7426/Nature11650.html">http://www.nature.com/journal/v491/n7426/Nature11650.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2011).23(5):1706-18. doi:10.1105/Tpc.111.086629; PMID:21622801</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Frequent gene movement and pseudogene evolution is common to the large and complex genomes of wheat, barley, and their relatives.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Thomas Wicker, Klaus F X Mayer, Heidrun Gundlach, Mihaela Martis, Burkhard Steuernagel, Uwe Scholz, Hana Simkov&#195;&#161;, Marie Kubal&#195;&#161;kov&#195;&#161;, Fr&#195;&#169;d&#195;&#169;ric Choulet, Stefan Taudien, Matthias Platzer, Catherine Feuillet, Tzion Fahima, Hikmet Budak, Jaroslav Dolezel, Beat Keller, Nils Stein</b>
      </span>
      <br/>
      <span>Affiliation: Institute of Plant Biology; University of Zurich; Zurich; Switzerland.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>All six arms of the group 1 chromosomes of hexaploid wheat (Triticum aestivum) were sequenced with Roche/454 to 1.3- to 2.2-fold coverage and compared with similar data sets from the homoeologous chromosome 1H of barley (Hordeum vulgare). Six to ten thousand gene sequences were sampled per chromosome. These were classified into genes that have their closest homologs in the Triticeae group 1 syntenic region in Brachypodium, rice (Oryza sativa), and/or sorghum (Sorghum bicolor) and genes that have their homologs elsewhere in these model grass genomes. Although the number of syntenic genes was similar between the homologous groups, the amount of nonsyntenic genes was found to be extremely diverse between wheat and barley and even between wheat subgenomes. Besides a small core group of genes that are nonsyntenic in other grasses but conserved among Triticeae, we found thousands of genic sequences that are specific to chromosomes of one single species or subgenome. By examining in detail 50 genes from chromosome 1H for which BAC sequences were available, we found that many represent pseudogenes that resulted from transposable element activity and double-strand break repair. Thus, Triticeae seem to accumulate nonsyntenic genes frequently. Since many of them are likely to be pseudogenes, total gene numbers in Triticeae are prone to pronounced overestimates.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21622801">http://www.ncbi.nlm.nih.gov/pubmed/21622801</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2011).23(5):1756-71. doi:10.1105/Tpc.111.084590; PMID:21540434</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">BARREN STALK FASTIGIATE1 is an AT-hook protein required for the formation of maize ears.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Andrea Gallavotti, Simon Malcomber, Craig Gaines, Sharon Stanfield, Clinton Whipple, Elizabeth Kellogg, Robert J Schmidt</b>
      </span>
      <br/>
      <span>Affiliation: Section of Cell and Developmental Biology; University of California-San Diego; La Jolla; CA 92093-0116; USA. agallavotti@ucsd.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Ears are the seed-bearing inflorescences of maize (Zea mays) plants and represent a crucial component of maize yield. The first step in the formation of ears is the initiation of axillary meristems in the axils of developing leaves. In the classic maize mutant barren stalk fastigiate1 (baf1), first discovered in the 1950s, ears either do not form or, if they do, are partially fused to the main stalk. We positionally cloned Baf1 and found that it encodes a transcriptional regulator containing an AT-hook DNA binding motif. Single coorthologs of Baf1 are found in syntenic regions of brachypodium (Brachypodium distachyon), rice (Oryza sativa), and sorghum (Sorghum bicolor), suggesting that the gene is likely present in all cereal species. Protein-protein interaction assays suggest that BAF1 is capable of forming homodimers and heterodimers with other members of the AT-hook family. Another transcriptional regulator required for ear initiation is the basic helix-loop-helix protein BARREN STALK1 (BA1). Genetic and expression analyses suggest that Baf1 is required to reach a threshold level of Ba1 expression for the initiation of maize ears. We propose that Baf1 functions in the demarcation of a boundary region essential for the specification of a stem cell niche.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21540434">http://www.ncbi.nlm.nih.gov/pubmed/21540434</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2011).23(4):1249-63. doi:10.1105/Tpc.110.082537; PMID:21467582</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Unlocking the barley genome by chromosomal and comparative genomics.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Klaus F X Mayer, Mihaela Martis, Pete E Hedley, Hana Simkov&#195;&#161;, Hui Liu, Jenny A Morris, Burkhard Steuernagel, Stefan Taudien, Stephan Roessner, Heidrun Gundlach, Marie Kubal&#195;&#161;kov&#195;&#161;, Pavla Such&#195;&#161;nkov&#195;&#161;, Florent Murat, Marius Felder, Thomas Nussbaumer, Andreas Graner, Jerome Salse, Takashi Endo, Hiroaki Sakai, Tsuyoshi Tanaka, Takeshi Itoh, Kazuhiro Sato, Matthias Platzer, Takashi Matsumoto, Uwe Scholz, Jaroslav Dolezel, Robbie Waugh, Nils Stein</b>
      </span>
      <br/>
      <span>Affiliation: Munich Information Center for Protein Sequences/Institute of Bioinformatics and Systems Biology; Institute for Bioinformatics and Systems Biology; Helmholtz Center Munich; 85764 Neuherberg; Germany.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>We used a novel approach that incorporated chromosome sorting, next-generation sequencing, array hybridization, and systematic exploitation of conserved synteny with model grasses to assign ~86% of the estimated ~32,000 barley (Hordeum vulgare) genes to individual chromosome arms. Using a series of bioinformatically constructed genome zippers that integrate gene indices of rice (Oryza sativa), sorghum (Sorghum bicolor), and Brachypodium distachyon in a conserved synteny model, we were able to assemble 21,766 barley genes in a putative linear order. We show that the barley (H) genome displays a mosaic of structural similarity to hexaploid bread wheat (Triticum aestivum) A, B, and D subgenomes and that orthologous genes in different grasses exhibit signatures of positive selection in different lineages. We present an ordered, information-rich scaffold of the barley genome that provides a valuable and robust framework for the development of novel strategies in cereal breeding.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21467582">http://www.ncbi.nlm.nih.gov/pubmed/21467582</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2011).23(1):27-37. doi:10.1105/Tpc.110.080622; PMID:21266659</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Seventy million years of concerted evolution of a homoeologous chromosome pair, in parallel, in major Poaceae lineages.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Xiyin Wang, Haibao Tang, Andrew H Paterson</b>
      </span>
      <br/>
      <span>Affiliation: Plant Genome Mapping Laboratory; University of Georgia; Athens; Georgia 30602; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Whole genome duplication ~70 million years ago provided raw material for Poaceae (grass) diversification. Comparison of rice (Oryza sativa), sorghum (Sorghum bicolor), maize (Zea mays), and Brachypodium distachyon genomes revealed that one paleo-duplicated chromosome pair has experienced very different evolution than all the others. For tens of millions of years, the two chromosomes have experienced illegitimate recombination that has been temporally restricted in a stepwise manner, producing structural stratification in the chromosomes. These strata formed independently in different grass lineages, with their similarities (low sequence divergence between paleo-duplicated genes) preserved in parallel for millions of years since the divergence of these lineages. The pericentromeric region of this homeologous chromosome pair accounts for two-thirds of the gene content differences between the modern chromosomes. Both intriguing and perplexing is a distal chromosomal region with the greatest DNA similarity between surviving duplicated genes but also with the highest concentration of lineage-specific gene pairs found anywhere in these genomes and with a significantly elevated gene evolutionary rate. Intragenomic similarity near this chromosomal terminus may be important in hom(e)ologous chromosome pairing. Chromosome structural stratification, together with enrichment of autoimmune response-related (nucleotide binding site-leucine-rich repeat) genes and accelerated DNA rearrangement and gene loss, confer a striking resemblance of this grass chromosome pair to the sex chromosomes of other taxa.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21266659">http://www.ncbi.nlm.nih.gov/pubmed/21266659</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2010).22(6):1686-701. doi:10.1105/Tpc.110.074187; PMID:20581307</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Megabase level sequencing reveals contrasted organization and evolution patterns of the wheat gene and transposable element spaces.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Fr&#195;&#169;d&#195;&#169;ric Choulet, Thomas Wicker, Camille Rustenholz, Etienne Paux, J&#195;&#169;rome Salse, Philippe Leroy, St&#195;&#169;phane Schlub, Marie-Christine Le Paslier, Ghislaine Magdelenat, Catherine Gonthier, Arnaud Couloux, Hikmet Budak, James Breen, Michael Pumphrey, Sixin Liu, Xiuying Kong, Jizeng Jia, Marta Gut, Dominique Brunel, James A Anderson, Bikram S Gill, Rudi Appels, Beat Keller, Catherine Feuillet</b>
      </span>
      <br/>
      <span>Affiliation: Institut National de la Recherche Agronomique; Universit&#195;&#169; Blaise Pascal; Unit&#195;&#169; Mixte de Recherche 1095 Genetics Diversity and Ecophysiology of Cereals; F-63100 Clermont-Ferrand; France. frederic.choulet@clermont.inra.fr</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>To improve our understanding of the organization and evolution of the wheat (Triticum aestivum) genome, we sequenced and annotated 13-Mb contigs (18.2 Mb) originating from different regions of its largest chromosome, 3B (1 Gb), and produced a 2x chromosome survey by shotgun Illumina/Solexa sequencing. All regions carried genes irrespective of their chromosomal location. However, gene distribution was not random, with 75% of them clustered into small islands containing three genes on average. A twofold increase of gene density was observed toward the telomeres likely due to high tandem and interchromosomal duplication events. A total of 3222 transposable elements were identified, including 800 new families. Most of them are complete but showed a highly nested structure spread over distances as large as 200 kb. A succession of amplification waves involving different transposable element families led to contrasted sequence compositions between the proximal and distal regions. Finally, with an estimate of 50,000 genes per diploid genome, our data suggest that wheat may have a higher gene number than other cereals. Indeed, comparisons with rice (Oryza sativa) and Brachypodium revealed that a high number of additional noncollinear genes are interspersed within a highly conserved ancestral grass gene backbone, supporting the idea of an accelerated evolution in the Triticeae lineages.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20581307">http://www.ncbi.nlm.nih.gov/pubmed/20581307</a>
    </p>
  </body>
  <hr/>
</html>
