<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2015).528(7581):218-24. doi:10.1038/Nature15749; PMID:26659182</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The histone chaperone CAF-1 safeguards somatic cell identity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Sihem Cheloufi, Ulrich Elling, Barbara Hopfgartner, Youngsook L Jung, Jernej Murn, Maria Ninova, Maria Hubmann, Aimee I Badeaux, Cheen Euong Ang, Danielle Tenen, Daniel J Wesche, Nadezhda Abazova, Max Hogue, Nilgun Tasdemir, Justin Brumbaugh, Philipp Rathert, Julian Jude, Francesco Ferrari, Andres Blanco, Michaela Fellner, Daniel Wenzel, Marietta Zinner, Simon E Vidal, Oliver Bell, Matthias Stadtfeld, Howard Y Chang, Genevieve Almouzni, Scott W Lowe, John Rinn, Marius Wernig, Alexei Aravin, Yang Shi, Peter J Park, Josef M Penninger, Johannes Zuber, Konrad Hochedlinger</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Biology; Cancer Center and Center for Regenerative Medicine; Massachusetts General Hospital; Boston; Massachusetts 02114; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Cellular differentiation involves profound remodelling of chromatic landscapes, yet the mechanisms by which somatic cell identity is subsequently maintained remain incompletely understood. To further elucidate regulatory pathways that safeguard the somatic state, we performed two comprehensive RNA interference (RNAi) screens targeting chromatin factors during transcription-factor-mediated reprogramming of mouse fibroblasts to induced pluripotent stem cells (iPS cells). Subunits of the chromatin assembly factor-1 (CAF-1) complex, including Chaf1a and Chaf1b, emerged as the most prominent hits from both screens, followed by modulators of lysine sumoylation and heterochromatin maintenance. Optimal modulation of both CAF-1 and transcription factor levels increased reprogramming efficiency by several orders of magnitude and facilitated iPS cell formation in as little as 4 days. Mechanistically, CAF-1 suppression led to a more accessible chromatin structure at enhancer elements early during reprogramming. These changes were accompanied by a decrease in somatic heterochromatin domains, increased binding of Sox2 to pluripotency-specific targets and activation of associated genes. Notably, suppression of CAF-1 also enhanced the direct conversion of B cells into macrophages and fibroblasts into neurons. Together, our findings reveal the histone chaperone CAF-1 to be a novel regulator of somatic cell identity during transcription-factor-induced cell-fate transitions and provide a potential strategy to modulate cellular plasticity in a regenerative setting.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v528/n7581/Nature15749.html">http://www.nature.com/journal/v528/n7581/Nature15749.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2015).163(1):230-45. doi:10.1016/J.Cell.2015.08.037; PMID:26365490</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Systematic identification of factors for provirus silencing in embryonic stem cells.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Bin Xia Yang, Chadi A El Farran, Hong Chao Guo, Tao Yu, Hai Tong Fang, Hao Fei Wang, Sharon Schlesinger, Yu Fen Samantha Seah, Germaine Yen Lin Goh, Suat Peng Neo, Yinghui Li, Matthew C Lorincz, Vinay Tergaonkar, Tit-Meng Lim, Lingyi Chen, Jayantha Gunaratne, James J Collins, Stephen P Goff, George Q Daley, Hu Li, Frederic A Bard, Yuin-Han Loh</b>
      </span>
      <br/>
      <span>Affiliation: Epigenetics and Cell Fates Laboratory; A(&#226;&#136;&#151;)STAR Institute of Molecular and Cell Biology; 61 Biopolis Drive Proteos; Singapore 138673; Singapore; Department of Biological Sciences; National University of Singapore; Singapore 117543; Singapore. Electronic address: yhloh@imcb.a-star.edu.sg.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Embryonic stem cells (ESCs) repress the expression of exogenous proviruses and endogenous retroviruses (ERVs). Here, we systematically dissected the cellular factors involved in provirus repression in embryonic carcinomas (ECs) and ESCs by a genome-wide siRNA screen. Histone chaperones (Chaf1a/b), sumoylation factors (Sumo2/Ube2i/Sae1/Uba2/Senp6), and chromatin modifiers (Trim28/Eset/Atf7ip) are key determinants that establish provirus silencing. RNA-seq analysis uncovered the roles of&#194;&#160;Chaf1a/b and sumoylation modifiers in the repression of ERVs. ChIP-seq analysis demonstrates direct&#194;&#160;recruitment of Chaf1a and Sumo2 to ERVs. Chaf1a reinforces transcriptional repression via its interaction with members of the NuRD complex (Kdm1a, Hdac1/2) and Eset, while Sumo2 orchestrates the provirus repressive function of the canonical Zfp809/Trim28/Eset machinery by sumoylation of Trim28. Our study reports a genome-wide atlas of functional nodes that mediate proviral silencing in ESCs and illuminates the comprehensive, interconnected, and multi-layered genetic and epigenetic mechanisms by which ESCs repress retroviruses within the genome.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(15)01089-2.html">http://www.cell.com/cell/full/S0092-8674(15)01089-2.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2014).511(7511):551-6. doi:10.1038/Nature13442; PMID:25043020</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Topoisomerase II mediates meiotic crossover interference.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Liangran Zhang, Shunxin Wang, Shen Yin, Soogil Hong, Keun P Kim, Nancy Kleckner</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular and Cellular Biology; Harvard University; Cambridge; Massachusetts 02138; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Spatial patterning is a ubiquitous feature of biological systems. Meiotic crossovers provide an interesting example, defined by the classic phenomenon of crossover interference. Here we identify a molecular pathway for interference by analysing crossover patterns in budding yeast. Topoisomerase II plays a central role, thus identifying a new function for this critical molecule. SUMOylation (of topoisomerase II and axis component Red1) and ubiquitin-mediated removal of SUMOylated proteins are also required. The findings support the hypothesis that crossover interference involves accumulation, relief and redistribution of mechanical stress along the protein/DNA meshwork of meiotic chromosome axes, with topoisomerase II required to adjust spatial relationships among DNA segments.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v511/n7511/Nature13442.html">http://www.nature.com/journal/v511/n7511/Nature13442.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2013).341(6144):395-9. doi:10.1126/Science.1236188; PMID:23888040</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Nuclear PTEN controls DNA repair and sensitivity to genotoxic stress.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>C Bassi, J Ho, T Srikumar, R J O Dowling, C Gorrini, S J Miller, T W Mak, B G Neel, B Raught, V Stambolic</b>
      </span>
      <br/>
      <span>Affiliation: Department of Medical Biophysics; University of Toronto; Toronto; Ontario M5G 2M9; Canada.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Loss of function of the phosphatase and tensin homolog deleted on chromosome 10 (PTEN) tumor suppressor gene is associated with many human cancers. In the cytoplasm, PTEN antagonizes the phosphatidylinositol 3-kinase (PI3K) signaling pathway. PTEN also accumulates in the nucleus, where its function remains poorly understood. We demonstrate that SUMOylation (SUMO, small ubiquitin-like modifier) of PTEN controls its nuclear localization. In cells exposed to genotoxic stress, SUMO-PTEN was rapidly excluded from the nucleus dependent on the protein kinase ataxia telangiectasia mutated (ATM). Cells lacking nuclear PTEN were hypersensitive to DNA damage, whereas PTEN-deficient cells were susceptible to killing by a combination of genotoxic stress and a small-molecule PI3K inhibitor both in vitro and in vivo. Our findings may have implications for individualized therapy for patients with PTEN-deficient tumors.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/341/6144/395.full">http://science.sciencemag.org/content/341/6144/395.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2012).151(4):807-20. doi:10.1016/J.Cell.2012.10.021; PMID:23122649</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Protein group modification and synergy in the SUMO pathway as exemplified in DNA repair.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Ivan Psakhye, Stefan Jentsch</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Cell Biology; Max Planck Institute of Biochemistry; Am Klopferspitz 18; 82152 Martinsried; Germany.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Protein modification by SUMO affects a wide range of protein substrates. Surprisingly, although SUMO pathway mutants display strong phenotypes, the function of individual SUMO modifications is often enigmatic, and SUMOylation-defective mutants commonly lack notable phenotypes. Here, we use DNA double-strand break repair as an example and show that DNA damage triggers a SUMOylation wave, leading to simultaneous multisite modifications of several repair proteins of the same pathway. Catalyzed by a DNA-bound SUMO ligase and triggered by single-stranded DNA, SUMOylation stabilizes physical interactions between the proteins. Notably, only wholesale elimination of SUMOylation of several repair proteins significantly affects the homologous recombination pathway by considerably slowing down DNA repair. Thus, SUMO acts synergistically on several proteins, and individual modifications only add up to efficient repair. We propose that SUMOylation may thus often target a protein group rather than individual proteins, whereas localized modification enzymes and highly specific triggers ensure specificity.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(12)01241-X.html">http://www.cell.com/cell/full/S0092-8674(12)01241-X.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2012).148(3):556-67. doi:10.1016/J.Cell.2011.11.062; PMID:22304921</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Fibroblast growth factor-21 regulates PPAR&#206;&#179; activity and the antidiabetic actions of thiazolidinediones.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Paul A Dutchak, Takeshi Katafuchi, Angie L Bookout, Jang Hyun Choi, Ruth T Yu, David J Mangelsdorf, Steven A Kliewer</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Biology; University of Texas Southwestern Medical Center; Dallas; TX 75390-9041; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Fibroblast growth factor-21 (FGF21) is a circulating hepatokine that beneficially affects carbohydrate and lipid metabolism. Here, we report that FGF21 is&#194;&#160;also an inducible, fed-state autocrine factor in adipose tissue that functions in a feed-forward loop to regulate the activity of peroxisome proliferator-activated receptor &#206;&#179; (PPAR&#206;&#179;), a master transcriptional regulator of adipogenesis. FGF21 knockout (KO) mice display defects in PPAR&#206;&#179; signaling including decreased body fat and attenuation of PPAR&#206;&#179;-dependent gene expression. Moreover, FGF21-KO mice are refractory to both the beneficial insulin-sensitizing effects and the detrimental weight gain and edema side effects of the PPAR&#206;&#179; agonist rosiglitazone. This loss of function in FGF21-KO mice is coincident with a marked increase in the sumoylation of PPAR&#206;&#179;, which reduces its transcriptional activity. Adding back FGF21 prevents sumoylation and restores PPAR&#206;&#179; activity. Collectively, these results reveal FGF21 as a key mediator of the physiologic and pharmacologic actions of PPAR&#206;&#179;.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(12)00008-6.html">http://www.cell.com/cell/full/S0092-8674(12)00008-6.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2012).335(6066):348-53. doi:10.1126/Science.1212728; PMID:22157079</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A SUMOylation-dependent transcriptional subprogram is required for Myc-driven tumorigenesis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jessica D Kessler, Kristopher T Kahle, Tingting Sun, Kristen L Meerbrey, Michael R Schlabach, Earlene M Schmitt, Samuel O Skinner, Qikai Xu, Mamie Z Li, Zachary C Hartman, Mitchell Rao, Peng Yu, Rocio Dominguez-Vidana, Anthony C Liang, Nicole L Solimini, Ronald J Bernardi, Bing Yu, Tiffany Hsu, Ido Golding, Ji Luo, C Kent Osborne, Chad J Creighton, Susan G Hilsenbeck, Rachel Schiff, Chad A Shaw, Stephen J Elledge, Thomas F Westbrook</b>
      </span>
      <br/>
      <span>Affiliation: Verna and Marrs McLean Department of Biochemistry and Molecular Biology; Baylor College of Medicine; Houston; TX 77030; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Myc is an oncogenic transcription factor frequently dysregulated in human cancer. To identify pathways supporting the Myc oncogenic program, we used a genome-wide RNA interference screen to search for Myc-synthetic lethal genes and uncovered a role for the SUMO-activating enzyme (SAE1/2). Loss of SAE1/2 enzymatic activity drives synthetic lethality with Myc. Inactivation of SAE2 leads to mitotic catastrophe and cell death upon Myc hyperactivation. Mechanistically, SAE2 inhibition switches a transcriptional subprogram of Myc from activated to repressed. A subset of these SUMOylation-dependent Myc switchers (SMS genes) is required for mitotic spindle function and to support the Myc oncogenic program. SAE2 is required for growth of Myc-dependent tumors in mice, and gene expression analyses of Myc-high human breast cancers suggest that low SAE1 and SAE2 abundance in the tumors correlates with longer metastasis-free survival of the patients. Thus, inhibition of SUMOylation may merit investigation as a possible therapy for Myc-driven human cancers.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/335/6066/348.full">http://science.sciencemag.org/content/335/6066/348.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2011).480(7375):99-103. doi:10.1038/Nature10630; PMID:22080950</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A novel recurrent mutation in MITF predisposes to familial and sporadic melanoma.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Satoru Yokoyama, Susan L Woods, Glen M Boyle, Lauren G Aoude, Stuart MacGregor, Victoria Zismann, Michael Gartside, Anne E Cust, Rizwan Haq, Mark Harland, John C Taylor, David L Duffy, Kelly Holohan, Ken Dutton-Regester, Jane M Palmer, Vanessa Bonazzi, Mitchell S Stark, Judith Symmons, Matthew H Law, Christopher Schmidt, Cathy Lanagan, Linda O'Connor, Elizabeth A Holland, Helen Schmid, Judith A Maskiell, Jodie Jetann, Megan Ferguson, Mark A Jenkins, Richard F Kefford, Graham G Giles, Bruce K Armstrong, Joanne F Aitken, John L Hopper, David C Whiteman, Paul D Pharoah, Douglas F Easton, Alison M Dunning, Julia A Newton-Bishop, Grant W Montgomery, Nicholas G Martin, Graham J Mann, D Timothy Bishop, Hensin Tsao, Jeffrey M Trent, David E Fisher, Nicholas K Hayward, Kevin M Brown</b>
      </span>
      <br/>
      <span>Affiliation: Department of Dermatology; Cutaneous Biology Research Center; Massachusetts General Hospital; Harvard Medical School; Massachusetts 02114; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>So far, two genes associated with familial melanoma have been identified, accounting for a minority of genetic risk in families. Mutations in CDKN2A account for approximately 40% of familial cases, and predisposing mutations in CDK4 have been reported in a very small number of melanoma kindreds. Here we report the whole-genome sequencing of probands from several melanoma families, which we performed in order to identify other genes associated with familial melanoma. We identify one individual carrying a novel germline variant (coding DNA sequence c.G1075A; protein sequence p.E318K; rs149617956) in the melanoma-lineage-specific oncogene microphthalmia-associated transcription factor (MITF). Although the variant co-segregated with melanoma in some but not all cases in the family, linkage analysis of 31 families subsequently identified to carry the variant generated a log of odds (lod) score of 2.7 under a dominant model, indicating E318K as a possible intermediate risk variant. Consistent with this, the E318K variant was significantly associated with melanoma in a large Australian case-control sample. Likewise, it was similarly associated in an independent case-control sample from the United Kingdom. In the Australian sample, the variant allele was significantly over-represented in cases with a family history of melanoma, multiple primary melanomas, or both. The variant allele was also associated with increased naevus count and non-blue eye colour. Functional analysis of E318K showed that MITF encoded by the variant allele had impaired sumoylation and differentially regulated several MITF targets. These data indicate that MITF is a melanoma-predisposition gene and highlight the utility of whole-genome sequencing to identify novel rare variants associated with disease susceptibility.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v480/n7375/Nature10630.html">http://www.nature.com/journal/v480/n7375/Nature10630.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2011).147(4):773-88. doi:10.1016/J.Cell.2011.08.054; PMID:22078878</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">ncRNA- and Pc2 methylation-dependent gene relocation between nuclear structures mediates gene activation programs.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Liuqing Yang, Chunru Lin, Wen Liu, Jie Zhang, Kenneth A Ohgi, Jonathan D Grinstein, Pieter C Dorrestein, Michael G Rosenfeld</b>
      </span>
      <br/>
      <span>Affiliation: Howard Hughes Medical Institute; University of California; San Diego; School of Medicine; 9500 Gilman Drive; La Jolla; CA 92093-0648; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Although eukaryotic nuclei contain distinct architectural structures associated with noncoding RNAs (ncRNAs), their potential relationship to regulated transcriptional programs remains poorly understood. Here, we report that methylation/demethylation of Polycomb 2 protein (Pc2) controls relocation of growth-control genes between Polycomb bodies (PcGs) and interchromatin granules (ICGs) in response to growth signals. This movement is the consequence of binding of methylated and unmethylated Pc2 to the ncRNAs TUG1 and MALAT1/NEAT2, located in PcGs and ICGs, respectively. These ncRNAs mediate assembly of multiple corepressors/coactivators and can serve to switch mark recognition by "readers" of the histone code. Additionally, binding of NEAT2 to unmethylated Pc2 promotes E2F1 SUMOylation, leading to activation of the growth-control gene program. These observations delineate a molecular pathway linking the actions of subnuclear structure-specific ncRNAs and nonhistone protein methylation to relocation of transcription units in the three-dimensional space of the nucleus, thus achieving coordinated gene expression programs.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(11)01216-5.html">http://www.cell.com/cell/full/S0092-8674(11)01216-5.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2011).477(7366):601-5. doi:10.1038/Nature10407; PMID:21900893</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SUMO1-dependent modulation of SERCA2a in heart failure.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Changwon Kho, Ahyoung Lee, Dongtak Jeong, Jae Gyun Oh, Antoine H Chaanine, Eddy Kizana, Woo Jin Park, Roger J Hajjar</b>
      </span>
      <br/>
      <span>Affiliation: Cardiovascular Research Center; Mount Sinai School of Medicine; 1 Gustave L. Levy Place; Box 1030; New York; New York 10029; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The calcium-transporting ATPase ATP2A2, also known as SERCA2a, is a critical ATPase responsible for Ca(2+) re-uptake during excitation-contraction coupling. Impaired Ca(2+) uptake resulting from decreased expression and reduced activity of SERCA2a is a hallmark of heart failure. Accordingly, restoration of SERCA2a expression by gene transfer has proved to be effective in improving cardiac function in heart-failure patients, as well as in animal models. The small ubiquitin-related modifier (SUMO) can be conjugated to lysine residues of target proteins, and is involved in many cellular processes. Here we show that SERCA2a is SUMOylated at lysines 480 and 585 and that this SUMOylation is essential for preserving SERCA2a ATPase activity and stability in mouse and human cells. The levels of SUMO1 and the SUMOylation of SERCA2a itself were greatly reduced in failing hearts. SUMO1 restitution by adeno-associated-virus-mediated gene delivery maintained the protein abundance of SERCA2a and markedly improved cardiac function in mice with heart failure. This effect was comparable to SERCA2A gene delivery. Moreover, SUMO1 overexpression in isolated cardiomyocytes augmented contractility and accelerated Ca(2+) decay. Transgene-mediated SUMO1 overexpression rescued cardiac dysfunction induced by pressure overload concomitantly with increased SERCA2a function. By contrast, downregulation of SUMO1 using small hairpin RNA (shRNA) accelerated pressure-overload-induced deterioration of cardiac function and was accompanied by decreased SERCA2a function. However, knockdown of SERCA2a resulted in severe contractile dysfunction both in vitro and in vivo, which was not rescued by overexpression of SUMO1. Taken together, our data show that SUMOylation is a critical post-translational modification that regulates SERCA2a function, and provide a platform for the design of novel therapeutic strategies for heart failure.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v477/n7366/Nature10407.html">http://www.nature.com/journal/v477/n7366/Nature10407.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2011).470(7334):414-8. doi:10.1038/Nature09703; PMID:21331046</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Coronin 2A mediates actin-dependent de-repression of inflammatory response genes.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Wendy Huang, Serena Ghisletti, Kaoru Saijo, Meghal Gandhi, Myriam Aouadi, Greg J Tesz, Dawn X Zhang, Joyee Yao, Michael P Czech, Bruce L Goode, Michael G Rosenfeld, Christopher K Glass</b>
      </span>
      <br/>
      <span>Affiliation: Department of Cellular and Molecular Medicine; University of California; San Diego; 9500 Gilman Drive; La Jolla; California 92093-0651; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Toll-like receptors (TLRs) function as initiators of inflammation through their ability to sense pathogen-associated molecular patterns and products of tissue damage. Transcriptional activation of many TLR-responsive genes requires an initial de-repression step in which nuclear receptor co-repressor (NCoR) complexes are actively removed from the promoters of target genes to relieve basal repression. Ligand-dependent SUMOylation of liver X receptors (LXRs) has been found to suppress TLR4-induced transcription potently by preventing the NCoR clearance step, but the underlying mechanisms remain enigmatic. Here we provide evidence that coronin 2A (CORO2A), a component of the NCoR complex of previously unknown function, mediates TLR-induced NCoR turnover by a mechanism involving interaction with oligomeric nuclear actin. SUMOylated LXRs block NCoR turnover by binding to a conserved SUMO2/SUMO3-interaction motif in CORO2A and preventing actin recruitment. Intriguingly, the LXR transrepression pathway can itself be inactivated by inflammatory signals that induce calcium/calmodulin-dependent protein kinase II&#206;&#179; (CaMKII&#206;&#179;)-dependent phosphorylation of LXRs, leading to their deSUMOylation by the SUMO protease SENP3 and release from CORO2A. These findings uncover a CORO2A-actin-dependent mechanism for the de-repression of inflammatory response genes that can be differentially regulated by phosphorylation and by nuclear receptor signalling pathways that control immunity and homeostasis.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v470/n7334/Nature09703.html">http://www.nature.com/journal/v470/n7334/Nature09703.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2010).464(7292):1192-5. doi:10.1038/Nature08963; PMID:20414307</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Listeria monocytogenes impairs SUMOylation for efficient infection.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>David Ribet, M&#195;&#169;lanie Hamon, Edith Gouin, Marie-Anne Nahori, Francis Impens, H&#195;&#169;l&#195;&#168;ne Neyret-Kahn, Kris Gevaert, Jo&#195;&#171;l Vandekerckhove, Anne Dejean, Pascale Cossart</b>
      </span>
      <br/>
      <span>Affiliation: Institut Pasteur; Unit&#195;&#169; des Interactions Bact&#195;&#169;ries-Cellules; D&#195;&#169;partement de Biologie Cellulaire et Infection; F-75015 Paris; France.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>During infection, pathogenic bacteria manipulate the host cell in various ways to allow their own replication, propagation and escape from host immune responses. Post-translational modifications are unique mechanisms that allow cells to rapidly, locally and specifically modify activity or interactions of key proteins. Some of these modifications, including phosphorylation and ubiquitylation, can be induced by pathogens. However, the effects of pathogenic bacteria on SUMOylation, an essential post-translational modification in eukaryotic cells, remain largely unknown. Here we show that infection with Listeria monocytogenes leads to a decrease in the levels of cellular SUMO-conjugated proteins. This event is triggered by the bacterial virulence factor listeriolysin O (LLO), which induces a proteasome-independent degradation of Ubc9, an essential enzyme of the SUMOylation machinery, and a proteasome-dependent degradation of some SUMOylated proteins. The effect of LLO on Ubc9 is dependent on the pore-forming capacity of the toxin and is shared by other bacterial pore-forming toxins like perfringolysin O (PFO) and pneumolysin (PLY). Ubc9 degradation was also observed in vivo in infected mice. Furthermore, we show that SUMO overexpression impairs bacterial infection. Together, our results reveal that Listeria, and probably other pathogens, dampen the host response by decreasing the SUMOylation level of proteins critical for infection.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v464/n7292/Nature08963.html">http://www.nature.com/journal/v464/n7292/Nature08963.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2010).328(5975):240-3. doi:10.1126/Science.1183424; PMID:20378816</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Arsenic trioxide controls the fate of the PML-RARalpha oncoprotein by directly binding PML.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Xiao-Wei Zhang, Xiao-Jing Yan, Zi-Ren Zhou, Fei-Fei Yang, Zi-Yu Wu, Hong-Bin Sun, Wen-Xue Liang, Ai-Xin Song, Val&#195;&#169;rie Lallemand-Breitenbach, Marion Jeanne, Qun-Ye Zhang, Huai-Yu Yang, Qiu-Hua Huang, Guang-Biao Zhou, Jian-Hua Tong, Yan Zhang, Ji-Hui Wu, Hong-Yu Hu, Hugues de Th&#195;&#169;, Sai-Juan Chen, Zhu Chen</b>
      </span>
      <br/>
      <span>Affiliation: State Key Laboratory of Medical Genomics; Shanghai Institute of Hematology; Rui Jin Hospital affiliated to Shanghai Jiao Tong University School of Medicine; 197 Rui Jin Road II; Shanghai 200025; China.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Arsenic, an ancient drug used in traditional Chinese medicine, has attracted worldwide interest because it shows substantial anticancer activity in patients with acute promyelocytic leukemia (APL). Arsenic trioxide (As2O3) exerts its therapeutic effect by promoting degradation of an oncogenic protein that drives the growth of APL cells, PML-RARalpha (a fusion protein containing sequences from the PML zinc finger protein and retinoic acid receptor alpha). PML and PML-RARalpha degradation is triggered by their SUMOylation, but the mechanism by which As2O3 induces this posttranslational modification is unclear. Here we show that arsenic binds directly to cysteine residues in zinc fingers located within the RBCC domain of PML-RARalpha and PML. Arsenic binding induces PML oligomerization, which increases its interaction with the small ubiquitin-like protein modifier (SUMO)-conjugating enzyme UBC9, resulting in enhanced SUMOylation and degradation. The identification of PML as a direct target of As2O3 provides new insights into the drug's mechanism of action and its specificity for APL.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/328/5975/240.full">http://science.sciencemag.org/content/328/5975/240.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2009).462(7275):935-9. doi:10.1038/Nature08657; PMID:20016603</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Mammalian SUMO E3-ligases PIAS1 and PIAS4 promote responses to DNA double-strand breaks.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yaron Galanty, Rimma Belotserkovskaya, Julia Coates, Sophie Polo, Kyle M Miller, Stephen P Jackson</b>
      </span>
      <br/>
      <span>Affiliation: The Wellcome Trust and Cancer Research UK Gurdon Institute; and Department of Biochemistry; University of Cambridge; Tennis Court Road; Cambridge CB2 1QN; UK.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>DNA double-strand breaks (DSBs) are highly cytotoxic lesions that are generated by ionizing radiation and various DNA-damaging chemicals. Following DSB formation, cells activate the DNA-damage response (DDR) protein kinases ATM, ATR and DNA-PK (also known as PRKDC). These then trigger histone H2AX (also known as H2AFX) phosphorylation and the accumulation of proteins such as MDC1, 53BP1 (also known as TP53BP1), BRCA1, CtIP (also known as RBBP8), RNF8 and RNF168/RIDDLIN into ionizing radiation-induced foci (IRIF) that amplify DSB signalling and promote DSB repair. Attachment of small ubiquitin-related modifier (SUMO) to target proteins controls diverse cellular functions. Here, we show that SUMO1, SUMO2 and SUMO3 accumulate at DSB sites in mammalian cells, with SUMO1 and SUMO2/3 accrual requiring the E3 ligase enzymes PIAS4 and PIAS1. We also establish that PIAS1 and PIAS4 are recruited to damage sites via mechanisms requiring their SAP domains, and are needed for the productive association of 53BP1, BRCA1 and RNF168 with such regions. Furthermore, we show that PIAS1 and PIAS4 promote DSB repair and confer ionizing radiation resistance. Finally, we establish that PIAS1 and PIAS4 are required for effective ubiquitin-adduct formation mediated by RNF8, RNF168 and BRCA1 at sites of DNA damage. These findings thus identify PIAS1 and PIAS4 as components of the DDR and reveal how protein recruitment to DSB sites is controlled by coordinated SUMOylation and ubiquitylation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v462/n7275/Nature08657.html">http://www.nature.com/journal/v462/n7275/Nature08657.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2009).462(7275):886-90. doi:10.1038/Nature08593; PMID:20016594</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The SUMO modification pathway is involved in the BRCA1 response to genotoxic stress.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Joanna R Morris, Chris Boutell, Melanie Keppler, Ruth Densham, Daniel Weekes, Amin Alamshah, Laura Butler, Yaron Galanty, Laurent Pangon, Tai Kiuchi, Tony Ng, Ellen Solomon</b>
      </span>
      <br/>
      <span>Affiliation: Department of Medical and Molecular Genetics; King's College London; Guy's Medical School Campus; London SE1 9RT; UK. jo.morris@genetics.kcl.ac.uk</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Mutations in BRCA1 are associated with a high risk of breast and ovarian cancer. BRCA1 participates in the DNA damage response and acts as a ubiquitin ligase. However, its regulation remains poorly understood. Here we report that BRCA1 is modified by small ubiquitin-like modifier (SUMO) in response to genotoxic stress, and co-localizes at sites of DNA damage with SUMO1, SUMO2/3 and the SUMO-conjugating enzyme Ubc9. PIAS SUMO E3 ligases co-localize with and modulate SUMO modification of BRCA1, and are required for BRCA1 ubiquitin ligase activity in cells. In vitro SUMO modification of the BRCA1/BARD1 heterodimer greatly increases its ligase activity, identifying it as a SUMO-regulated ubiquitin ligase (SRUbL). Further, PIAS SUMO ligases are required for complete accumulation of double-stranded DNA (dsDNA) damage-repair proteins subsequent to RNF8 accrual, and for proficient double-strand break repair. These data demonstrate that the SUMOylation pathway plays a significant role in mammalian DNA damage response.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v462/n7275/Nature08593.html">http://www.nature.com/journal/v462/n7275/Nature08593.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2009).324(5932):1327-30. doi:10.1126/Science.1172871; PMID:19498170</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Rhes, a striatal specific protein, mediates mutant-huntingtin cytotoxicity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Srinivasa Subramaniam, Katherine M Sixt, Roxanne Barrow, Solomon H Snyder</b>
      </span>
      <br/>
      <span>Affiliation: Solomon H. Snyder Department of Neuroscience; Johns Hopkins University School of Medicine; 725 North Wolfe Street; Baltimore; MD 21205; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Huntington's disease (HD) is caused by a polyglutamine repeat in the protein huntingtin (Htt) with mutant Htt (mHtt) expressed throughout the body and similarly in all brain regions. Yet, HD neuropathology is largely restricted to the corpus striatum. We report that the small guanine nucleotide-binding protein Rhes, which is localized very selectively to the striatum, binds physiologically to mHtt. Using cultured cells, we found Rhes induces sumoylation of mHtt, which leads to cytotoxicity. Thus, Rhes-mHtt interactions can account for the localized neuropathology of HD.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/324/5932/1327.full">http://science.sciencemag.org/content/324/5932/1327.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2008).456(7224):915-20. doi:10.1038/Nature07587; PMID:19092928</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SUMOylation regulates Rad18-mediated template switch.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Dana Branzei, Fabio Vanoli, Marco Foiani</b>
      </span>
      <br/>
      <span>Affiliation: IFOM; the FIRC Institute for Molecular Oncology Foundation; IFOM-IEO Campus; Via Adamello 16; 20139 Milan; Italy. dana.branzei@ifom-ieo-campus.it</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Replication by template switch is thought to mediate DNA damage-bypass and fillings of gaps. Gap-filling repair requires homologous recombination as well as Rad18- and Rad5-mediated proliferating cell nuclear antigen (PCNA) polyubiquitylation. However, it is unclear whether these processes are coordinated, and the physical evidence for Rad18-Rad5-dependent template switch at replication forks is still elusive. Here we show, using genetic and physical approaches, that in budding yeast (Saccharomyces cerevisiae) Rad18 is required for the formation of X-shaped sister chromatid junctions (SCJs) at damaged replication forks through a process involving PCNA polyubiquitylation and the ubiquitin-conjugating enzymes Mms2 and Ubc13. The Rad18-Mms2-mediated damage-bypass through SCJs requires the small ubiquitin-like modifier (SUMO)-conjugating enzyme Ubc9 and SUMOylated PCNA, and is coordinated with Rad51-dependent recombination events. We propose that the Rad18-Rad5-Mms2-dependent SCJs represent template switch events. Altogether, our results unmask a role for PCNA ubiquitylation and SUMOylation pathways in promoting transient damage-induced replication-coupled recombination events involving sister chromatids at replication forks.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v456/n7224/Nature07587.html">http://www.nature.com/journal/v456/n7224/Nature07587.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2008).133(1):103-15. doi:10.1016/J.Cell.2008.01.045; PMID:18394993</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Resolution of sister centromeres requires RanBP2-mediated SUMOylation of topoisomerase IIalpha.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Meelad M Dawlaty, Liviu Malureanu, Karthik B Jeganathan, Esther Kao, Claudio Sustmann, Samuel Tahk, Ke Shuai, Rudolf Grosschedl, Jan M van Deursen</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry and Molecular Biology; Mayo Clinic College of Medicine; Rochester; MN 55905; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>RanBP2 is a nucleoporin with SUMO E3 ligase activity that functions in both nucleocytoplasmic transport and mitosis. However, the biological relevance of RanBP2 and the in vivo targets of its E3 ligase activity are unknown. Here we show that animals with low amounts of RanBP2 develop severe aneuploidy in the absence of overt transport defects. The main chromosome segregation defect in cells from these mice is anaphase-bridge formation. Topoisomerase IIalpha (Topo IIalpha), which decatenates sister centromeres prior to anaphase onset to prevent bridges, fails to accumulate at inner centromeres when RanBP2 levels are low. We find that RanBP2 sumoylates Topo IIalpha in mitosis and that this modification is required for its proper localization to inner centromeres. Furthermore, mice with low amounts of RanBP2 are highly sensitive to tumor formation. Together, these data identify RanBP2 as a chromosomal instability gene that regulates Topo IIalpha by sumoylation and suppresses tumorigenesis.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(08)00210-9.html">http://www.cell.com/cell/full/S0092-8674(08)00210-9.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2008).451(7174):81-5. doi:10.1038/Nature06420; PMID:18172500</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Poly(ADP-ribose)-binding zinc finger motifs in DNA repair/checkpoint proteins.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Ivan Ahel, Dragana Ahel, Takahiro Matsusaka, Allison J Clark, Jonathon Pines, Simon J Boulton, Stephen C West</b>
      </span>
      <br/>
      <span>Affiliation: Genetic Recombination and; Cancer Research UK London Research Institute; Clare Hall Laboratories; South Mimms; Herts EN6 3LD; UK.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Post-translational modification (PTM) of proteins plays an important part in mediating protein interactions and/or the recruitment of specific protein targets. PTM can be mediated by the addition of functional groups (for example, acetylation or phosphorylation), peptides (for example, ubiquitylation or sumoylation), or nucleotides (for example, poly(ADP-ribosyl)ation). Poly(ADP-ribosyl)ation often involves the addition of long chains of ADP-ribose units, linked by glycosidic ribose-ribose bonds, and is critical for a wide range of processes, including DNA repair, regulation of chromosome structure, transcriptional regulation, mitosis and apoptosis. Here we identify a novel poly(ADP-ribose)-binding zinc finger (PBZ) motif in a number of eukaryotic proteins involved in the DNA damage response and checkpoint regulation. The PBZ motif is also required for post-translational poly(ADP-ribosyl)ation. We demonstrate interaction of poly(ADP-ribose) with this motif in two representative human proteins, APLF (aprataxin PNK-like factor) and CHFR (checkpoint protein with FHA and RING domains), and show that the actions of CHFR in the antephase checkpoint are abrogated by mutations in PBZ or by inhibition of poly(ADP-ribose) synthesis.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v451/n7174/Nature06420.html">http://www.nature.com/journal/v451/n7174/Nature06420.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2007).131(3):584-95. doi:10.1016/J.Cell.2007.08.045; PMID:17981124</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SUMO-specific protease 1 is essential for stabilization of HIF1alpha during hypoxia.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jinke Cheng, Xunlei Kang, Sui Zhang, Edward T H Yeh</b>
      </span>
      <br/>
      <span>Affiliation: Department of Cardiology; The University of Texas MD Anderson Cancer Center; The University of Texas; Houston Health Science Center; Houston; TX 77030; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>SUMOylation is a dynamic process, catalyzed by SUMO-specific ligases and reversed by Sentrin/SUMO-specific proteases (SENPs). The physiologic consequences of SUMOylation and deSUMOylation are not fully understood. Here we investigate the phenotypes of mice lacking SENP1 and find that SENP1(-/-) embryos show severe fetal anemia stemming from deficient erythropoietin (Epo) production and die midgestation. We determine that SENP1 controls Epo production by regulating the stability of hypoxia-inducible factor 1alpha (HIF1alpha) during hypoxia. Hypoxia induces SUMOylation of HIF1alpha, which promotes its binding to a ubiquitin ligase, von Hippel-Lindau (VHL) protein, through a proline hydroxylation-independent mechanism, leading to its ubiquitination and degradation. In SENP1(-/-) MEFs, hypoxia-induced transcription of HIF1alpha-dependent genes such as vascular endothelial growth factor (VEGF) and glucose transporter 1 (Glut-1) is markedly reduced. These results show that SENP1 plays a key role in the regulation of the hypoxic response through regulation of HIF1alpha stability and that SUMOylation can serve as a direct signal for ubiquitin-dependent degradation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(07)01143-9.html">http://www.cell.com/cell/full/S0092-8674(07)01143-9.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2007).131(2):309-23. doi:10.1016/J.Cell.2007.07.044; PMID:17956732</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">RSUME, a small RWD-containing protein, enhances SUMO conjugation and stabilizes HIF-1alpha during hypoxia.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Alberto Carbia-Nagashima, Juan Gerez, Carolina Perez-Castro, Marcelo Paez-Pereda, Susana Silberstein, G&#195;&#188;nter K Stalla, Florian Holsboer, Eduardo Arzt</b>
      </span>
      <br/>
      <span>Affiliation: Departamento de Fisiolog&#195;&#173;a y Biolog&#195;&#173;a Molecular y Celular; Facultad de Ciencias Exactas y Naturales; Universidad de Buenos Aires; Buenos Aires; Argentina.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>SUMO conjugation to proteins is involved in the regulation of diverse cellular functions. We have identified a protein, RWD-containing sumoylation enhancer (RSUME), that enhances overall SUMO-1, -2, and -3 conjugation by interacting with the SUMO conjugase Ubc9. RSUME increases noncovalent binding of SUMO-1 to Ubc9 and enhances Ubc9 thioester formation and SUMO polymerization. RSUME enhances the sumoylation of IkB in vitro and in cultured cells, leading to an inhibition of NF-kB transcriptional activity. RSUME is induced by hypoxia and enhances the sumoylation of HIF-1alpha, promoting its stabilization and transcriptional activity during hypoxia. Disruption of the RWD domain structure of RSUME demonstrates that this domain is critical for RSUME action. Together, these findings point to a central role of RSUME in the regulation of sumoylation and, hence, several critical regulatory pathways in mammalian cells.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(07)01024-0.html">http://www.cell.com/cell/full/S0092-8674(07)01024-0.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2007).130(3):563-9. doi:10.1016/J.Cell.2007.06.012; PMID:17693262</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Does sumoylation control K2P1/TWIK1 background K+ channels?</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Sylvain Feliciangeli, Sa&#195;&#175;d Bendahhou, Guillaume Sandoz, Pierre Gounon, Markus Reichold, Richard Warth, Michel Lazdunski, Jacques Barhanin, Florian Lesage</b>
      </span>
      <br/>
      <span>Affiliation: Institut de Pharmacologie Mol&#195;&#169;culaire et Cellulaire; CNRS UMR6097; Institut Paul Hamel; 660; route des lucioles; 06560 Valbonne; France.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>A novel model for the regulation of cell excitability has recently been proposed. It originates from the observation that the background K(+) channel K2P1 (TWIK1) may be silenced by sumoylation in Xenopus oocytes and that inactivation of the putative sumoylation site (mutation K274E) gives rise to robust current expression in transfected COS-7 cells. Here, we show that only the mutation K274E, and not K274R, is associated with an increase of K2P1 current density, suggesting a charge effect of K274E. Furthermore, we failed to observe any band shift by western blot analysis that would confirm an eventual sumoylation of K2P1 in COS-7 cells and oocytes.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(07)00778-7.html">http://www.cell.com/cell/full/S0092-8674(07)00778-7.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2007).129(5):903-14. doi:10.1016/J.Cell.2007.03.056; PMID:17540171</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Proinflammatory stimuli induce IKKalpha-mediated phosphorylation of PIAS1 to restrict inflammation and immunity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Bin Liu, Yonghui Yang, Vasili Chernishof, Rachel R Ogorzalek Loo, Hyunduk Jang, Samuel Tahk, Randy Yang, Sheldon Mink, David Shultz, Clifford J Bellone, Joseph A Loo, Ke Shuai</b>
      </span>
      <br/>
      <span>Affiliation: Division of Hematology-Oncology; Department of Medicine; University of California Los Angeles; Los Angeles; CA 90095; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>How inflammatory stimuli signal to the nucleus to restrict inflammation is poorly understood. Protein inhibitor of activated STAT1 (PIAS1), a transcriptional regulator that possesses small ubiquitin-related modifier (SUMO) E3 ligase activity, inhibits immune responses by selectively blocking the binding of NF-kappaB and STAT1 to gene promoters. We report here that PIAS1 becomes rapidly phosphorylated on Ser90 residue in response to various inflammatory stimuli. Mutational studies indicate that Ser90 phosphorylation is required for PIAS1 to repress transcription. Upon TNF treatment, wild-type PIAS1, but not the Ser90A mutant, becomes rapidly associated with the promoters of NF-kappaB target genes. Furthermore, IKKalpha, but not IKKbeta, interacts with PIAS1 in vivo and mediates PIAS1 Ser90 phosphorylation, a process that requires the SUMO ligase activity of PIAS1. Our results identify a signaling pathway in which proinflammatory stimuli activate the IKKalpha-mediated sumoylation-dependent phosphorylation of PIAS1 for the immediate repression of inflammatory gene activation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(07)00538-7.html">http://www.cell.com/cell/full/S0092-8674(07)00538-7.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2007).447(7142):321-5. doi:10.1038/Nature05736; PMID:17486098</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SUMOylation regulates kainate-receptor-mediated synaptic transmission.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>St&#195;&#169;phane Martin, Atsushi Nishimune, Jack R Mellor, Jeremy M Henley</b>
      </span>
      <br/>
      <span>Affiliation: MRC Centre for Synaptic Plasticity; Anatomy Department; University Walk; University of Bristol; Bristol; BS8 1TD; UK.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The small ubiquitin-like modifier protein (SUMO) regulates transcriptional activity and the translocation of proteins across the nuclear membrane. The identification of SUMO substrates outside the nucleus is progressing but little is yet known about the wider cellular role of protein SUMOylation. Here we report that in rat hippocampal neurons multiple SUMOylation targets are present at synapses and we show that the kainate receptor subunit GluR6 is a SUMO substrate. SUMOylation of GluR6 regulates endocytosis of the kainate receptor and modifies synaptic transmission. GluR6 exhibits low levels of SUMOylation under resting conditions and is rapidly SUMOylated in response to a kainate but not an N-methyl-D-aspartate (NMDA) treatment. Reducing GluR6 SUMOylation using the SUMO-specific isopeptidase SENP-1 prevents kainate-evoked endocytosis of the kainate receptor. Furthermore, a mutated non-SUMOylatable form of GluR6 is not endocytosed in response to kainate in COS-7 cells. Consistent with this, electrophysiological recordings in hippocampal slices demonstrate that kainate-receptor-mediated excitatory postsynaptic currents are decreased by SUMOylation and enhanced by deSUMOylation. These data reveal a previously unsuspected role for SUMO in the regulation of synaptic function.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v447/n7142/Nature05736.html">http://www.nature.com/journal/v447/n7142/Nature05736.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2006).127(3):509-22. doi:10.1016/J.Cell.2006.08.050; PMID:17081974</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Ubc9- and mms21-mediated sumoylation counteracts recombinogenic events at damaged replication forks.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Dana Branzei, Julie Sollier, Giordano Liberi, Xiaolan Zhao, Daisuke Maeda, Masayuki Seki, Takemi Enomoto, Kunihiro Ohta, Marco Foiani</b>
      </span>
      <br/>
      <span>Affiliation: FIRC Institute of Molecular Oncology Foundation and Department of Biomedical Sciences and Biotechnology; Universit&#195;&#160; degli Studi di Milano; Via Adamello 16; 20139 Milan; Italy. dana.branzei@ifom-ieo-campus.it</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The Ubc9 SUMO-conjugating enzyme and the Siz1 SUMO ligase sumoylate several repair and recombination proteins, including PCNA. Sumoylated PCNA binds Srs2, a helicase counteracting certain recombination events. Here we show that ubc9 mutants depend on checkpoint, recombination, and replication genes for growth. ubc9 cells maintain stalled-fork stability but exhibit a Rad51-dependent accumulation of cruciform structures during replication of damaged templates. Mutations in the Mms21 SUMO ligase resemble the ubc9 mutations. However, siz1, srs2, or pcna mutants altered in sumoylation do not exhibit the ubc9/mms21 phenotype. Like ubc9/mms21 mutants, sgs1 and top3 mutants also accumulate X molecules at damaged forks, and Sgs1/BLM is sumoylated. We propose that Ubc9 and Mms21 act in concert with Sgs1 to resolve the X structures formed during replication. Our results indicate that Ubc9- and Mms21-mediated sumoylation functions as a regulatory mechanism, different from that of replication checkpoints, to prevent pathological accumulation of cruciform structures at damaged forks.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(06)01278-5.html">http://www.cell.com/cell/full/S0092-8674(06)01278-5.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2006).127(3):455-7. doi:10.1016/J.Cell.2006.10.019; PMID:17081966</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A SUMOry of DNA replication: synthesis, damage, and repair.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Hannah L Klein</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry; New York University School of Medicine; New York; NY 10016; USA. hannah.klein@med.nyu.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Recombination at stalled replication forks is regulated at an early stage by sumoylation. In this issue of Cell, Branzei et al. show that the Ubc9/SUMO modification pathway controls the accumulation of cruciform structures at stalled forks.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(06)01349-3.html">http://www.cell.com/cell/full/S0092-8674(06)01349-3.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2006).313(5794):1751. doi:10.1126/Science.1128406; PMID:16990542</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SUMO1 haploinsufficiency leads to cleft lip and palate.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Fowzan S Alkuraya, Irfan Saadi, Jennifer J Lund, Annick Turbe-Doan, Cynthia C Morton, Richard L Maas</b>
      </span>
      <br/>
      <span>Affiliation: Division of Genetics; Department of Medicine; Brigham and Women's Hospital and Harvard Medical School; NRB-458; 77 Louis Pasteur; Boston; MA 02115; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The posttranslational modification sumoylation can have multiple effects on its substrate proteins. We studied a patient with isolated cleft lip and palate and a balanced chromosomal translocation that disrupts the SUMO1 (small ubiquitin-related modifier) gene, resulting in haploinsufficiency. In mouse, we found that Sumo1 is expressed in the developing lip and palate and that a Sumo1 hypomorphic allele manifests an incompletely penetrant orofacial clefting phenotype. Products of several genes implicated in clefting are sumoylated, and the Sumo1 hypomorphic allele interacts genetically with a loss-of-function allele for one of these loci. Thus, sumoylation defines a network of genes important for palatogenesis.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/313/5794/1751.full">http://science.sciencemag.org/content/313/5794/1751.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2006).311(5763):1012-7. doi:10.1126/Science.1122513; PMID:16484498</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A calcium-regulated MEF2 sumoylation switch controls postsynaptic differentiation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Aryaman Shalizi, Brice Gaudilli&#195;&#168;re, Zengqiang Yuan, Judith Stegm&#195;&#188;ller, Takahiro Shirogane, Qingyuan Ge, Yi Tan, Brenda Schulman, J Wade Harper, Azad Bonni</b>
      </span>
      <br/>
      <span>Affiliation: Department of Pathology; Harvard Medical School; 77 Louis Pasteur Avenue; Boston; MA 02115; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Postsynaptic differentiation of dendrites is an essential step in synapse formation. We report here a requirement for the transcription factor myocyte enhancer factor 2A (MEF2A) in the morphogenesis of postsynaptic granule neuron dendritic claws in the cerebellar cortex. A transcriptional repressor form of MEF2A that is sumoylated at lysine-403 promoted dendritic claw differentiation. Activity-dependent calcium signaling induced a calcineurin-mediated dephosphorylation of MEF2A at serine-408 and, thereby, promoted a switch from sumoylation to acetylation at lysine-403, which led to inhibition of dendritic claw differentiation. Our findings define a mechanism underlying postsynaptic differentiation that may modulate activity-dependent synapse development and plasticity in the brain.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/311/5763/1012.full">http://science.sciencemag.org/content/311/5763/1012.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2005).123(6):1079-92. doi:10.1016/J.Cell.2005.09.036; PMID:16360037</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">PIDD mediates NF-kappaB activation in response to DNA damage.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Sophie Janssens, Antoine Tinel, Saskia Lippens, J&#195;&#188;rg Tschopp</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry; University of Lausanne; BIL Biomedical Research Center; Chemin des Boveresses 155; CH-1066 Epalinges; Switzerland.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Activation of NF-kappaB following genotoxic stress allows time for DNA-damage repair and ensures cell survival accounting for acquired chemoresistance, an impediment to effective cancer therapy. Despite this clinical relevance, little is known about pathways that enable genotoxic-stress-induced NF-kappaB induction. Previously, we reported a role for the p53-inducible death-domain-containing protein, PIDD, in caspase-2 activation and apoptosis in response to DNA damage. We now demonstrate that PIDD plays a critical role in DNA-damage-induced NF-kappaB activation. Upon genotoxic stress, a complex between PIDD, the kinase RIP1, and a component of the NF-kappaB-activating kinase complex, NEMO, is formed. PIDD expression enhances genotoxic-stress-induced NF-kappaB activation through augmented sumoylation and ubiquitination of NEMO. Depletion of PIDD and RIP1, but not caspase-2, abrogates DNA-damage-induced NEMO modification and NF-kappaB activation. We propose that PIDD acts as a molecular switch, controlling the balance between life and death upon DNA damage.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(05)01042-1.html">http://www.cell.com/cell/full/S0092-8674(05)01042-1.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2005).123(6):980-2. doi:10.1016/J.Cell.2005.11.025; PMID:16360026</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">PIDD: a switch hitter.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zhao-Hui Wu, Angela Mabb, Shigeki Miyamoto</b>
      </span>
      <br/>
      <span>Affiliation: Department of Pharmacology; University of Wisconsin-Madison; 1300 University Avenue; Madison; WI 53706; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>In response to genotoxic stress, the PIDD protein promotes apoptosis downstream of the tumor-suppressor protein p53. In this issue of Cell, Janssens et al. (2005) demonstrate that, in response to such stress, PIDD forms a nuclear complex that enhances sumoylation of NEMO. This modification is important for the activation of the antiapoptotic transcription factor NF-kappaB.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(05)01276-6.html">http://www.cell.com/cell/full/S0092-8674(05)01276-6.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2005).437(7059):759-63. doi:10.1038/Nature03988; PMID:16127449</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A SUMOylation-dependent pathway mediates transrepression of inflammatory response genes by PPAR-gamma.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Gabriel Pascual, Amy L Fong, Sumito Ogawa, Amir Gamliel, Andrew C Li, Valentina Perissi, David W Rose, Timothy M Willson, Michael G Rosenfeld, Christopher K Glass</b>
      </span>
      <br/>
      <span>Affiliation: Department of Cellular and Molecular Medicine; University of California San Diego; 9500 Gilman Drive; La Jolla; California 92093; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Peroxisome proliferator-activated receptor-gamma (PPAR-gamma) has essential roles in adipogenesis and glucose homeostasis, and is a molecular target of insulin-sensitizing drugs. Although the ability of PPAR-gamma agonists to antagonize inflammatory responses by transrepression of nuclear factor kappa B (NF-kappaB) target genes is linked to antidiabetic and antiatherogenic actions, the mechanisms remain poorly understood. Here we report the identification of a molecular pathway by which PPAR-gamma represses the transcriptional activation of inflammatory response genes in mouse macrophages. The initial step of this pathway involves ligand-dependent SUMOylation of the PPAR-gamma ligand-binding domain, which targets PPAR-gamma to nuclear receptor corepressor (NCoR)-histone deacetylase-3 (HDAC3) complexes on inflammatory gene promoters. This in turn prevents recruitment of the ubiquitylation/19S proteosome machinery that normally mediates the signal-dependent removal of corepressor complexes required for gene activation. As a result, NCoR complexes are not cleared from the promoter and target genes are maintained in a repressed state. This mechanism provides an explanation for how an agonist-bound nuclear receptor can be converted from an activator of transcription to a promoter-specific repressor of NF-kappaB target genes that regulate immunity and homeostasis.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v437/n7059/Nature03988.html">http://www.nature.com/journal/v437/n7059/Nature03988.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2005).309(5739):1390-4. doi:10.1126/Science.1110689; PMID:16109848</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Circadian clock control by SUMOylation of BMAL1.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Luca Cardone, Jun Hirayama, Francesca Giordano, Teruya Tamaru, Jorma J Palvimo, Paolo Sassone-Corsi</b>
      </span>
      <br/>
      <span>Affiliation: Institut de G&#195;&#169;n&#195;&#169;tique et de Biologie Mol&#195;&#169;culaire et Cellulaire; 1 rue Laurent Fries; 67404 Illkirch; Strasbourg; France.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The molecular machinery that governs circadian rhythmicity is based on clock proteins organized in regulatory feedback loops. Although posttranslational modification of clock proteins is likely to finely control their circadian functions, only limited information is available to date. Here, we show that BMAL1, an essential transcription factor component of the clock mechanism, is SUMOylated on a highly conserved lysine residue (Lys259) in vivo. BMAL1 shows a circadian pattern of SUMOylation that parallels its activation in the mouse liver. SUMOylation of BMAL1 requires and is induced by CLOCK, the heterodimerization partner of BMAL1. Ectopic expression of a SUMO-deficient BMAL1 demonstrates that SUMOylation plays an important role in BMAL1 circadian expression and clock rhythmicity. This reveals an additional level of regulation within the core mechanism of the circadian clock.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/309/5739/1390.full">http://science.sciencemag.org/content/309/5739/1390.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2005).121(1):37-47. doi:10.1016/J.Cell.2005.01.019; PMID:15820677</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Sumoylation silences the plasma membrane leak K+ channel K2P1.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Sindhu Rajan, Leigh D Plant, Michael L Rabin, Margaret H Butler, Steve A N Goldstein</b>
      </span>
      <br/>
      <span>Affiliation: Department of Pediatrics; Institute for Molecular Pediatric Sciences; Pritzker School of Medicine; University of Chicago; 5721 South Maryland Avenue; Chicago; Illinois 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Reversible, covalent modification with small ubiquitin-related modifier proteins (SUMOs) is known to mediate nuclear import/export and activity of transcription factors. Here, the SUMO pathway is shown to operate at the plasma membrane to control ion channel function. SUMO-conjugating enzyme is seen to be resident in plasma membrane, to assemble with K2P1, and to modify K2P1 lysine 274. K2P1 had not previously shown function despite mRNA expression in heart, brain, and kidney and sequence features like other two-P loop K+ leak (K2P) pores that control activity of excitable cells. Removal of the peptide adduct by SUMO protease reveals K2P1 to be a K+-selective, pH-sensitive, openly rectifying channel regulated by reversible peptide linkage.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(05)00093-0.html">http://www.cell.com/cell/full/S0092-8674(05)00093-0.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2004).304(5667):100-4. doi:10.1126/Science.1092194; PMID:15064418</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SUMO modification of Huntingtin and Huntington's disease pathology.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Joan S Steffan, Namita Agrawal, Judit Pallos, Erica Rockabrand, Lloyd C Trotman, Natalia Slepko, Katalin Illes, Tamas Lukacsovich, Ya-Zhen Zhu, Elena Cattaneo, Pier Paolo Pandolfi, Leslie Michels Thompson, J Lawrence Marsh</b>
      </span>
      <br/>
      <span>Affiliation: Department of Psychiatry and Human Behavior; Gillespie 2121; University of California; Irvine; CA 92697; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Huntington's disease (HD) is characterized by the accumulation of a pathogenic protein, Huntingtin (Htt), that contains an abnormal polyglutamine expansion. Here, we report that a pathogenic fragment of Htt (Httex1p) can be modified either by small ubiquitin-like modifier (SUMO)-1 or by ubiquitin on identical lysine residues. In cultured cells, SUMOylation stabilizes Httex1p, reduces its ability to form aggregates, and promotes its capacity to repress transcription. In a Drosophila model of HD, SUMOylation of Httex1p exacerbates neurodegeneration, whereas ubiquitination of Httex1p abrogates neurodegeneration. Lysine mutations that prevent both SUMOylation and ubiquitination of Httex1p reduce HD pathology, indicating that the contribution of SUMOylation to HD pathology extends beyond preventing Htt ubiquitination and degradation.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/304/5667/100.full">http://science.sciencemag.org/content/304/5667/100.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2003).115(5):565-76. doi:; PMID:14651848</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Sequential modification of NEMO/IKKgamma by SUMO-1 and ubiquitin mediates NF-kappaB activation by genotoxic stress.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Tony T Huang, Shelly M Wuerzberger-Davis, Zhao-Hui Wu, Shigeki Miyamoto</b>
      </span>
      <br/>
      <span>Affiliation: Department of Pharmacology; University of Wisconsin-Madison; 301 SMI; 1300 University Avenue; Madison; WI 53706; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The transcription factor NF-kappaB is critical for setting the cellular sensitivities to apoptotic stimuli, including DNA damaging anticancer agents. Central to NF-kappaB signaling pathways is NEMO/IKKgamma, the regulatory subunit of the cytoplasmic IkappaB kinase (IKK) complex. While NF-kappaB activation by genotoxic stress provides an attractive paradigm for nuclear-to-cytoplasmic signaling pathways, the mechanism by which nuclear DNA damage modulates NEMO to activate cytoplasmic IKK remains unknown. Here, we show that genotoxic stress causes nuclear localization of IKK-unbound NEMO via site-specific SUMO-1 attachment. Surprisingly, this sumoylation step is ATM-independent, but nuclear localization allows subsequent ATM-dependent ubiquitylation of NEMO to ultimately activate IKK in the cytoplasm. Thus, genotoxic stress induces two independent signaling pathways, SUMO-1 modification and ATM activation, which work in concert to sequentially cause nuclear targeting and ubiquitylation of free NEMO to permit the NF-kappaB survival pathway. These SUMO and ubiquitin modification pathways may serve as anticancer drug targets.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S009286740300895X.html">http://www.cell.com/cell/full/S009286740300895X.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2003).113(1):127-37. doi:; PMID:12679040</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The polycomb protein Pc2 is a SUMO E3.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Michael H Kagey, Tiffany A Melhuish, David Wotton</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry and Molecular Genetics; Center for Cell Signaling; University of Virginia; Charlottesville; VA 22908; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Polycomb group (PcG) proteins form large multimeric complexes (PcG bodies) which are involved in the stable repression of gene expression. The human PcG protein, Pc2, has been shown to recruit the transcriptional corepressor, CtBP, to PcG bodies. We show that CtBP is sumoylated at a single lysine. In vitro, CtBP sumoylation minimally requires the SUMO E1 and E2 (Ubc9) and SUMO-1. However, Pc2 dramatically enhances CtBP sumoylation. In vivo, this is likely due to the ability of Pc2 to recruit both CtBP and Ubc9 to PcG bodies, thereby bringing together substrate and E2, and stimulating the transfer of SUMO to CtBP. These results demonstrate that Pc2 is a SUMO E3, and suggest that PcG bodies may be sumoylation centers.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092867403001594.html">http://www.cell.com/cell/full/S0092867403001594.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Review</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2003).112(1):11-7. doi:; PMID:12526789</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Regulating the regulators: lysine modifications make their mark.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Richard N Freiman, Robert Tjian</b>
      </span>
      <br/>
      <span>Affiliation: Howard Hughes Medical Institute; Department of Molecular and Cell Biology; University of California; Berkeley; CA 94720; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Decades of research have uncovered much of the molecular machinery responsible for establishing and maintaining proper gene transcription patterns in eukaryotes. Although the composition of this machinery is largely known, mechanisms regulating its activity by covalent modification are just coming into focus. Here, we review several cases of ubiquitination, sumoylation, and acetylation that link specific covalent modification of the transcriptional apparatus to their regulatory function. We propose that potential cascades of modifications serve as molecular rheostats that fine-tune the control of transcription in diverse organisms.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092867402012783.html">http://www.cell.com/cell/full/S0092867402012783.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2002).108(1):109-20. doi:; PMID:11792325</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The nucleoporin RanBP2 has SUMO1 E3 ligase activity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Andrea Pichler, Andreas Gast, Jacob S Seeler, Anne Dejean, Frauke Melchior</b>
      </span>
      <br/>
      <span>Affiliation: Max-Planck Institute for Biochemistry; Am Klopferspitz 18a; 82152 Martinsried; Germany.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Posttranslational modification with SUMO1 regulates protein/protein interactions, localization, and stability. SUMOylation requires the E1 enzyme Aos1/Uba2 and the E2 enzyme Ubc9. A family of E3-like factors, PIAS proteins, was discovered recently. Here we show that the nucleoporin RanBP2/Nup358 also has SUMO1 E3-like activity. RanBP2 directly interacts with the E2 enzyme Ubc9 and strongly enhances SUMO1-transfer from Ubc9 to the SUMO1 target Sp100. The E3-like activity is contained within a 33 kDa domain of RanBP2 that lacks RING finger motifs and does not resemble PIAS family proteins. Our findings place SUMOylation at the cytoplasmic filaments of the NPC and suggest that, at least for some substrates, modification and nuclear import are linked events.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S009286740100633X.html">http://www.cell.com/cell/full/S009286740100633X.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2001).106(6):735-44. doi:; PMID:11572779</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">An E3-like factor that promotes SUMO conjugation to the yeast septins.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>E S Johnson, A A Gupta</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry and Molecular Pharmacology; Thomas Jefferson University; Philadelphia; PA 19107; USA. erica.johnson@mail.tju.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Covalent attachment of the ubiquitin-related protein SUMO to other proteins participates in many processes including signal transduction, transcriptional regulation, and growth control. We report the characterization of Siz1 as an E3-like factor in the SUMO pathway. Siz1 is required for SUMO attachment to the S. cerevisiae septins in vivo and strongly stimulates septin sumoylation in vitro. Siz1 and the related protein Siz2 promote SUMO conjugation to different substrates at different stages of the cell cycle and, together, are required for most SUMO conjugation in yeast. Siz1, Siz2, and the PIAS (protein inhibitor of activated STAT) proteins form a conserved family defined by an unusual RING-related motif. Our results suggest that this family functions by promoting SUMO conjugation to specific substrates.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(01)00491-3.html">http://www.cell.com/cell/full/S0092-8674(01)00491-3.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" color="#6666FF" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2000).101(7):753-62. doi:; PMID:10892746</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SUMO-1 modification of Mdm2 prevents its self-ubiquitination and increases Mdm2 ability to ubiquitinate p53.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>T Buschmann, S Y Fuchs, C G Lee, Z Q Pan, Z Ronai</b>
      </span>
      <br/>
      <span>Affiliation: Derald H Ruttenberg Cancer Center; Mount Sinai School of Medicine; New York; New York 10029; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Mdm2 is an E3 ubiquitin ligase for the p53 tumor suppressor protein. We demonstrate that Mdm2 is conjugated with SUMO-1 (sumoylated) at Lys-446, which is located within the RING finger domain and plays a critical role in Mdm2 self-ubiquitination. Whereas mutant Mdm2(K446R) is stabilized, it elicits increased degradation of p53 and concomitant inhibition of p53-mediated apoptosis. In vitro sumoylation of Mdm2 abrogates its self-ubiquitination and increases its ubiquitin ligase activity toward p53. Radiation caused a dose- and time-dependent decrease in the degree of Mdm2 SUMO-1 modification, which is inversely correlated with the levels of p53. Our results suggest that the maintenance of the intrinsic activity of a RING finger E3 ubiquitin ligase is sumoylation dependent and that reduced Mdm2 sumoylation in response to DNA damage contributes to p53 stability.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092-8674(00)80887-9.html">http://www.cell.com/cell/full/S0092-8674(00)80887-9.html</a>
    </p>
  </body>
  <hr/>
</html>
