<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Nature(2014).509(7500):376-80. doi:10.1038/Nature13084; PMID:24670657</span>
    <br/>
    <span>
      <font size="5" weight="10"/>
      <b CLASS="title">Disruption of Mediator rescues the stunted growth of a lignin-deficient Arabidopsis mutant.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Nicholas D Bonawitz, Jeong Im Kim, Yuki Tobimatsu, Peter N Ciesielski, Nickolas A Anderson, Eduardo Ximenes, Junko Maeda, John Ralph, Bryon S Donohoe, Michael Ladisch, Clint Chapple</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry; Purdue University; West Lafayette; Indiana 47907; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Lignin is a phenylpropanoid-derived heteropolymer important for the strength and rigidity of the plant secondary cell wall. Genetic disruption of lignin biosynthesis has been proposed as a means to improve forage and bioenergy crops, but frequently results in stunted growth and developmental abnormalities, the mechanisms of which are poorly understood. Here we show that the phenotype of a lignin-deficient Arabidopsis mutant is dependent on the transcriptional co-regulatory complex, Mediator. Disruption of the Mediator complex subunits MED5a (also known as REF4) and MED5b (also known as RFR1) rescues the stunted growth, lignin deficiency and widespread changes in gene expression seen in the phenylpropanoid pathway mutant ref8, without restoring the synthesis of guaiacyl and syringyl lignin subunits. Cell walls of rescued med5a/5b ref8 plants instead contain a novel lignin consisting almost exclusively of p-hydroxyphenyl lignin subunits, and moreover exhibit substantially facilitated polysaccharide saccharification. These results demonstrate that guaiacyl and syringyl lignin subunits are largely dispensable for normal growth and development, implicate Mediator in an active transcriptional process responsible for dwarfing and inhibition of lignin biosynthesis, and suggest that the transcription machinery and signalling pathways responding to cell wall defects may be important targets to include in efforts to reduce biomass recalcitrance.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v509/n7500/Nature13084.html">http://www.nature.com/journal/v509/n7500/Nature13084.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Science(2012).337(6097):960-4. doi:10.1126/Science.1221614; PMID:22923580</span>
    <br/>
    <span>
      <font size="5" weight="10"/>
      <b CLASS="title">Assembly of an evolutionarily new pathway for &#206;&#177;-pyrone biosynthesis in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jing-Ke Weng, Yi Li, Huaping Mo, Clint Chapple</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry; Purdue University; West Lafayette; IN 47907; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plants possess arrays of functionally diverse specialized metabolites, many of which are distributed taxonomically. Here, we describe the evolution of a class of substituted &#206;&#177;-pyrone metabolites in Arabidopsis, which we have named arabidopyrones. The biosynthesis of arabidopyrones requires a cytochrome P450 enzyme (CYP84A4) to generate the catechol-substituted substrate for an extradiol ring-cleavage dioxygenase (AtLigB). Unlike other ring-cleavage-derived plant metabolites made from tyrosine, arabidopyrones are instead derived from phenylalanine through the early steps of phenylpropanoid metabolism. Whereas CYP84A4, an Arabidopsis-specific paralog of the lignin-biosynthetic enzyme CYP84A1, has neofunctionalized relative to its ancestor, AtLigB homologs are widespread among land plants and many bacteria. This study exemplifies the rapid evolution of a biochemical pathway formed by the addition of a new biological activity into an existing metabolic infrastructure.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/337/6097/960.full">http://science.sciencemag.org/content/337/6097/960.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Nature(2005).434(7032):505-9. doi:10.1038/Nature03380; PMID:15785770</span>
    <br/>
    <span>
      <font size="5" weight="10"/>
      <b CLASS="title">Genome-wide non-mendelian inheritance of extra-genomic information in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Susan J Lolle, Jennifer L Victor, Jessica M Young, Robert E Pruitt</b>
      </span>
      <br/>
      <span>Affiliation: Department of Botany and Plant Pathology; Purdue University; 915 W. State Street; West Lafayette; Indiana 47907-2054; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>A fundamental tenet of classical mendelian genetics is that allelic information is stably inherited from one generation to the next, resulting in predictable segregation patterns of differing alleles. Although several exceptions to this principle are known, all represent specialized cases that are mechanistically restricted to either a limited set of specific genes (for example mating type conversion in yeast) or specific types of alleles (for example alleles containing transposons or repeated sequences). Here we show that Arabidopsis plants homozygous for recessive mutant alleles of the organ fusion gene HOTHEAD (HTH) can inherit allele-specific DNA sequence information that was not present in the chromosomal genome of their parents but was present in previous generations. This previously undescribed process is shown to occur at all DNA sequence polymorphisms examined and therefore seems to be a general mechanism for extra-genomic inheritance of DNA sequence information. We postulate that these genetic restoration events are the result of a template-directed process that makes use of an ancestral RNA-sequence cache.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v434/n7032/Nature03380.html">http://www.nature.com/journal/v434/n7032/Nature03380.html</a>
    </p>
  </body>
  <hr/>
</html>
