<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2014).343(6168):290-4. doi:10.1126/Science.343.6168.290; PMID:24436418</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification of a plant receptor for extracellular ATP.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jeongmin Choi, Kiwamu Tanaka, Yangrong Cao, Yue Qi, Jing Qiu, Yan Liang, Sang Yeol Lee, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Biochemistry and Plant Sciences; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Extracellular adenosine 5'-triphosphate (ATP) is an essential signaling molecule that is perceived in mammals by plasma membrane P2-type purinoceptors. Similar ATP receptors do not exist in plants, although extracellular ATP has been shown to play critical roles in plant growth, development, and stress responses. Here, we identify an ATP-insensitive Arabidopsis mutant, dorn1 (Does not Respond to Nucleotides 1), defective in lectin receptor kinase I.9 (Arabidopsis Information Resource accession code At5g60300). DORN1 binds ATP with high affinity (dissociation constant of 45.7 &#194;&#177; 3.1 nanomolar) and is required for ATP-induced calcium response, mitogen-activated protein kinase activation, and gene expression. Ectopic expression of DORN1 increased the plant response to physical wounding. We propose that DORN1 is essential for perception of extracellular ATP and likely plays a variety of roles in plant stress resistance.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/343/6168/290.full">http://science.sciencemag.org/content/343/6168/290.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2013).341(6152):1384-7. doi:10.1126/Science.1242736; PMID:24009356</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Nonlegumes respond to rhizobial Nod factors by suppressing the innate immune response.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yan Liang, Yangrong Cao, Kiwamu Tanaka, Sandra Thibivilliers, Jinrong Wan, Jeongmin Choi, Chang ho Kang, Jing Qiu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Science; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Virtually since the discovery of nitrogen-fixing Rhizobium-legume symbioses, researchers have dreamed of transferring this capability into nonlegume crop species (for example, corn). In general, nonlegumes were assumed to lack the ability to respond to the rhizobial lipo-chitin Nod factors, which are the essential signal molecules that trigger legume nodulation. However, our data indicate that Arabidopsis thaliana plants, as well as other nonlegumes, recognize the rhizobial Nod factor via a mechanism that results in strong suppression of microbe-associated molecular pattern (MAMP)-triggered immunity. The mechanism of action leads to reduced levels of pattern-recognition receptors on the plasma membrane involved in MAMP recognition.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/341/6152/1384.full">http://science.sciencemag.org/content/341/6152/1384.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2010).463(7278):178-83. doi:10.1038/Nature08670; PMID:20075913</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genome sequence of the palaeopolyploid soybean.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jeremy Schmutz, Steven B Cannon, Jessica Schlueter, Jianxin Ma, Therese Mitros, William Nelson, David L Hyten, Qijian Song, Jay J Thelen, Jianlin Cheng, Dong Xu, Uffe Hellsten, Gregory D May, Yeisoo Yu, Tetsuya Sakurai, Taishi Umezawa, Madan K Bhattacharyya, Devinder Sandhu, Babu Valliyodan, Erika Lindquist, Myron Peto, David Grant, Shengqiang Shu, David Goodstein, Kerrie Barry, Montona Futrell-Griggs, Brian Abernathy, Jianchang Du, Zhixi Tian, Liucun Zhu, Navdeep Gill, Trupti Joshi, Marc Libault, Anand Sethuraman, Xue-Cheng Zhang, Kazuo Shinozaki, Henry T Nguyen, Rod A Wing, Perry Cregan, James Specht, Jane Grimwood, Dan Rokhsar, Gary Stacey, Randy C Shoemaker, Scott A Jackson</b>
      </span>
      <br/>
      <span>Affiliation: HudsonAlpha Genome Sequencing Center; 601 Genome Way; Huntsville; Alabama 35806; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean (Glycine max) is one of the most important crop plants for seed protein and oil content, and for its capacity to fix atmospheric nitrogen through symbioses with soil-borne microorganisms. We sequenced the 1.1-gigabase genome by a whole-genome shotgun approach and integrated it with physical and high-density genetic maps to create a chromosome-scale draft sequence assembly. We predict 46,430 protein-coding genes, 70% more than Arabidopsis and similar to the poplar genome which, like soybean, is an ancient polyploid (palaeopolyploid). About 78% of the predicted genes occur in chromosome ends, which comprise less than one-half of the genome but account for nearly all of the genetic recombination. Genome duplications occurred at approximately 59 and 13 million years ago, resulting in a highly duplicated genome with nearly 75% of the genes present in multiple copies. The two duplication events were followed by gene diversification and loss, and numerous chromosome rearrangements. An accurate soybean genome sequence will facilitate the identification of the genetic basis of many soybean traits, and accelerate the creation of improved soybean varieties.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v463/n7278/Nature08670.html">http://www.nature.com/journal/v463/n7278/Nature08670.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2007).316(5829):1307-12. doi:10.1126/Science.1139548; PMID:17540897</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Eric Giraud, Lionel Moulin, David Vallenet, Val&#195;&#169;rie Barbe, Eddie Cytryn, Jean-Christophe Avarre, Marianne Jaubert, Damien Simon, Fabienne Cartieaux, Yves Prin, Gilles Bena, Laure Hannibal, Joel Fardoux, Mila Kojadinovic, Laurie Vuillet, Aur&#195;&#169;lie Lajus, St&#195;&#169;phane Cruveiller, Zoe Rouy, Sophie Mangenot, B&#195;&#169;atrice Segurens, Carole Dossat, William L Franck, Woo-Suk Chang, Elizabeth Saunders, David Bruce, Paul Richardson, Philippe Normand, Bernard Dreyfus, David Pignol, Gary Stacey, David Emerich, Andr&#195;&#169; Verm&#195;&#169;glio, Claudine M&#195;&#169;digue, Michael Sadowsky</b>
      </span>
      <br/>
      <span>Affiliation: Institut de Recherche pour le D&#195;&#169;veloppement; Centre de Coop&#195;&#169;ration International en Recherche Agronomique pour le D&#195;&#169;veloppement; Institut National de la Recherche Agronomique; Universit&#195;&#169; Montpellier 2; France. giraud@mpl.ird.fr</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Leguminous plants (such as peas and soybeans) and rhizobial soil bacteria are symbiotic partners that communicate through molecular signaling pathways, resulting in the formation of nodules on legume roots and occasionally stems that house nitrogen-fixing bacteria. Nodule formation has been assumed to be exclusively initiated by the binding of bacterial, host-specific lipochito-oligosaccharidic Nod factors, encoded by the nodABC genes, to kinase-like receptors of the plant. Here we show by complete genome sequencing of two symbiotic, photosynthetic, Bradyrhizobium strains, BTAi1 and ORS278, that canonical nodABC genes and typical lipochito-oligosaccharidic Nod factors are not required for symbiosis in some legumes. Mutational analyses indicated that these unique rhizobia use an alternative pathway to initiate symbioses, where a purine derivative may play a key role in triggering nodule formation.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/316/5829/1307.full">http://science.sciencemag.org/content/316/5829/1307.full</a>
    </p>
  </body>
  <hr/>
</html>
