<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Front Plant Sci(2016).7():517. doi:10.3389/Fpls.2016.00517; PMID:27200004</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Soybean Roots Grown under Heat Stress Show Global Changes in Their Transcriptional and Proteomic Profiles.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Oswaldo Vald&#195;&#169;s-L&#195;&#179;pez, Josef Batek, Nicolas Gomez-Hernandez, Cuong T Nguyen, Mariel C Isidra-Arellano, Ning Zhang, Trupti Joshi, Dong Xu, Kim K Hixson, Karl K Weitz, Joshua T Aldrich, Ljiljana Pa&#197;&#161;a-Toli&#196;&#135;, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and Biochemistry; National Center for Soybean Biotechnology; C.S. Bond Life Sciences Center; University of Missouri Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Heat stress is likely to be a key factor in the negative impact of climate change on crop production. Heat stress significantly influences the functions of roots, which provide support, water, and nutrients to other plant organs. Likewise, roots play an important role in the establishment of symbiotic associations with different microorganisms. Despite the physiological relevance of roots, few studies have examined their response to heat stress. In this study, we performed genome-wide transcriptomic and proteomic analyses on isolated root hairs, which are a single, epidermal cell type, and compared their response to stripped roots. On average, we identified 1849 and 3091 genes differentially regulated in root hairs and stripped roots, respectively, in response to heat stress. Our gene regulatory module analysis identified 10 key modules that might control the majority of the transcriptional response to heat stress. We also conducted proteomic analysis on membrane fractions isolated from root hairs and compared these responses to stripped roots. These experiments identified a variety of proteins whose expression changed within 3 h of application of heat stress. Most of these proteins were predicted to play a significant role in thermo-tolerance, as well as in chromatin remodeling and post-transcriptional regulation. The data presented represent an in-depth analysis of the heat stress response of a single cell type in soybean.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/27200004">http://www.ncbi.nlm.nih.gov/pubmed/27200004</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Mol. Biol.(2016).90(6):689-97. doi:10.1007/S11103-016-0449-8; PMID:26873699</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Differential growth responses of Brachypodium distachyon genotypes to inoculation with plant growth promoting rhizobacteria.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Fernanda P do Amaral, V&#195;&#162;nia C S Pankievicz, Ana Carolina M Arisi, Emanuel M de Souza, Fabio Pedrosa, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Science and Biochemistry; C. S. Bond Life Science Center; University of Missouri; Columbia; MO; 65211; USA. staceyg@missouri.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plant growth promoting rhizobacteria (PGPR) can associate and enhance the growth of important crop grasses. However, in most cases, the molecular mechanisms responsible for growth promotion are not known. Such research could benefit by the adoption of a grass model species that showed a positive response to bacterial inoculation and was amenable to genetic and molecular research methods. In this work we inoculated different genotypes of the model grass Brachypodium distachyon with two, well-characterized PGPR bacteria, Azospirillum brasilense and Herbaspirillum seropedicae, and evaluated the growth response. Plants were grown in soil under no nitrogen or with low nitrogen (i.e., 0.5&#194;&#160;mM KNO3). A variety of growth parameters (e.g., shoot height, root length, number of lateral roots, fresh and dry weight) were measured 35&#194;&#160;days after inoculation. The data indicate that plant genotype plays a very important role in determining the plant response to PGPR inoculation. A positive growth response was observed with only four genotypes grown under no nitrogen and three genotypes tested under low nitrogen. However, in contrast, relatively good root colonization was seen with most genotypes, as measured by drop plate counting and direct, microscopic examination of roots. In particular, the endophytic bacteria H. seropedicae showed strong epiphytic and endophytic colonization of roots.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/26873699">http://www.ncbi.nlm.nih.gov/pubmed/26873699</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Proteome Res.(2015).14(10):4137-46. doi:10.1021/Acs.Jproteome.5B00146; PMID:26350764</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification of Soybean Proteins and Genes Differentially Regulated in Near Isogenic Lines Differing in Resistance to Aphid Infestation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Laurent Brechenmacher, Tran Hong Nha Nguyen, Ning Zhang, Tae-Hwan Jun, Dong Xu, M A Rouf Mian, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: USDA-ARS; Department of Horticulture and Crop Science; The Ohio State University ; 1680 Madison Avenue; Wooster; Ohio 44691; United States.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean aphid is an important pest causing significant yield losses. The Rag2 locus confers resistance to soybean aphid biotypes 1 and 2. Transcriptomic and proteomic analyses were done over a 48 h period after aphid infestation using near isogenic lines (NILs) differing at the Rag2 locus. Comparing the Rag2 and/or rag2 lines identified 3445 proteins, of which 396 were differentially regulated between the two lines, including proteins involved in cell wall metabolism, carbohydrate metabolism, and stress response. RNA-seq transcriptomic analysis identified 2361 genes significantly regulated between the resistant and susceptible lines. Genes upregulated in the Rag2 line were annotated as being involved in cell wall, secondary, and hormone metabolism as well as in stress, signaling, and transcriptional responses. Genes downregulated in the Rag2 line were annotated as being involved in photosynthesis and carbon metabolism. Interestingly, two genes (unknown and mitochondrial protease) located within the defined Rag2 locus were expressed significantly higher in the resistant genotype. The expression of a putative NBS-LRR resistant gene within the Rag2 locus was not different between the two soybean lines, but a second NBL-LRR gene located just at the border of the defined Rag2 locus was. Therefore, this gene may be a candidate R gene controlling aphid resistance.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/26350764">http://www.ncbi.nlm.nih.gov/pubmed/26350764</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Front Plant Sci(2015).6():401. doi:10.3389/Fpls.2015.00401; PMID:26082790</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Does plant immunity play a critical role during initiation of the legume-rhizobium symbiosis?</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Katalin T&#195;&#179;th, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and Biochemistry; Christopher S. Bond Life Sciences Center; National Center for Soybean Biotechnology; University of Missouri-Columbia Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plants are exposed to many different microbes in their habitats. These microbes may be benign or pathogenic, but in some cases they are beneficial for the host. The rhizosphere provides an especially rich palette for colonization by beneficial (associative and symbiotic) microorganisms, which raises the question as to how roots can distinguish such 'friends' from possible 'foes' (i.e., pathogens). Plants possess an innate immune system that can recognize pathogens, through an arsenal of protein receptors, including receptor-like kinases (RLKs) and receptor-like proteins (RLPs) located at the plasma membrane. In addition, the plant host has intracellular receptors (so called NBS-LRR proteins or R proteins) that directly or indirectly recognize molecules released by microbes into the plant cell. A successful cooperation between legume plants and rhizobia leads to beneficial symbiotic interaction. The key rhizobial, symbiotic signaling molecules [lipo-chitooligosaccharide Nod factors (NF)] are perceived by the host legume plant using lysin motif-domain containing RLKs. Perception of the symbiotic NFs trigger signaling cascades leading to bacterial infection and accommodation of the symbiont in a newly formed root organ, the nodule, resulting in a nitrogen-fixing root nodule symbiosis. The net result of this symbiosis is the intracellular colonization of the plant with thousands of bacteria; a process that seems to occur in spite of the immune ability of plants to prevent pathogen infection. In this review, we discuss the potential of the invading rhizobial symbiont to actively avoid this innate immune response, as well as specific examples of where the plant immune response may modulate rhizobial infection and host range.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/26082790">http://www.ncbi.nlm.nih.gov/pubmed/26082790</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Planta(2015).242(5):1123-38. doi:10.1007/S00425-015-2344-Y; PMID:26067758</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Xyloglucan, galactomannan, glucuronoxylan, and rhamnogalacturonan I do not have identical structures in soybean root and root hair cell walls.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Artur Muszy&#197;&#132;ski, Malcolm A O'Neill, Easwaran Ramasamy, Sivakumar Pattathil, Utku Avci, Maria J Pe&#195;&#177;a, Marc Libault, Md Shakhawat Hossain, Laurent Brechenmacher, William S York, Rommel M Barbosa, Michael G Hahn, Gary Stacey, Russell W Carlson</b>
      </span>
      <br/>
      <span>Affiliation: Complex Carbohydrate Research Center; University of Georgia; Athens; GA; 30602; USA. mao@ccrc.uga.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chemical analyses and glycome profiling demonstrate differences in the structures of the xyloglucan, galactomannan, glucuronoxylan, and rhamnogalacturonan I isolated from soybean ( Glycine max ) roots and root hair cell walls. The root hair is a plant cell that extends only at its tip. All other root cells have the ability to grow in different directions (diffuse growth). Although both growth modes require controlled expansion of the cell wall, the types and structures of polysaccharides in the walls of diffuse and tip-growing cells from the same plant have not been determined. Soybean (Glycine max) is one of the few plants whose root hairs can be isolated in amounts sufficient for cell wall chemical characterization. Here, we describe the structural features of rhamnogalacturonan I, rhamnogalacturonan II, xyloglucan, glucomannan, and 4-O-methyl glucuronoxylan present in the cell walls of soybean root hairs and roots stripped of root hairs. Irrespective of cell type, rhamnogalacturonan II exists as a dimer that is cross-linked by a borate ester. Root hair rhamnogalacturonan I contains more neutral oligosaccharide side chains than its root counterpart. At least 90% of the glucuronic acid is 4-O-methylated in root glucuronoxylan. Only 50% of this glycose is 4-O-methylated in the root hair counterpart. Mono O-acetylated fucose-containing subunits account for at least 60% of the neutral xyloglucan from root and root hair walls. By contrast, a galacturonic acid-containing xyloglucan was detected only in root hair cell walls. Soybean homologs of the Arabidopsis xyloglucan-specific galacturonosyltransferase are highly expressed only in root hairs. A mannose-rich polysaccharide was also detected only in root hair cell walls. Our data demonstrate that the walls of tip-growing root hairs cells have structural features that distinguish them from the walls of other roots cells.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/26067758">http://www.ncbi.nlm.nih.gov/pubmed/26067758</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Exp. Bot.(2015).66(19):5727-38. doi:10.1093/Jxb/Erv260; PMID:26049159</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Effect of lipo-chitooligosaccharide on early growth of C4 grass seedlings.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Sung-Hwan Cho, Hyeyoung Lee, An Q Pham, Josef M Batek, Shiqi Cui, Jing Qiu, Saad M Khan, Trupti Joshi, Zhanyuan J Zhang, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Science and Biochemistry University of Missouri; Columbia; MO 65211; USA Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA staceyg@missouri.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Although lipo-chitooligosaccharides (LCOs) are important signal molecules for plant-symbiont interactions, a number of reports suggest that LCOs can directly impact plant growth and development, separate from any role in plant symbioses. In order to investigate this more closely, maize and Setaria seedlings were treated with LCO and their growth was evaluated. The data indicate that LCO treatment significantly enhanced root growth. RNA-seq transcriptomic analysis of LCO-treated maize roots identified a number of genes whose expression was significantly affected by the treatment. Among these genes, some LCO-up-regulated genes are likely involved in root growth promotion. Interestingly, some stress-related genes were down-regulated after LCO treatment, which might indicate reallocation of resources from defense responses to plant growth. The promoter activity of several LCO-up-regulated genes using a &#206;&#178;-glucuronidase reporter system was further analysed. The results showed that the promoters were activated by LCO treatment. The data indicate that LCO can directly impact maize root growth and gene expression.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/26049159">http://www.ncbi.nlm.nih.gov/pubmed/26049159</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Front Plant Sci(2015).6():363. doi:10.3389/Fpls.2015.00363; PMID:26042143</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">System approaches to study root hairs as a single cell plant model: current status and future perspectives.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Md Shakhawat Hossain, Trupti Joshi, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and Biochemistry; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri ; Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Our current understanding of plant functional genomics derives primarily from measurements of gene, protein and/or metabolite levels averaged over the whole plant or multicellular tissues. These approaches risk diluting the response of specific cells that might respond strongly to the treatment but whose signal is diluted by the larger proportion of non-responding cells. For example, if a gene is expressed at a low level, does this mean that it is indeed lowly expressed or is it highly expressed, but only in a few cells? In order to avoid these issues, we adopted the soybean root hair cell, derived from a single, differentiated root epidermal cell, as a single-cell model for functional genomics. Root hair cells are intrinsically interesting since they are major conduits for root water and nutrient uptake and are also the preferred site of infection by nitrogen-fixing rhizobium bacteria. Although a variety of other approaches have been used to study single plant cells or single cell types, the root hair system is perhaps unique in allowing application of the full repertoire of functional genomic and biochemical approaches. In this mini review, we summarize our published work and place this within the broader context of root biology, with a significant focus on understanding the initial events in the soybean-rhizobium interaction.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/26042143">http://www.ncbi.nlm.nih.gov/pubmed/26042143</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Appl. Environ. Microbiol.(2015).81(16):5299-308. doi:10.1128/Aem.00609-15; PMID:26025905</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">DNA Microarray-Based Identification of Genes Regulated by NtrC in Bradyrhizobium japonicum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>William L Franck, Jing Qiu, Hae-In Lee, Woo-Suk Chang, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biology; University of Texas; Arlington; Texas; USA Division of Biotechnology; College of Environmental and Bioresource Sciences; Chonbuk National University; Iksan; Republic of Korea wschang@uta.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The Bradyrhizobium japonicum NtrBC two-component system is a critical regulator of cellular nitrogen metabolism, including the acquisition and catabolism of nitrogenous compounds. To better define the roles of this system, genome-wide transcriptional profiling was performed to identify the NtrC regulon during the response to nitrogen limitation. Upon cells perceiving low intracellular nitrogen, they stimulate the phosphorylation of NtrC, which induces genes responsible for alteration of the core glutamine synthetase/glutamate synthase nitrogen assimilation pathway, including the genes for the glutamine synthetases and PII proteins. In addition, genes responsible for the import and utilization of multiple nitrogen sources, specifically nitrate and nitrite, were upregulated by NtrC activation. Mutational analysis of a candidate nitrite reductase revealed a role for NtrC in regulating the assimilation of nitrite, since mutations in both ntrC and the gene encoding the candidate nitrite reductase abolished the ability to grow on nitrite as a sole nitrogen source.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/26025905">http://www.ncbi.nlm.nih.gov/pubmed/26025905</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Biotechnol. J.(2016).14(1):332-41. doi:10.1111/Pbi.12387; PMID:25973713</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification and functional characterization of soybean root hair microRNAs expressed in response to Bradyrhizobium japonicum infection.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zhe Yan, Md Shakhawat Hossain, Oswaldo Vald&#195;&#169;s-L&#195;&#179;pez, Nhung T Hoang, Jixian Zhai, Jun Wang, Marc Libault, Laurent Brechenmacher, Seth Findley, Trupti Joshi, Lijuan Qiu, D Janine Sherrier, Tieming Ji, Blake C Meyers, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Science and Biochemistry; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Three soybean [Glycine max (L) Merr.] small RNA libraries were generated and sequenced using the Illumina platform to examine the role of miRNAs during soybean nodulation. The small RNA libraries were derived from root hairs inoculated with Bradyrhizobium japonicum (In_RH) or mock-inoculated with water (Un_RH), as well as from the comparable inoculated stripped root samples (i.e. inoculated roots with the root hairs removed). Sequencing of these libraries identified a total of 114 miRNAs, including 22 novel miRNAs. A comparison of miRNA abundance among the 114 miRNAs identified 66 miRNAs that were differentially expressed between root hairs and stripped roots, and 48 miRNAs that were differentially regulated in infected root hairs in response to B.&#194;&#160;japonicum when compared to uninfected root hairs (P&#194;&#160;&#226;&#137;&#164;&#194;&#160;0.05). A parallel analysis of RNA ends (PARE) library was constructed and sequenced to reveal a total of 405 soybean miRNA targets, with most predicted to encode transcription factors or proteins involved in protein modification, protein degradation and hormone pathways. The roles of gma-miR4416 and gma-miR2606b during nodulation were further analysed. Ectopic expression of these two miRNAs in soybean roots resulted in significant changes in nodule numbers. miRNA target information suggested that gma-miR2606b regulates a Mannosyl-oligosaccharide 1, 2-alpha-mannosidase gene, while gma-miR4416 regulates the expression of a rhizobium-induced peroxidase 1 (RIP1)-like peroxidase gene, GmRIP1, during nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25973713">http://www.ncbi.nlm.nih.gov/pubmed/25973713</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>New Phytol.(2015).207(3):748-59. doi:10.1111/Nph.13365; PMID:25783944</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification of microRNAs and their mRNA targets during soybean nodule development: functional analysis of the role of miR393j-3p in soybean nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zhe Yan, Md Shakhawat Hossain, Siwaret Arikit, Oswaldo Vald&#195;&#169;s-L&#195;&#179;pez, Jixian Zhai, Jun Wang, Marc Libault, Tieming Ji, Lijuan Qiu, Blake C Meyers, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Science and Biochemistry; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO; 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plant microRNAs (miRNAs) play important regulatory roles in a number of developmental processes. The present work investigated the roles of miRNAs during nodule development in the crop legume soybean (Glycine max). Fifteen soybean small RNA libraries were sequenced from different stages of nodule development, including young nodules, mature nodules and senescent nodules. In order to identify the regulatory targets of the miRNAs, five parallel analysis of RNA ends (PARE) libraries were also sequenced from the same stages of nodule development. Sequencing identified 284 miRNAs, including 178 novel soybean miRNAs. Analysis of miRNA abundance identified 139 miRNAs whose expression was significantly regulated during nodule development, including 12 miRNAs whose expression changed &gt; 10-fold. Analysis of the PARE libraries identified 533 miRNA targets, including three nodulation-related genes and eight nodule-specific genes. miR393j-3p was selected for detailed analysis as its expression was significantly regulated during nodule formation, and it targeted a nodulin gene, Early Nodulin 93 (ENOD93). Strong, ectopic expression of miR393j-3p, as well as RNAi silencing of ENOD93 expression, significantly reduced nodule formation. The data indicate that miR393j-3p regulation of ENOD93 mRNA abundance is a key control point for soybean nodule formation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25783944">http://www.ncbi.nlm.nih.gov/pubmed/25783944</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant J.(2015).81(6):907-19. doi:10.1111/Tpj.12777; PMID:25645593</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Robust biological nitrogen fixation in a model grass-bacterial association.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>V&#195;&#162;nia C S Pankievicz, Fernanda P do Amaral, Karina F D N Santos, Beverly Agtuca, Youwen Xu, Michael J Schueller, Ana Carolina M Arisi, Maria B R Steffens, Emanuel M de Souza, F&#195;&#161;bio O Pedrosa, Gary Stacey, Richard A Ferrieri</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry and Molecular Biology; Federal University of Paran&#195;&#161;; 81531-980; Curitiba; Brazil.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Nitrogen-fixing rhizobacteria can promote plant growth; however, it is controversial whether biological nitrogen fixation (BNF) from associative interaction contributes to growth promotion. The roots of Setaria viridis, a model C4 grass, were effectively colonized by bacterial inoculants resulting in a significant enhancement of growth. Nitrogen-13 tracer studies provided direct evidence for tracer uptake by the host plant and incorporation into protein. Indeed, plants showed robust growth under nitrogen-limiting conditions when inoculated with an ammonium-excreting strain of Azospirillum brasilense. (11)C-labeling experiments showed that patterns in central carbon metabolism and resource allocation exhibited by nitrogen-starved plants were largely reversed by bacterial inoculation, such that they resembled plants grown under nitrogen-sufficient conditions. Adoption of S. viridis as a model should promote research into the mechanisms of associative nitrogen fixation with the ultimate goal of greater adoption of BNF for sustainable crop production.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25645593">http://www.ncbi.nlm.nih.gov/pubmed/25645593</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Trends Plant Sci.(2015).20(3):186-94. doi:10.1016/J.Tplants.2014.11.008; PMID:25543258</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Rhizobium-legume symbioses: the crucial role of plant immunity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Benjamin Gourion, Fathi Berrabah, Pascal Ratet, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Science and Biochemistry; National Center for Soybean Biotechnology; C.S. Bond Life Science Center; University of Missouri; Columbia; MO 65203; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>New research results have significantly revised our understanding of the rhizobium-legume infection process. For example, Nod factors (NFs), previously thought to be absolutely essential for this symbiosis, were shown to be dispensable under particular conditions. Similarly, an NF receptor, previously considered to be solely involved in symbiosis, was shown to function during plant pathogen infections. Indeed, there is a growing realization that plant innate immunity is a crucial component in the establishment and maintenance of symbiosis. We review here the factors involved in the suppression of plant immunity during rhizobium-legume symbiosis, and we attempt to place this information into context with the most recent and sometimes surprising research results.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25543258">http://www.ncbi.nlm.nih.gov/pubmed/25543258</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Cell(2014).26(12):4584-601. doi:10.1105/Tpc.114.131847; PMID:25465409</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">An atlas of soybean small RNAs identifies phased siRNAs from hundreds of coding genes.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Siwaret Arikit, Rui Xia, Atul Kakrana, Kun Huang, Jixian Zhai, Zhe Yan, Oswaldo Vald&#195;&#169;s-L&#195;&#179;pez, Silvas Prince, Theresa A Musket, Henry T Nguyen, Gary Stacey, Blake C Meyers</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant and Soil Sciences; University of Delaware; Newark; Delaware 19711 Delaware Biotechnology Institute; University of Delaware; Newark; Delaware 19711 meyers@dbi.udel.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Small RNAs are ubiquitous, versatile repressors and include (1) microRNAs (miRNAs), processed from mRNA forming stem-loops; and (2) small interfering RNAs (siRNAs), the latter derived in plants by a process typically requiring an RNA-dependent RNA polymerase. We constructed and analyzed an expression atlas of soybean (Glycine max) small RNAs, identifying over 500 loci generating 21-nucleotide phased siRNAs (phasiRNAs; from PHAS loci), of which 483 overlapped annotated protein-coding genes. Via the integration of miRNAs with parallel analysis of RNA end (PARE) data, 20 miRNA triggers of 127 PHAS loci were detected. The primary class of PHAS loci (208 or 41% of the total) corresponded to NB-LRR genes; some of these small RNAs preferentially accumulate in nodules. Among the PHAS loci, novel representatives of TAS3 and noncanonical phasing patterns were also observed. A noncoding PHAS locus, triggered by miR4392, accumulated preferentially in anthers; the phasiRNAs are predicted to target transposable elements, with their peak abundance during soybean reproductive development. Thus, phasiRNAs show tremendous diversity in dicots. We identified novel miRNAs and assessed the veracity of soybean miRNAs registered in miRBase, substantially improving the soybean miRNA annotation, facilitating an improvement of miRBase annotations and identifying at high stringency novel miRNAs and their targets.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25465409">http://www.ncbi.nlm.nih.gov/pubmed/25465409</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>New Phytol.(2014).204(2):289-96. doi:; PMID:25453133</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Lipochitooligosaccharide recognition: an ancient story.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yan Liang, Katalin T&#195;&#179;th, Yangrong Cao, Kiwamu Tanaka, Catherine Espinoza, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Science and Biochemistry; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin is the second most abundant polysaccharide in nature, found in crustacean shells, insect exoskeletons and fungal cell walls. The action of chitin and chitin derivatives on plants has become a very interesting story of late. Chitin is a b1-4-linked polymer of N-acetyl-Dglucosamine(GlcNAc). In this unmodified form, chitooligosaccharides (degree of polymerization(dp) = 6&#226;&#128;&#147;8)) are strong inducers of plant innate immunity. By contrast, when these chitooligosaccharides are acylated (so-called lipochitooligosaccharides, LCOs) and further modified, they can act as Nod factors, the key signaling molecules that play an important role in the initiation of the legume&#226;&#128;&#147;rhizobium symbiosis. In a similar form, these molecules can also act as Myc factors, the key signaling molecules involved in the arbuscular mycorrhizal (AM)symbiosis. It has been proposed that Nod factor perception might have evolved from the more ancient AM symbiosis. Increasing evidence now suggests that LCO perception might have evolved from plant innate immunity signaling. In this review, we will discuss the evolutionary origin of symbiotic LCO recognition.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25453133">http://www.ncbi.nlm.nih.gov/pubmed/25453133</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Elife(2014).3():. doi:10.7554/Elife.03766; PMID:25340959</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The kinase LYK5 is a major chitin receptor in Arabidopsis and forms a chitin-induced complex with related kinase CERK1.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yangrong Cao, Yan Liang, Kiwamu Tanaka, Cuong T Nguyen, Robert P Jedrzejczak, Andrzej Joachimiak, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; University of Missouri; Columbia; United States.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin is a fungal microbe-associated molecular pattern recognized in Arabidopsis by a lysin motif receptor kinase (LYK), AtCERK1. Previous research suggested that AtCERK1 is the major chitin receptor and mediates chitin-induced signaling through homodimerization and phosphorylation. However, the reported chitin binding affinity of AtCERK1 is quite low, suggesting another receptor with high chitin binding affinity might be present. Here, we propose that AtLYK5 is the primary chitin receptor in Arabidopsis. Mutations in AtLYK5 resulted in a significant reduction in chitin response. However, AtLYK5 shares overlapping function with AtLYK4 and, therefore, Atlyk4/Atlyk5-2 double mutants show a complete loss of chitin response. AtLYK5 interacts with AtCERK1 in a chitin-dependent manner. Chitin binding to AtLYK5 is indispensable for chitin-induced AtCERK1 phosphorylation. AtLYK5 binds chitin at a much higher affinity than AtCERK1. The data suggest that AtLYK5 is the primary receptor for chitin, forming a chitin inducible complex with AtCERK1 to induce plant immunity.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25340959">http://www.ncbi.nlm.nih.gov/pubmed/25340959</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Biochem. J.(2014).463(3):429-37. doi:10.1042/Bj20140666; PMID:25301072</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Extracellular ATP, a danger signal, is recognized by DORN1&#194;&#160;in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jeongmin Choi, Kiwamu Tanaka, Yan Liang, Yangrong Cao, Sang Yeol Lee, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: *Divisions of Biochemistry and Plant Sciences; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; U.S.A.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>ATP, the universal energy currency of all organisms, is released into the extracellular matrix and serves as a signal among cells, where it is referred to as an extracellular ATP. Although a signalling role for extracellular ATP has been well studied in mammals over the last 40&#194;&#160;years, investigations of such a role in plants are at an early stage. Recently, the first plant receptor for extracellular ATP, DOes not Respond to Nucleotides (DORN1), was identified in Arabidopsis thaliana by mutant screening. DORN1 encodes a legume-type lectin receptor kinase that is structurally distinct from the mammalian extracellular ATP receptors. In the present review, we highlight the genetic and biochemical evidence for the role of DORN1&#194;&#160;in extracellular ATP signalling, placing this within the wider context of extracellular ATP signalling during plant stress responses.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25301072">http://www.ncbi.nlm.nih.gov/pubmed/25301072</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Front Plant Sci(2014).5():446. doi:10.3389/Fpls.2014.00446; PMID:25232361</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Extracellular ATP acts as a damage-associated molecular pattern (DAMP) signal in plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Jeongmin Choi, Yangrong Cao, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and Biochemistry; Christopher S. Bond Life Sciences Center; University of Missouri Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>As sessile organisms, plants have evolved effective mechanisms to protect themselves from environmental stresses. Damaged (i.e., wounded) plants recognize a variety of endogenous molecules as danger signals, referred to as damage-associated molecular patterns (DAMPs). ATP is among the molecules that are released by cell damage, and recent evidence suggests that ATP can serve as a DAMP. Although little studied in plants, extracellular ATP is well known for its signaling roles in animals, including acting as a DAMP during the inflammatory response and wound healing. If ATP acts outside the cell, then it is reasonable to expect that it is recognized by a plasma membrane-localized receptor. Recently, DORN1, a lectin receptor kinase, was shown to recognize extracellular ATP in Arabidopsis. DORN1 is the founding member of a new purinoceptor subfamily, P2K (P2 receptor kinase), which is plant-specific. P2K1 (DORN1) is required for ATP-induced cellular responses (e.g., cytosolic Ca(2+) elevation, MAPK phosphorylation, and gene expression). Genetic analysis of loss-of-function mutants and overexpression lines showed that P2K1 participates in the plant wound response, consistent with the role of ATP as a DAMP. In this review, we summarize past research on the roles and mechanisms of extracellular ATP signaling in plants, and discuss the direction of future research on extracellular ATP as a DAMP signal.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25232361">http://www.ncbi.nlm.nih.gov/pubmed/25232361</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Colloids Surf B Biointerfaces(2014).122():241-9. doi:10.1016/J.Colsurfb.2014.06.067; PMID:25051306</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Interfacing whispering gallery mode optical microresonator biosensors with the plant defense elicitor chitin.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jeremy L Dahmen, Yongqiang Yang, C Michael Greenlief, Gary Stacey, Heather K Hunt</b>
      </span>
      <br/>
      <span>Affiliation: Department of Bioengineering; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; United States. Electronic address: hunthk@missouri.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The biomaterial class of chitooligosaccharides (chitin), commonly found in insects and fungi, is one of the most abundant on earth. Substantial evidence implicates chitin in mediating a diverse array of plant cellular signaling events, including the induction of plant defense mechanisms against invading pests. However, these recognition and mediation mechanisms, including the binding kinetics between chitin and their plant recognition receptors, are not fully understood. Therefore, the creation of a platform capable of both interfacing with chitin and plant cell receptors, and monitoring their interactions, would significantly advance our understanding of this plant defense elicitor. Recently, a label-free, highly sensitive biosensor platform, based on Whispering Gallery Mode optical microresonators, has been developed to study such biomolecular interactions. Here, we demonstrate how this unique platform can be interfaced with chitin using simple carbohydrate chemistry. The surface chemistry is demonstrated using X-ray photoelectron spectroscopy, fluorescence microscopy, optical profilometry, ellipsometry, and contact angle measurements. The resulting surface is uniform, with an average surface roughness of 1.25nm, and is active toward chitin recognition elements. Optical loss measurements using standard quantitative cavity analysis techniques demonstrate that the bioconjugated platforms maintain the high performance (Q&gt;10(6)) required to track binding interactions in this system. The platform is able to detect lectin, which binds COs, at 10&#206;&#188;g/mL concentration. This biosensor platform's unique capabilities for label-free, high sensitivity biodetection, when properly interfaced with the biomaterials of interest, could provide the basis for a robust analytical technique to probe the binding dynamics of chitin-plant cell receptors.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25051306">http://www.ncbi.nlm.nih.gov/pubmed/25051306</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>PLoS ONE(2014).9(7):e102245. doi:10.1371/Journal.Pone.0102245; PMID:25036661</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">LIK1, a CERK1-interacting kinase, regulates plant immune responses in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Mi Ha Le, Yangrong Cao, Xue-Cheng Zhang, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Sciences and Biochemistry; National Center for Soybean Biotechnology; C.S. Bond Life Sciences Center; University of Missouri; Columbia; Missouri; United States of America.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin, an integral component of the fungal cell wall, is one of the best-studied microbe-associated molecular patterns. Previous work identified a LysM receptor-like kinase (LysM-RLK1/CERK1) as the primary chitin receptor in Arabidopsis. In order to identify proteins that interact with CERK1, we conducted a yeast two-hybrid screen using the intracellular kinase domain of CERK1 as the bait. This screen identified 54 putative CERK1-interactors. Screening mutants defective in 43 of these interacting proteins identified only two, a calmodulin like protein (At3g10190) and a leucine-rich repeat receptor like kinase (At3g14840), which differed in their response to pathogen challenge. In the present work, we focused on characterizing the LRR-RLK gene where mutations altered responses to chitin elicitation. This LRR-RLK was named LysM RLK1-interacting kinase 1 (LIK1). The interaction between CERK1 and LIK1 was confirmed by co-immunoprecipitation using protoplasts and transgenic plants. In vitro experiments showed that LIK1 was directly phosphorylated by CERK1. In vivo phosphorylation assays showed that Col-0 wild-type plants have more phosphorylated LIK1 than cerk1 mutant plants, suggesting that LIK1 may be directly phosphorylated by CERK1. Lik1 mutant plants showed an enhanced response to both chitin and flagellin elicitors. In comparison to the wild-type plants, lik1 mutant plants were more resistant to the hemibiotrophic pathogen Pseudomonas syringae, but more susceptible to the necrotrophic pathogen Sclerotinia sclerotiorum. Consistent with the enhanced susceptibility to necrotrophs, lik1 mutants showed reduced expression of genes involved in jasmonic acid and ethylene signaling pathways. These data suggest that LIK1 directly interacts with CERK1 and regulates MAMP-triggered innate immunity.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25036661">http://www.ncbi.nlm.nih.gov/pubmed/25036661</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Plant Biol.(2014).14():143. doi:10.1186/1471-2229-14-143; PMID:24886084</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Deletions of the SACPD-C locus elevate seed stearic acid levels but also result in fatty acid and morphological alterations in nitrogen fixing nodules.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jason D Gillman, Minviluz G Stacey, Yaya Cui, Howard R Berg, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: USDA-ARS; University of Missouri-Columbia; 205 Curtis Hall; Columbia MO 65211; USA. Jason.Gillman@ars.usda.gov.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean (Glycine max) seeds are the primary source of edible oil in the United States. Despite its widespread utility, soybean oil is oxidatively unstable. Until recently, the majority of soybean oil underwent chemical hydrogenation, a process which also generates trans fats. An alternative to chemical hydrogenation is genetic modification of seed oil through identification and introgression of mutant alleles. One target for improvement is the elevation of a saturated fat with no negative cardiovascular impacts, stearic acid, which typically constitutes a minute portion of seed oil (~3%).We examined radiation induced soybean mutants with moderately increased stearic acid (10-15% of seed oil, ~3-5 X the levels in wild-type soybean seeds) via comparative whole genome hybridization and genetic analysis. The deletion of one SACPD isoform encoding gene (SACPD-C) was perfectly correlated with moderate elevation of seed stearic acid content. However, SACPD-C deletion lines were also found to have altered nodule fatty acid composition and grossly altered morphology. Despite these defects, overall nodule accumulation and nitrogen fixation were unaffected, at least under laboratory conditions.Although no yield penalty has been reported for moderate elevated seed stearic acid content in soybean seeds, our results demonstrate that genetic alteration of seed traits can have unforeseen pleiotropic consequences. We have identified a role for fatty acid biosynthesis, and SACPD activity in particular, in the establishment and maintenance of symbiotic nitrogen fixation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24886084">http://www.ncbi.nlm.nih.gov/pubmed/24886084</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol Plant(2014).7(9):1455-69. doi:10.1093/Mp/Ssu067; PMID:24880337</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">OPT3 is a component of the iron-signaling network between leaves and roots and misregulation of OPT3 leads to an over-accumulation of cadmium in seeds.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>David G Mendoza-C&#195;&#179;zatl, Qingqing Xie, Garo Z Akmakjian, Timothy O Jobe, Ami Patel, Minviluz G Stacey, Lihui Song, Dustin Wayne Demoin, Silvia S Jurisson, Gary Stacey, Julian I Schroeder</b>
      </span>
      <br/>
      <span>Affiliation: Division of Biological Sciences; Cell and Developmental Biology Section and Center for Food and Fuel for the 21st Century; University of California; San Diego; La Jolla; CA 92093; USA jischroeder@ucsd.edu mendozacozatld@missouri.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plants and seeds are the main dietary sources of zinc, iron, manganese, and copper, but are also the main entry point for toxic elements such as cadmium into the food chain. We report here that an Arabidopsis oligopeptide transporter mutant, opt3-2, over-accumulates cadmium (Cd) in seeds and roots but, unexpectedly, under-accumulates Cd in leaves. The cadmium distribution in opt3-2 differs from iron, zinc, and manganese, suggesting a metal-specific mechanism for metal partitioning within the plant. The opt3-2 mutant constitutively up-regulates the Fe/Zn/Cd transporter IRT1 and FRO2 in roots, indicative of an iron-deficiency response. No genetic mutants that impair the shoot-to-root signaling of iron status in leaves have been identified. Interestingly, shoot-specific expression of OPT3 rescues the Cd sensitivity and complements the aberrant expression of IRT1 in opt3-2 roots, suggesting that OPT3 is required to relay the iron status from leaves to roots. OPT3 expression was found in the vasculature with preferential expression in the phloem at the plasma membrane. Using radioisotope experiments, we found that mobilization of Fe from leaves is severely affected in opt3-2, suggesting that Fe mobilization out of leaves is required for proper trace-metal homeostasis. When expressed in yeast, OPT3 does not localize to the plasma membrane, precluding the identification of the OPT3 substrate. Our in planta results show that OPT3 is important for leaf phloem-loading of iron and plays a key role regulating Fe, Zn, and Cd distribution within the plant. Furthermore, ferric chelate reductase activity analyses provide evidence that iron is not the sole signal transferred from leaves to roots in leaf iron status signaling.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24880337">http://www.ncbi.nlm.nih.gov/pubmed/24880337</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Curr. Opin. Plant Biol.(2014).20():82-7. doi:10.1016/J.Pbi.2014.04.009; PMID:24865948</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Extracellular ATP is a central signaling molecule in plant stress responses.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yangrong Cao, Kiwamu Tanaka, Cuong T Nguyen, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Sciences and Biochemistry; National Center for Soybean Biotechnology; C.S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA. Electronic address: staceyg@missouri.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Because of their sessile nature, plants have developed a number of sophisticated signaling systems to adapt to environmental changes. Previous research has shown that extracellular ATP is an important signaling molecule used by plants and functions in a variety of processes, including growth, development, and stress responses. Recently, DORN1 was identified as the first plant purinoceptor, essential for the plant response to ATP. The identification of the receptor is a milestone for our overall understanding of various physiological events regulated by extracellular ATP. In this review, we will discuss the possible roles of DORN1 providing future direction for research into the role of extracellular ATP in plants.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24865948">http://www.ncbi.nlm.nih.gov/pubmed/24865948</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Agric. Food Chem.(2014).62(16):3517-24. doi:10.1021/Jf5000612; PMID:24716625</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Soybean ureases, but not that of Bradyrhizobium japonicum, are involved in the process of soybean root nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>M&#195;&#180;nica Medeiros-Silva, William L Franck, Marcela P Borba, Sabrina B Pizzato, Kent N Strodtman, David W Emerich, Gary Stacey, Joseph C Polacco, C&#195;&#169;lia R Carlini</b>
      </span>
      <br/>
      <span>Affiliation: Graduate Program in Cellular and Molecular Biology; Center of Biotechnology and #Department of Biophysics and Center of Biotechnology; Universidade Federal do Rio Grande do Sul ; Porto Alegre; Rio Grande do Sul CEP 91501-970; Brazil.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Ureases are abundant in plants, bacteria, and in the soil, but their role in signaling between soybean and soil microorganisms has not been investigated. The bacterium Bradyrhizobium japonicum forms nitrogen-fixing nodules on soybean roots. Here, we evaluated the role(s) of ureases in the process of soybean nodulation. Chemotaxis assays demonstrated that soybean and jack bean ureases were more chemotactic toward bacterial cells than the corresponding plant lectins. The eu1-a,eu4 soybean, deficient in urease isoforms, formed fewer but larger nodules than the wild-type, regardless of the bacterial urease phenotype. Leghemoglobin production in wild-type plants was higher and peaked earlier than in urease-deficient plants. Inhibition of urease activity in wild-type plants did not result in the alterations seen in mutated plants. We conclude that soybean urease(s) play(s) a role in the soybean-B. japonicum symbiosis, which is independent of its ureolytic activity. Bacterial urease does not play a role in nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24716625">http://www.ncbi.nlm.nih.gov/pubmed/24716625</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Science(2014).343(6168):290-4. doi:10.1126/Science.343.6168.290; PMID:24436418</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification of a plant receptor for extracellular ATP.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jeongmin Choi, Kiwamu Tanaka, Yangrong Cao, Yue Qi, Jing Qiu, Yan Liang, Sang Yeol Lee, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Biochemistry and Plant Sciences; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Extracellular adenosine 5'-triphosphate (ATP) is an essential signaling molecule that is perceived in mammals by plasma membrane P2-type purinoceptors. Similar ATP receptors do not exist in plants, although extracellular ATP has been shown to play critical roles in plant growth, development, and stress responses. Here, we identify an ATP-insensitive Arabidopsis mutant, dorn1 (Does not Respond to Nucleotides 1), defective in lectin receptor kinase I.9 (Arabidopsis Information Resource accession code At5g60300). DORN1 binds ATP with high affinity (dissociation constant of 45.7 &#194;&#177; 3.1 nanomolar) and is required for ATP-induced calcium response, mitogen-activated protein kinase activation, and gene expression. Ectopic expression of DORN1 increased the plant response to physical wounding. We propose that DORN1 is essential for perception of extracellular ATP and likely plays a variety of roles in plant stress resistance.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/343/6168/290.full">http://science.sciencemag.org/content/343/6168/290.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2014).27(5):415-23. doi:10.1094/Mpmi-09-13-0269-R; PMID:24400939</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A soybean acyl carrier protein, GmACP, is important for root nodule symbiosis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jun Wang, Katalin T&#195;&#179;th, Kiwamu Tanaka, Cuong T Nguyen, Zhe Yan, Laurent Brechenmacher, Jeremy Dahmen, Mingjie Chen, Jay J Thelen, Lijuan Qiu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: </span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Legumes (members of family Fabaceae) establish a symbiotic relationship with nitrogen-fixing soil bacteria (rhizobia) to overcome nitrogen source limitation. Single root hair epidermal cells serve as the entry point for bacteria to infect the host root, leading to development of a new organ, the nodule, which the bacteria colonize. In the present study, the putative role of a soybean acyl carrier protein (ACP), GmACP (Glyma18g47950), was examined in nodulation. ACP represent an essential cofactor protein in fatty acid biosynthesis. Phylogenetic analysis of plant ACP protein sequences showed that GmACP was classified in a legume-specific clade. Quantitative reverse-transcription polymerase chain reaction analysis demonstrated that GmACP was expressed in all soybean tissues but showed higher transcript accumulation in nodule tissue. RNA interference-mediated gene silencing of GmACP resulted in a significant reduction in nodule numbers on soybean transgenic roots. Fluorescent protein-labeled GmACP was localized to plastids in planta, the site of de novo fatty acid biosynthesis in plants. Analysis of the fatty acid content of root tissue silenced for GmACP expression, as determined by gas chromatography-mass spectrometry, showed an approximately 22% reduction, specifically in palmitic and stearic acid. Taken together, our data provide evidence that GmACP plays an important role in nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24400939">http://www.ncbi.nlm.nih.gov/pubmed/24400939</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Nucleic Acids Res.(2014).42(Database issue):D1245-52. doi:10.1093/Nar/Gkt905; PMID:24136998</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Soybean knowledge base (SoyKB): a web resource for integration of soybean translational genomics and molecular breeding.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Trupti Joshi, Michael R Fitzpatrick, Shiyuan Chen, Yang Liu, Hongxin Zhang, Ryan Z Endacott, Eric C Gaudiello, Gary Stacey, Henry T Nguyen, Dong Xu</b>
      </span>
      <br/>
      <span>Affiliation: Department of Computer Science; University of Missouri; Columbia; MO 65211; USA; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA; National Center for Soybean Biotechnology; University of Missouri; Columbia; MO 65211; USA; Informatics Institute; University of Missouri; Columbia; MO 65211; USA and Division of Plant Sciences; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean Knowledge Base (http://soykb.org) is a comprehensive web resource developed for bridging soybean translational genomics and molecular breeding research. It provides information for six entities including genes/proteins, microRNAs/sRNAs, metabolites, single nucleotide polymorphisms, plant introduction lines and traits. It also incorporates many multi-omics datasets including transcriptomics, proteomics, metabolomics and molecular breeding data, such as quantitative trait loci, traits and germplasm information. Soybean Knowledge Base has a new suite of tools such as In Silico Breeding Program for soybean breeding, which includes a graphical chromosome visualizer for ease of navigation. It integrates quantitative trait loci, traits and germplasm information along with genomic variation data, such as single nucleotide polymorphisms, insertions, deletions and genome-wide association studies data, from multiple soybean cultivars and Glycine soja.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24136998">http://www.ncbi.nlm.nih.gov/pubmed/24136998</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Bioinformatics(2013).14():278. doi:10.1186/1471-2105-14-278; PMID:24053776</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Predicting gene regulatory networks of soybean nodulation from RNA-Seq transcriptome data.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Mingzhu Zhu, Jeremy L Dahmen, Gary Stacey, Jianlin Cheng</b>
      </span>
      <br/>
      <span>Affiliation: Department of Computer Science; University of Missouri; Columbia; MO 65211; USA. chengji@missouri.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>High-throughput RNA sequencing (RNA-Seq) is a revolutionary technique to study the transcriptome of a cell under various conditions at a systems level. Despite the wide application of RNA-Seq techniques to generate experimental data in the last few years, few computational methods are available to analyze this huge amount of transcription data. The computational methods for constructing gene regulatory networks from RNA-Seq expression data of hundreds or even thousands of genes are particularly lacking and urgently needed.We developed an automated bioinformatics method to predict gene regulatory networks from the quantitative expression values of differentially expressed genes based on RNA-Seq transcriptome data of a cell in different stages and conditions, integrating transcriptional, genomic and gene function data. We applied the method to the RNA-Seq transcriptome data generated for soybean root hair cells in three different development stages of nodulation after rhizobium infection. The method predicted a soybean nodulation-related gene regulatory network consisting of 10 regulatory modules common for all three stages, and 24, 49 and 70 modules separately for the first, second and third stage, each containing both a group of co-expressed genes and several transcription factors collaboratively controlling their expression under different conditions. 8 of 10 common regulatory modules were validated by at least two kinds of validations, such as independent DNA binding motif analysis, gene function enrichment test, and previous experimental data in the literature.We developed a computational method to reliably reconstruct gene regulatory networks from RNA-Seq transcriptome data. The method can generate valuable hypotheses for interpreting biological data and designing biological experiments such as ChIP-Seq, RNA interference, and yeast two hybrid experiments.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24053776">http://www.ncbi.nlm.nih.gov/pubmed/24053776</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Science(2013).341(6152):1384-7. doi:10.1126/Science.1242736; PMID:24009356</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Nonlegumes respond to rhizobial Nod factors by suppressing the innate immune response.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yan Liang, Yangrong Cao, Kiwamu Tanaka, Sandra Thibivilliers, Jinrong Wan, Jeongmin Choi, Chang ho Kang, Jing Qiu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Science; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Virtually since the discovery of nitrogen-fixing Rhizobium-legume symbioses, researchers have dreamed of transferring this capability into nonlegume crop species (for example, corn). In general, nonlegumes were assumed to lack the ability to respond to the rhizobial lipo-chitin Nod factors, which are the essential signal molecules that trigger legume nodulation. However, our data indicate that Arabidopsis thaliana plants, as well as other nonlegumes, recognize the rhizobial Nod factor via a mechanism that results in strong suppression of microbe-associated molecular pattern (MAMP)-triggered immunity. The mechanism of action leads to reduced levels of pattern-recognition receptors on the plasma membrane involved in MAMP recognition.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/341/6152/1384.full">http://science.sciencemag.org/content/341/6152/1384.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2013).26(12):1371-7. doi:10.1094/Mpmi-04-13-0111-R; PMID:23980625</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">miR172 regulates soybean nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zhe Yan, Md Shakhawat Hossain, Jun Wang, Oswaldo Vald&#195;&#169;s-L&#195;&#179;pez, Yan Liang, Marc Libault, Lijuan Qiu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: </span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Micro-RNAs (miRNAs) play a pivotal role in the control of gene expression and regulate plant developmental processes. miRNA 172 (miR172) is a conserved miRNA in plants reported to control the expression of genes involved in developmental phase transition, floral organ identity, and flowering time. However, the specific role of miR172 in legume nodulation is undefined. Ectopic expression of soybean miR172 resulted in an increase in nodule numbers in transgenic roots and an increase in the expression of both symbiotic leghemoglobin and nonsymbiotic hemoglobin. These nodules showed higher levels of nitrogenase activity. Further analysis revealed a complex regulatory circuit in which miR156 regulates miR172 expression and controls the level of an AP2 transcription factor. The latter, either directly or indirectly, controls the expression of nonsymbiotic hemoglobin, which is essential for regulating the levels of nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23980625">http://www.ncbi.nlm.nih.gov/pubmed/23980625</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Methods Mol. Biol.(2013).1043():45-54. doi:10.1007/978-1-62703-532-3_5; PMID:23913034</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Aequorin luminescence-based functional calcium assay for heterotrimeric G-proteins in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Jeongmin Choi, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Science and Biochemistry; University of Missouri; Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Heterotrimeric GTP-binding proteins (G-proteins) and G-protein-coupled receptors are important signaling components in eukaryotes. In plants, the G-proteins are involved in diverse physiological processes, some of which are exerted via changes in the level of cytosolic free calcium concentration ([Ca(2+)]cyt). Various techniques have been developed to measure the change of [Ca(2+)]cyt, e.g., calcium-sensitive microelectrodes, chemical fluorescent dyes, and biosensors based on luminescent or fluorescent indicators. In this chapter, we describe a protocol for in vivo [Ca(2+)]cyt measurement in G-protein mutants expressing aequorin, a luminescent-based calcium biosensor, to extend our knowledge about G-protein mediated Ca(2+) signaling. This method is also applicable to other early signaling events that are mediated by changes in [Ca(2+)]cyt levels.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23913034">http://www.ncbi.nlm.nih.gov/pubmed/23913034</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Genome Res.(2013).23(10):1663-74. doi:10.1101/Gr.152538.112; PMID:23739894</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Robert J Schmitz, Yupeng He, Oswaldo Vald&#195;&#169;s-L&#195;&#179;pez, Saad M Khan, Trupti Joshi, Mark A Urich, Joseph R Nery, Brian Diers, Dong Xu, Gary Stacey, Joseph R Ecker</b>
      </span>
      <br/>
      <span>Affiliation: Plant Biology Laboratory; The Salk Institute for Biological Studies; La Jolla; California 92037; USA;</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Cytosine DNA methylation is one avenue for passing information through cell divisions. Here, we present epigenomic analyses of soybean recombinant inbred lines (RILs) and their parents. Identification of differentially methylated regions (DMRs) revealed that DMRs mostly cosegregated with the genotype from which they were derived, but examples of the uncoupling of genotype and epigenotype were identified. Linkage mapping of methylation states assessed from whole-genome bisulfite sequencing of 83 RILs uncovered widespread evidence for local methylQTL. This epigenomics approach provides a comprehensive study of the patterns and heritability of methylation variants in a complex genetic population over multiple generations, paving the way for understanding how methylation variants contribute to phenotypic variation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23739894">http://www.ncbi.nlm.nih.gov/pubmed/23739894</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Signal Behav(2013).8(1):e22598. doi:10.4161/Psb.22598; PMID:23221760</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Role of LysM receptors in chitin-triggered plant innate immunity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Cuong T Nguyen, Yan Liang, Yangrong Cao, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Sciences and Biochemistry; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Recent research findings clearly indicate that lysin motif (LysM)-containing cell surface receptors are involved in the recognition of specific oligosaccharide elicitors (chitin and peptidoglycan), which trigger an innate immunity response in plants. These receptors are either LysM-containing receptor-like kinases (LYKs) or LysM-containing receptor proteins (LYPs). In Arabidopsis, five LYKs (AtCERK1/AtLYK1 and AtLYK2-5) and three LYPs (AtLYP1-3) are likely expressed on the plasma membrane. In this review, we summarize recent research results on the role of these receptors in plant innate immunity, including the recent structural characterization of AtCERK1 and composition of the various receptor complexes in Arabidopsis.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23221760">http://www.ncbi.nlm.nih.gov/pubmed/23221760</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2013).161(1):36-47. doi:10.1104/Pp.112.205369; PMID:23124322</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Tnt1 retrotransposon mutagenesis: a tool for soybean functional genomics.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yaya Cui, Shyam Barampuram, Minviluz G Stacey, C Nathan Hancock, Seth Findley, Melanie Mathieu, Zhanyuan Zhang, Wayne A Parrott, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Divisions of Plant Sciences and Biochemistry; National Center for Soybean Biotechnology; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Insertional mutagenesis is a powerful tool for determining gene function in both model and crop plant species. Tnt1, the transposable element of tobacco (Nicotiana tabacum) cell type 1, is a retrotransposon that replicates via an RNA copy that is reverse transcribed and integrated elsewhere in the plant genome. Based on studies in a variety of plants, Tnt1 appears to be inactive in normal plant tissue but can be reactivated by tissue culture. Our goal was to evaluate the utility of the Tnt1 retrotransposon as a mutagenesis strategy in soybean (Glycine max). Experiments showed that the Tnt1 element was stably transformed into soybean plants by Agrobacterium tumefaciens-mediated transformation. Twenty-seven independent transgenic lines carrying Tnt1 insertions were generated. Southern-blot analysis revealed that the copy number of transposed Tnt1 elements ranged from four to 19 insertions, with an average of approximately eight copies per line. These insertions showed Mendelian segregation and did not transpose under normal growth conditions. Analysis of 99 Tnt1 flanking sequences revealed insertions into 62 (62%) annotated genes, indicating that the element preferentially inserts into protein-coding regions. Tnt1 insertions were found in all 20 soybean chromosomes, indicating that Tnt1 transposed throughout the soybean genome. Furthermore, fluorescence in situ hybridization experiments validated that Tnt1 inserted into multiple chromosomes. Passage of transgenic lines through two different tissue culture treatments resulted in Tnt1 transposition, significantly increasing the number of insertions per line. Thus, our data demonstrate the Tnt1 retrotransposon to be a powerful system that can be used for effective large-scale insertional mutagenesis in soybean.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23124322">http://www.ncbi.nlm.nih.gov/pubmed/23124322</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2012).160(4):2125-36. doi:10.1104/Pp.112.207027; PMID:23060368</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">14-3-3 proteins SGF14c and SGF14l play critical roles during soybean nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Osman Radwan, Xia Wu, Manjula Govindarajulu, Marc Libault, David J Neece, Man-Ho Oh, R Howard Berg, Gary Stacey, Christopher G Taylor, Steven C Huber, Steven J Clough</b>
      </span>
      <br/>
      <span>Affiliation: Department of Crop Sciences; University of Illinois; Urbana; Illinois 61801; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The soybean (Glycine max) genome contains 18 members of the 14-3-3 protein family, but little is known about their association with specific phenotypes. Here, we report that the Glyma0529080 Soybean G-box Factor 14-3-3c (SGF14c) and Glyma08g12220 (SGF14l) genes, encoding 14-3-3 proteins, appear to play essential roles in soybean nodulation. Quantitative reverse transcription-polymerase chain reaction and western-immunoblot analyses showed that SGF14c mRNA and protein levels were specifically increased in abundance in nodulated soybean roots 10, 12, 16, and 20 d after inoculation with Bradyrhizobium japonicum. To investigate the role of SGF14c during soybean nodulation, RNA interference was employed to silence SGF14c expression in soybean roots using Agrobacterium rhizogenes-mediated root transformation. Due to the paleopolyploid nature of soybean, designing a specific RNA interference sequence that exclusively targeted SGF14c was not possible. Therefore, two highly similar paralogs (SGF14c and SGF14l) that have been shown to function as dimers were silenced. Transcriptomic and proteomic analyses showed that mRNA and protein levels were significantly reduced in the SGF14c/SGF14l-silenced roots, and these roots exhibited reduced numbers of mature nodules. In addition, SGF14c/SGF14l-silenced roots contained large numbers of arrested nodule primordia following B. japonicum inoculation. Transmission electron microscopy further revealed that the host cytoplasm and membranes, except the symbiosome membrane, were severely degraded in the failed nodules. Altogether, transcriptomic, proteomic, and cytological data suggest a critical role of one or both of these 14-3-3 proteins in early development stages of soybean nodules.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23060368">http://www.ncbi.nlm.nih.gov/pubmed/23060368</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Proteomics(2012).12(22):3365-73. doi:10.1002/Pmic.201200160; PMID:22997094</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification of soybean proteins from a single cell type: the root hair.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Laurent Brechenmacher, Tran Hong Nha Nguyen, Kim Hixson, Marc Libault, Joshua Aldrich, Ljiljana Pasa-Tolic, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Root hairs (RH) are a terminally differentiated single cell type, mainly involved in water and nutrient uptake from the soil. The soybean RH cell represents an excellent model for the study of single cell systems biology. In this study, we identified 5702 proteins, with at least two peptides, from soybean RH using an accurate mass and time tag approach, establishing a comprehensive proteome reference map of this single cell type. We also showed that trypsin is the most appropriate enzyme for soybean proteomic studies by performing an in silico digestion of the soybean proteome using different proteases. Although the majority of proteins identified in this study are involved in basal metabolism, the function of others are more related to RH formation/function and include proteins involved in nutrient uptake (transporters) or vesicular trafficking (cytoskeleton and ras-associated binding proteins). Interestingly, some of these proteins appear to be specifically detected in RH and constitute promising candidates for further studies to elucidate unique features of this single-cell model.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22997094">http://www.ncbi.nlm.nih.gov/pubmed/22997094</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Planta(2013).237(1):65-75. doi:10.1007/S00425-012-1757-0; PMID:22983673</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Shoot to root communication is necessary to control the expression of iron-acquisition genes in Strategy I plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Mar&#195;&#173;a J Garc&#195;&#173;a, Francisco J Romera, Minviluz G Stacey, Gary Stacey, Eduardo Villar, Esteban Alc&#195;&#161;ntara, Rafael P&#195;&#169;rez-Vicente</b>
      </span>
      <br/>
      <span>Affiliation: Department of Botany; Ecology and Plant Physiology; Edificio Celestino Mutis C-4; Campus de Rabanales; University of C&#195;&#179;rdoba; 14014 C&#195;&#179;rdoba; Spain.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Previous research showed that auxin, ethylene, and nitric oxide (NO) can activate the expression of iron (Fe)-acquisition genes in the roots of Strategy I plants grown with low levels of Fe, but not in plants grown with high levels of Fe. However, it is still an open question as to how Fe acts as an inhibitor and which pool of Fe (e.g., root, phloem, etc.) in the plant acts as the key regulator for gene expression control. To further clarify this, we studied the effect of the foliar application of Fe on the expression of Fe-acquisition genes in several Strategy I plants, including wild-type cultivars of Arabidopsis [Arabidopsis thaliana (L.) Heynh], pea [Pisum sativum L.], tomato [Solanum lycopersicon Mill.], and cucumber [Cucumis sativus L.], as well as mutants showing constitutive expression of Fe-acquisition genes when grown under Fe-sufficient conditions [Arabidopsis opt3-2 and frd3-3, pea dgl and brz, and tomato chln (chloronerva)]. The results showed that the foliar application of Fe blocked the expression of Fe-acquisition genes in the wild-type cultivars and in the frd3-3, brz, and chln mutants, but not in the opt3-2 and dgl mutants, probably affected in the transport of a Fe-related repressive signal in the phloem. Moreover, the addition of either ACC (ethylene precursor) or GSNO (NO donor) to Fe-deficient plants up-regulated the expression of Fe-acquisition genes, but this effect did not occur in Fe-deficient plants sprayed with foliar Fe, again suggesting the existence of a Fe-related repressive signal moving from leaves to roots.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22983673">http://www.ncbi.nlm.nih.gov/pubmed/22983673</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant J.(2013).73(1):143-53. doi:10.1111/Tpj.12026; PMID:22974547</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The fate of duplicated genes in a polyploid plant genome.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Anne Roulin, Paul L Auer, Marc Libault, Jessica Schlueter, Andrew Farmer, Greg May, Gary Stacey, Rebecca W Doerge, Scott A Jackson</b>
      </span>
      <br/>
      <span>Affiliation: Institute for Plant Breeding; Genetics and Genomics; University of Georgia; 111 Riverbend Road; Athens; GA; 30602; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Polyploidy is generally not tolerated in animals, but is widespread in plant genomes and may result in extensive genetic redundancy. The fate of duplicated genes is poorly understood, both functionally and evolutionarily. Soybean (Glycine max L.) has undergone two separate polyploidy events (13 and 59 million years ago) that have resulted in 75% of its genes being present in multiple copies. It therefore constitutes a good model to study the impact of whole-genome duplication on gene expression. Using RNA-seq, we tested the functional fate of a set of approximately 18&#194;&#160;000 duplicated genes. Across seven tissues tested, approximately 50% of paralogs were differentially expressed and thus had undergone expression sub-functionalization. Based on gene ontology and expression data, our analysis also revealed that only a small proportion of the duplicated genes have been neo-functionalized or non-functionalized. In addition, duplicated genes were often found in collinear blocks, and several blocks of duplicated genes were co-regulated, suggesting some type of epigenetic or positional regulation. We also found that transcription factors and ribosomal protein genes were differentially expressed in many tissues, suggesting that the main consequence of polyploidy in soybean may be at the regulatory level.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22974547">http://www.ncbi.nlm.nih.gov/pubmed/22974547</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Genomics(2012).13():437. doi:10.1186/1471-2164-13-437; PMID:22938179</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Reconstructing differentially co-expressed gene modules and regulatory networks of soybean cells.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Mingzhu Zhu, Xin Deng, Trupti Joshi, Dong Xu, Gary Stacey, Jianlin Cheng</b>
      </span>
      <br/>
      <span>Affiliation: Department of Computer Science; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Current experimental evidence indicates that functionally related genes show coordinated expression in order to perform their cellular functions. In this way, the cell transcriptional machinery can respond optimally to internal or external stimuli. This provides a research opportunity to identify and study co-expressed gene modules whose transcription is controlled by shared gene regulatory networks.We developed and integrated a set of computational methods of differential gene expression analysis, gene clustering, gene network inference, gene function prediction, and DNA motif identification to automatically identify differentially co-expressed gene modules, reconstruct their regulatory networks, and validate their correctness. We tested the methods using microarray data derived from soybean cells grown under various stress conditions. Our methods were able to identify 42 coherent gene modules within which average gene expression correlation coefficients are greater than 0.8 and reconstruct their putative regulatory networks. A total of 32 modules and their regulatory networks were further validated by the coherence of predicted gene functions and the consistency of putative transcription factor binding motifs. Approximately half of the 32 modules were partially supported by the literature, which demonstrates that the bioinformatic methods used can help elucidate the molecular responses of soybean cells upon various environmental stresses.The bioinformatics methods and genome-wide data sources for gene expression, clustering, regulation, and function analysis were integrated seamlessly into one modular protocol to systematically analyze and infer modules and networks from only differential expression genes in soybean cells grown under stress conditions. Our approach appears to effectively reduce the complexity of the problem, and is sufficiently robust and accurate to generate a rather complete and detailed view of putative soybean gene transcription logic potentially underlying the responses to the various environmental challenges. The same automated method can also be applied to reconstruct differentially co-expressed gene modules and their regulatory networks from gene expression data of any other transcriptome.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22938179">http://www.ncbi.nlm.nih.gov/pubmed/22938179</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Cell Proteomics(2012).11(11):1140-55. doi:10.1074/Mcp.M112.018028; PMID:22843990</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Quantitative phosphoproteomic analysis of soybean root hairs inoculated with Bradyrhizobium japonicum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Tran Hong Nha Nguyen, Laurent Brechenmacher, Joshua T Aldrich, Therese R Clauss, Marina A Gritsenko, Kim K Hixson, Marc Libault, Kiwamu Tanaka, Feng Yang, Qiuming Yao, Ljiljana Pasa-Toli&#196;&#135;, Dong Xu, Henry T Nguyen, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; C.S. Bond Life Sciences Center; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Root hairs are single hair-forming cells on roots that function to increase root surface area, enhancing water and nutrient uptake. In leguminous plants, root hairs also play a critical role as the site of infection by symbiotic nitrogen fixing rhizobia, leading to the formation of a novel organ, the nodule. The initial steps in the rhizobia-root hair infection process are known to involve specific receptor kinases and subsequent kinase cascades. Here, we characterize the phosphoproteome of the root hairs and the corresponding stripped roots (i.e. roots from which root hairs were removed) during rhizobial colonization and infection to gain insight into the molecular mechanism of root hair cell biology. We chose soybean (Glycine max L.), one of the most important crop plants in the legume family, for this study because of its larger root size, which permits isolation of sufficient root hair material for phosphoproteomic analysis. Phosphopeptides derived from root hairs and stripped roots, mock inoculated or inoculated with the soybean-specific rhizobium Bradyrhizobium japonicum, were labeled with the isobaric tag eight-plex iTRAQ, enriched using Ni-NTA magnetic beads and subjected to nanoRPLC-MS/MS1 analysis using HCD and decision tree guided CID/ETD strategy. A total of 1625 unique phosphopeptides, spanning 1659 nonredundant phosphorylation sites, were detected from 1126 soybean phosphoproteins. Among them, 273 phosphopeptides corresponding to 240 phosphoproteins were found to be significantly regulated (&gt;1.5-fold abundance change) in response to inoculation with B. japonicum. The data reveal unique features of the soybean root hair phosphoproteome, including root hair and stripped root-specific phosphorylation suggesting a complex network of kinase-substrate and phosphatase-substrate interactions in response to rhizobial inoculation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22843990">http://www.ncbi.nlm.nih.gov/pubmed/22843990</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2012).160(1):396-406. doi:10.1104/Pp.112.201699; PMID:22744984</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">LYK4, a lysin motif receptor-like kinase, is important for chitin signaling and plant innate immunity in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jinrong Wan, Kiwamu Tanaka, Xue-Cheng Zhang, Geon Hui Son, Laurent Brechenmacher, Tran Hong Nha Nguyen, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center ; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin is commonly found in fungal cell walls and is one of the well-studied microbe/pathogen-associated molecular patterns. Previous studies showed that lysin motif (LysM)-containing proteins are essential for plant recognition of chitin, leading to the activation of plant innate immunity. In Arabidopsis (Arabidopsis thaliana), the LYK1/CERK1 (for LysM-containing receptor-like kinase1/chitin elicitor receptor kinase1) was shown to be essential for chitin recognition, whereas in rice (Oryza sativa), the LysM-containing protein, CEBiP (for chitin elicitor-binding protein), was shown to be involved in chitin recognition. Unlike LYK1/CERK1, CEBiP lacks an intracellular kinase domain. Arabidopsis possesses three CEBiP-like genes. Our data show that mutations in these genes, either singly or in combination, did not compromise the response to chitin treatment. Arabidopsis also contains five LYK genes. Analysis of mutations in LYK2, -3, -4, or -5 showed that LYK4 is also involved in chitin signaling. The lyk4 mutants showed reduced induction of chitin-responsive genes and diminished chitin-induced cytosolic calcium elevation as well as enhanced susceptibility to both the bacterial pathogen Pseudomonas syringae pv tomato DC3000 and the fungal pathogen Alternaria brassicicola, although these phenotypes were not as dramatic as that seen in the lyk1/cerk1 mutants. Similar to LYK1/CERK1, the LYK4 protein was also localized to the plasma membrane. Therefore, LYK4 may play a role in the chitin recognition receptor complex to assist chitin signal transduction and plant innate immunity.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22744984">http://www.ncbi.nlm.nih.gov/pubmed/22744984</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>G3 (Bethesda)(2012).2(3):321-9. doi:10.1534/G3.111.001834; PMID:22413085</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Integration of the Draft Sequence and Physical Map as a Framework for Genomic Research in Soybean (Glycine max (L.) Merr.) and Wild Soybean (Glycine soja Sieb. and Zucc.).</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jungmin Ha, Brian Abernathy, William Nelson, David Grant, Xiaolei Wu, Henry T Nguyen, Gary Stacey, Yeisoo Yu, Rod A Wing, Randy C Shoemaker, Scott A Jackson</b>
      </span>
      <br/>
      <span>Affiliation: </span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean is a model for the legume research community because of its importance as a crop, densely populated genetic maps, and the availability of a genome sequence. Even though a whole-genome shotgun sequence and bacterial artificial chromosome (BAC) libraries are available, a high-resolution, chromosome-based physical map linked to the sequence assemblies is still needed for whole-genome alignments and to facilitate map-based gene cloning. Three independent G. max BAC libraries combined with genetic and gene-based markers were used to construct a minimum tiling path (MTP) of BAC clones. A total of 107,214 clones were assembled into 1355 FPC (FingerPrinted Contigs) contigs, incorporating 4628 markers and aligned to the G. max reference genome sequence using BAC end-sequence information. Four different MTPs were made for G. max that covered from 92.6% to 95.0% of the soybean draft genome sequence (gmax1.01). Because our purpose was to pick the most reliable and complete MTP, and not the MTP with the minimal number of clones, the FPC map and draft sequence were integrated and clones with unpaired BES were added to build a high-quality physical map with the fewest gaps possible (http://soybase.org). A physical map was also constructed for the undomesticated ancestor (G. soja) of soybean to explore genome variation between G. max and G. soja. 66,028 G. soja clones were assembled into 1053 FPC contigs covering approximately 547 Mbp of the G. max genome sequence. These physical maps for G. max and its undomesticated ancestor, G. soja, will serve as a framework for ordering sequence fragments, comparative genomics, cloning genes, and evolutionary analyses of legume genomes.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22413085">http://www.ncbi.nlm.nih.gov/pubmed/22413085</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>G3 (Bethesda)(2011).1(2):117-29. doi:10.1534/G3.111.000034; PMID:22384324</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Fluorescence in situ hybridization-based karyotyping of soybean translocation lines.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Seth D Findley, Allison L Pappas, Yaya Cui, James A Birchler, Reid G Palmer, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: </span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean (Glycine max [L.] Merr.) is a major crop species and, therefore, a major target of genomic and genetic research. However, in contrast to other plant species, relatively few chromosomal aberrations have been identified and characterized in soybean. This is due in part to the difficulty of cytogenetic analysis of its small, morphologically homogeneous chromosomes. The recent development of a fluorescence in situ hybridization -based karyotyping system for soybean has enabled our characterization of most of the chromosomal translocation lines identified to date. Utilizing genetic data from existing translocation studies in soybean, we identified the chromosomes and approximate breakpoints involved in five translocation lines.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22384324">http://www.ncbi.nlm.nih.gov/pubmed/22384324</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Genomics(2012).13 Suppl 1():S15. doi:10.1186/1471-2164-13-S1-S15; PMID:22369646</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Soybean Knowledge Base (SoyKB): a web resource for soybean translational genomics.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Trupti Joshi, Kapil Patil, Michael R Fitzpatrick, Levi D Franklin, Qiuming Yao, Jeffrey R Cook, Zheng Wang, Marc Libault, Laurent Brechenmacher, Babu Valliyodan, Xiaolei Wu, Jianlin Cheng, Gary Stacey, Henry T Nguyen, Dong Xu</b>
      </span>
      <br/>
      <span>Affiliation: Department of Computer Science; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean Knowledge Base (SoyKB) is a comprehensive all-inclusive web resource for soybean translational genomics. SoyKB is designed to handle the management and integration of soybean genomics, transcriptomics, proteomics and metabolomics data along with annotation of gene function and biological pathway. It contains information on four entities, namely genes, microRNAs, metabolites and single nucleotide polymorphisms (SNPs).SoyKB has many useful tools such as Affymetrix probe ID search, gene family search, multiple gene/metabolite search supporting co-expression analysis, and protein 3D structure viewer as well as download and upload capacity for experimental data and annotations. It has four tiers of registration, which control different levels of access to public and private data. It allows users of certain levels to share their expertise by adding comments to the data. It has a user-friendly web interface together with genome browser and pathway viewer, which display data in an intuitive manner to the soybean researchers, producers and consumers.SoyKB addresses the increasing need of the soybean research community to have a one-stop-shop functional and translational omics web resource for information retrieval and analysis in a user-friendly way. SoyKB can be publicly accessed at http://soykb.org/.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22369646">http://www.ncbi.nlm.nih.gov/pubmed/22369646</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Evol. Biol.(2012).12():6. doi:10.1186/1471-2148-12-6; PMID:22252370</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Evolutionary dynamics of protein domain architecture in plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Xue-Cheng Zhang, Zheng Wang, Xinyan Zhang, Mi Ha Le, Jianguo Sun, Dong Xu, Jianlin Cheng, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; University of Missouri; Columbia; MO 65211; USA. xzhang@molbio.mgh.harvard.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Protein domains are the structural, functional and evolutionary units of the protein. Protein domain architectures are the linear arrangements of domain(s) in individual proteins. Although the evolutionary history of protein domain architecture has been extensively studied in microorganisms, the evolutionary dynamics of domain architecture in the plant kingdom remains largely undefined. To address this question, we analyzed the lineage-based protein domain architecture content in 14 completed green plant genomes.Our analyses show that all 14 plant genomes maintain similar distributions of species-specific, single-domain, and multi-domain architectures. Approximately 65% of plant domain architectures are universally present in all plant lineages, while the remaining architectures are lineage-specific. Clear examples are seen of both the loss and gain of specific protein architectures in higher plants. There has been a dynamic, lineage-wise expansion of domain architectures during plant evolution. The data suggest that this expansion can be largely explained by changes in nuclear ploidy resulting from rounds of whole genome duplications. Indeed, there has been a decrease in the number of unique domain architectures when the genomes were normalized into a presumed ancestral genome that has not undergone whole genome duplications.Our data show the conservation of universal domain architectures in all available plant genomes, indicating the presence of an evolutionarily conserved, core set of protein components. However, the occurrence of lineage-specific domain architectures indicates that domain architecture diversity has been maintained beyond these core components in plant genomes. Although several features of genome-wide domain architecture content are conserved in plants, the data clearly demonstrate lineage-wise, progressive changes and expansions of individual protein domain architectures, reinforcing the notion that plant genomes have undergone dynamic evolution.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22252370">http://www.ncbi.nlm.nih.gov/pubmed/22252370</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Fungal Genet. Biol.(2012).49(2):95-100. doi:10.1016/J.Fgb.2011.12.010; PMID:22233882</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Nucleotide polymorphism in the 5.8S nrDNA gene and internal transcribed spacers in Phakopsora pachyrhizi viewed from structural models.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Ma&#195;&#173;ra Cristina Menezes Freire, Maria Rom&#195;&#169;ria da Silva, Xuecheng Zhang, &#195;&#129;lvaro Manuel Rodrigues Almeida, Gary Stacey, Luiz Orlando de Oliveira</b>
      </span>
      <br/>
      <span>Affiliation: Departamento de Bioqu&#195;&#173;mica e Biologia Molecular; Universidade Federal de Vi&#195;&#167;osa; 36570-000 Vi&#195;&#167;osa (MG); Brazil.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The assessment of nucleotide polymorphisms in environmental samples of obligate pathogens requires DNA amplification through the polymerase chain reaction (PCR) and bacterial cloning of PCR products prior to sequencing. The drawback of this strategy is that it can give rise to false polymorphisms owing to DNA polymerase misincorporation during PCR or bacterial cloning. We investigated patterns of nucleotide polymorphism in the internal transcribed spacer (ITS) region for Phakopsora pachyrhizi, an obligate biotrophic fungus that causes the Asian soybean rust. Field-collected samples of P. pachyrhizi were obtained from all major soybean production areas worldwide, including Brazil and the United States. Bacterially-cloned, PCR products were obtained using a high fidelity DNA polymerase. A total of 370 ITS sequences that were subjected to an array of complementary sequence analyses, which included analyses of secondary structure stability, the pattern of nucleotide polymorphisms, GC content, and the presence of conserved motifs. The sequences exhibited features of functional rRNAs. Overall, polymorphisms took place within less conserved motives, such as loops and bulges; alternatively, they gave rise to non-canonical G-U pairs within conserved regions of double stranded helices. We discuss the usefulness of structural analyses to filter out putative 'suspicious' bacterially cloned ITS sequences, thus keeping artificially-induced sequence variation to a minimum.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22233882">http://www.ncbi.nlm.nih.gov/pubmed/22233882</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Genes Dev.(2011).25(23):2540-53. doi:10.1101/Gad.177527.111; PMID:22156213</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">MicroRNAs as master regulators of the plant NB-LRR defense gene family via the production of phased, trans-acting siRNAs.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jixian Zhai, Dong-Hoon Jeong, Emanuele De Paoli, Sunhee Park, Benjamin D Rosen, Yupeng Li, Alvaro J Gonz&#195;&#161;lez, Zhe Yan, Sherry L Kitto, Michael A Grusak, Scott A Jackson, Gary Stacey, Douglas R Cook, Pamela J Green, D Janine Sherrier, Blake C Meyers</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant and Soil Sciences; University of Delaware; Newark; Delaware 19717; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Legumes and many nonleguminous plants enter symbiotic interactions with microbes, and it is poorly understood how host plants respond to promote beneficial, symbiotic microbial interactions while suppressing those that are deleterious or pathogenic. Trans-acting siRNAs (tasiRNAs) negatively regulate target transcripts and are characterized by siRNAs spaced in 21-nucleotide (nt) "phased" intervals, a pattern formed by DICER-LIKE 4 (DCL4) processing. A search for phased siRNAs (phasiRNAs) found at least 114 Medicago loci, the majority of which were defense-related NB-LRR-encoding genes. We identified three highly abundant 22-nt microRNA (miRNA) families that target conserved domains in these NB-LRRs and trigger the production of trans-acting siRNAs. High levels of small RNAs were matched to &gt;60% of all &#226;&#136;&#188;540 encoded Medicago NB-LRRs; in the potato, a model for mycorrhizal interactions, phasiRNAs were also produced from NB-LRRs. DCL2 and SGS3 transcripts were also cleaved by these 22-nt miRNAs, generating phasiRNAs, suggesting synchronization between silencing and pathogen defense pathways. In addition, a new example of apparent "two-hit" phasiRNA processing was identified. Our data reveal complex tasiRNA-based regulation of NB-LRRs that potentially evolved to facilitate symbiotic interactions and demonstrate miRNAs as master regulators of a large gene family via the targeting of highly conserved, protein-coding motifs, a new paradigm for miRNA function.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22156213">http://www.ncbi.nlm.nih.gov/pubmed/22156213</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2011).157(4):1975-86. doi:10.1104/Pp.111.183327; PMID:21963820</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification of quantitative trait loci controlling gene expression during the innate immunity response of soybean.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Oswaldo Vald&#195;&#169;s-L&#195;&#179;pez, Sandra Thibivilliers, Jing Qiu, Wayne Wenzhong Xu, Tran H N Nguyen, Marc Libault, Brandon H Le, Robert B Goldberg, Curtis B Hill, Glen L Hartman, Brian Diers, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Statistics; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Microbe-associated molecular pattern-triggered immunity (MTI) is an important component of the plant innate immunity response to invading pathogens. However, most of our knowledge of MTI comes from studies of model systems with relatively little work done with crop plants. In this work, we report on variation in both the microbe-associated molecular pattern-triggered oxidative burst and gene expression across four soybean (Glycine max) genotypes. Variation in MTI correlated with the level of pathogen resistance for each genotype. A quantitative trait locus analysis on these traits identified four loci that appeared to regulate gene expression during MTI in soybean. Likewise, we observed that both MTI variation and pathogen resistance were quantitatively inherited. The approach utilized in this study may have utility for identifying key resistance loci useful for developing improved soybean cultivars.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21963820">http://www.ncbi.nlm.nih.gov/pubmed/21963820</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Appl. Environ. Microbiol.(2011).77(22):8106-13. doi:10.1128/Aem.05925-11; PMID:21948825</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Activity-based metagenomic screening and biochemical characterization of bovine ruminal protozoan glycoside hydrolases.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Seth D Findley, Melanie R Mormile, Andrea Sommer-Hurley, Xue-Cheng Zhang, Peter Tipton, Krista Arnett, James H Porter, Monty Kerley, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Center for Sustainable Energy; National Center for Soybean Biotechnology; Division of Plant Sciences; University of Missouri; Columbia; Missouri; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The rumen, the foregut of herbivorous ruminant animals such as cattle, functions as a bioreactor to process complex plant material. Among the numerous and diverse microbes involved in ruminal digestion are the ruminal protozoans, which are single-celled, ciliated eukaryotic organisms. An activity-based screen was executed to identify genes encoding fibrolytic enzymes present in the metatranscriptome of a bovine ruminal protozoan-enriched cDNA expression library. Of the four novel genes identified, two were characterized in biochemical assays. Our results provide evidence for the effective use of functional metagenomics to retrieve novel enzymes from microbial populations that cannot be maintained in axenic cultures.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21948825">http://www.ncbi.nlm.nih.gov/pubmed/21948825</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2012).25(1):48-60. doi:10.1094/Mpmi-06-11-0165; PMID:21936663</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Ethylene-responsive element-binding factor 5, ERF5, is involved in chitin-induced innate immunity response.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Geon Hui Son, Jinrong Wan, Hye Jin Kim, Xuan Canh Nguyen, Woo Sik Chung, Jong Chan Hong, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: University of Missouri; Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Our recent work demonstrated that chitin treatment modulated the expression of 118 transcription factor (TF) genes in Arabidopsis. To investigate the potential roles of these TF in chitin signaling and plant defense, we initiated an interaction study among these TF proteins, as well as two chitin-activated mitogen-activated protein kinases (MPK3 and MPK6), using a yeast two-hybrid system. This study revealed interactions among the following proteins: three ethylene-responsive element-binding factors (ERF), five WRKY transcription factors, one scarecrow-like (SCL), and the two MPK, in addition to many other interactions, reflecting a complex TF interaction network. Most of these interactions were subsequently validated by other methods, such as pull-down and in planta bimolecular fluorescence complementation assays. The key node ERF5 was shown to interact with multiple proteins in the network, such as ERF6, ERF8, and SCL13, as well as MPK3 and MPK6. Interestingly, ERF5 appeared to negatively regulate chitin signaling and plant defense against the fungal pathogen Alternaria brassicicola and positively regulate salicylic acid signaling and plant defense against the bacterial pathogen Pseudomonas syringae pv. tomato DC3000. Therefore, ERF5 may play an important role in plant innate immunity, likely through coordinating chitin and other defense pathways in plants in response to different pathogens.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21936663">http://www.ncbi.nlm.nih.gov/pubmed/21936663</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Proc. Natl. Acad. Sci. U.S.A.(2011).108(37):15390-5. doi:10.1073/Pnas.1110137108; PMID:21880957</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Bacterial outer membrane channel for divalent metal ion acquisition.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Thomas H Hohle, William L Franck, Gary Stacey, Mark R O'Brian</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry; State University of New York; Buffalo; NY 14214; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The prevailing model of bacterial membrane function predicts that the outer membrane is permeable to most small solutes because of pores with limited selectivity based primarily on size. Here, we identified mnoP in the Gram-negative bacterium Bradyrhizobium japonicum as a gene coregulated with the inner membrane Mn(2+) transporter gene mntH. MnoP is an outer membrane protein expressed specifically under manganese limitation. MnoP acts as a channel to facilitate the tranlocation of Mn(2+), but not Co(2+) or Cu(2+), into reconstituted proteoliposomes. An mnoP mutant is defective in high-affinity Mn(2+) transport into cells and has a severe growth phenotype under manganese limitation. We suggest that the outer membrane is a barrier to divalent metal ions that requires a selective channel to meet the nutritional needs of the cell.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21880957">http://www.ncbi.nlm.nih.gov/pubmed/21880957</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2011).24(9):1051-60. doi:10.1094/Mpmi-12-10-0281; PMID:21815830</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A dual-targeted soybean protein is involved in Bradyrhizobium japonicum infection of soybean root hair and cortical cells.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Manjula Govindarajulu, R Howard Berg, Yee Tsuey Ong, Kari Puricelli, Christopher G Taylor, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; University of Missouri; Columbia; MO; USA. libaultm@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The symbiotic interaction between legumes and soil bacteria (e.g., soybean [Glycine max L.] and Bradyrhizobium japonicum]) leads to the development of a new root organ, the nodule, where bacteria differentiate into bacteroids that fix atmospheric nitrogen for assimilation by the plant host. In exchange, the host plant provides a steady carbon supply to the bacteroids. This carbon can be stored within the bacteroids in the form of poly-3-hydroxybutyrate granules. The formation of this symbiosis requires communication between both partners to regulate the balance between nitrogen fixation and carbon utilization. In the present study, we describe the soybean gene GmNMNa that is specifically expressed during the infection of soybean cells by B. japonicum. GmNMNa encodes a protein of unknown function. The GmNMNa protein was localized to the nucleolus and also to the mitochondria. Silencing of GmNMNa expression resulted in reduced nodulation, a reduction in the number of bacteroids per infected cell in the nodule, and a clear reduction in the accumulation of poly-3-hydroxybutyrate in the bacteroids. Our results highlight the role of the soybean GmNMNa gene in regulating symbiotic bacterial infection, potentially through the regulation of the accumulation of carbon reserves.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21815830">http://www.ncbi.nlm.nih.gov/pubmed/21815830</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Signal Behav(2011).6(7):1034-6. doi:; PMID:21617385</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Enzymatic role for soybean ecto-apyrase in nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Tran H N Nguyen, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; University of Missouri; Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Root nodulation is regulated by a variety of mechanisms. Ecto-apyrase is an enzyme proposed to control the concentration of extracellular nucleotides. Transgenic expression of the soybean GS52 ecto-apyrase was shown to stimulate nodulation. However, mutation of the enzyme to disrupt enzymatic activity prevented this effect. Therefore, the data suggest that the enzymatic activity of the ecto-apyrase is critical for nodulation enhancement, suggesting a direct effect on extracellular nucleotide hydrolysis. In this article, we propose a hypothetical mechanism for plant ecto-apyrase function during nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21617385">http://www.ncbi.nlm.nih.gov/pubmed/21617385</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>PLoS ONE(2011).6(3):e17906. doi:10.1371/Journal.Pone.0017906; PMID:21455299</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A protein domain co-occurrence network approach for predicting protein function and inferring species phylogeny.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zheng Wang, Xue-Cheng Zhang, Mi Ha Le, Dong Xu, Gary Stacey, Jianlin Cheng</b>
      </span>
      <br/>
      <span>Affiliation: Department of Computer Science; University of Missouri; Columbia; Missouri; United States of America.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Protein Domain Co-occurrence Network (DCN) is a biological network that has not been fully-studied. We analyzed the properties of the DCNs of H. sapiens, S. cerevisiae, C. elegans, D. melanogaster, and 15 plant genomes. These DCNs have the hallmark features of scale-free networks. We investigated the possibility of using DCNs to predict protein and domain functions. Based on our experiment conducted on 66 randomly selected proteins, the best of top 3 predictions made by our DCN-based aggregated neighbor-counting method achieved a semantic similarity score of 0.81 to the actual Gene Ontology terms of the proteins. Moreover, the top 3 predictions using neighbor-counting, &#207;&#135;(2), and a SVM-based method achieved an accuracy of 66%, 59%, and 61%, respectively, when used to predict specific Gene Ontology terms of human target domains. These predictions on average had a semantic similarity score of 0.82, 0.80, and 0.79 to the actual Gene Ontology terms, respectively. We also used DCNs to predict whether a domain is an enzyme domain, and our SVM-based and neighbor-inference method correctly classified 79% and 77% of the target domains, respectively. When using DCNs to classify a target domain into one of the six enzyme classes, we found that, as long as there is one EC number available in the neighboring domains, our SVM-based and neighboring-counting method correctly classified 92.4% and 91.9% of the target domains, respectively. Furthermore, we benchmarked the performance of using DCNs to infer species phylogenies on six different combinations of 398 single-chromosome prokaryotic genomes. The phylogenetic tree of 54 prokaryotic taxa generated by our DCNs-alignment-based method achieved a 93.45% similarity score compared to the Bergey's taxonomy. In summary, our studies show that genome-wide DCNs contain rich information that can be effectively used to decipher protein function and reveal the evolutionary relationship among species.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21455299">http://www.ncbi.nlm.nih.gov/pubmed/21455299</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2011).155(4):1988-98. doi:10.1104/Pp.110.170910; PMID:21346172</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Enzymatic activity of the soybean ecto-apyrase GS52 is essential for stimulation of nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Cuong T Nguyen, Marc Libault, Jianlin Cheng, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; C.S. Bond Life Sciences Center; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Nitrogen is an essential nutrient for plant growth. In the Rhizobium-legume symbiosis, root nodules are the sites of bacterial nitrogen fixation, in which atmospheric nitrogen is converted into a form that plants can utilize. While recent studies suggested an important role for the soybean (Glycine max) ecto-apyrase GS52 in rhizobial root hair infection and root nodule formation, precisely how this protein impacts the nodulation process remains undetermined. In this study, the biochemical characteristics of the GS52 enzyme were investigated. Computer modeling of the GS52 apyrase structure identified key amino acid residues important for catalytic activity, which were subsequently mutagenized. Although the GS52 enzyme exhibited broad substrate specificity, its activity on pyrimidine nucleotides and diphosphate nucleotides was significantly higher than on ATP. This result was corroborated by structural modeling of GS52, which predicted a low specificity for the adenine base within the substrate-binding pocket of the enzyme. The wild-type enzyme and its inactive mutant forms were expressed in soybean roots in order to evaluate the importance of GS52 enzymatic activity for nodulation. The results indicated a clear correlation between GS52 enzymatic activity and nodule number. Altogether, our study indicates that the catalytic activity of the GS52 apyrase, likely acting on extracellular nucleotides, is critical for rhizobial infection and nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21346172">http://www.ncbi.nlm.nih.gov/pubmed/21346172</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2011).156(1):240-53. doi:10.1104/Pp.110.170811; PMID:21321255</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Phenotypic and genomic analyses of a fast neutron mutant population resource in soybean.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Yung-Tsi Bolon, William J Haun, Wayne W Xu, David Grant, Minviluz G Stacey, Rex T Nelson, Daniel J Gerhardt, Jeffrey A Jeddeloh, Gary Stacey, Gary J Muehlbauer, James H Orf, Seth L Naeve, Robert M Stupar, Carroll P Vance</b>
      </span>
      <br/>
      <span>Affiliation: Plant Science Research Unit; United States Department of Agriculture-Agricultural Research Service; St. Paul; Minnesota 55108; USA. hsie0024@umn.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Mutagenized populations have become indispensable resources for introducing variation and studying gene function in plant genomics research. In this study, fast neutron (FN) radiation was used to induce deletion mutations in the soybean (Glycine max) genome. Approximately 120,000 soybean seeds were exposed to FN radiation doses of up to 32 Gray units to develop over 23,000 independent M2 lines. Here, we demonstrate the utility of this population for phenotypic screening and associated genomic characterization of striking and agronomically important traits. Plant variation was cataloged for seed composition, maturity, morphology, pigmentation, and nodulation traits. Mutants that showed significant increases or decreases in seed protein and oil content across multiple generations and environments were identified. The application of comparative genomic hybridization (CGH) to lesion-induced mutants for deletion mapping was validated on a midoleate x-ray mutant, M23, with a known FAD2-1A (for fatty acid desaturase) gene deletion. Using CGH, a subset of mutants was characterized, revealing deletion regions and candidate genes associated with phenotypes of interest. Exome resequencing and sequencing of PCR products confirmed FN-induced deletions detected by CGH. Beyond characterization of soybean FN mutants, this study demonstrates the utility of CGH, exome sequence capture, and next-generation sequencing approaches for analyses of mutant plant genomes. We present this FN mutant soybean population as a valuable public resource for future genetic screens and functional genomics research.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21321255">http://www.ncbi.nlm.nih.gov/pubmed/21321255</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Signal Behav(2010).5(10):1226-8. doi:10.4161/Psb.5.10.12808; PMID:20855956</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Evolution of FW2.2-like (FWL) and PLAC8 genes in eukaryotes.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; C.S. Bond Life Sciences Center; University of Missouri; Columbia; MO; USA. libaultm@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The tomato FW2.2 quantitative trait locus, which regulates tomato fruit size, was genetically and physically mapped around 15 years ago. Subsequently, the FW2.2 gene was cloned and shown to contain a PLAC8 domain, originally identified in mammalian placental proteins. Data suggest that FW2.2 likely controls tomato cell size, perhaps by direct interaction with casein kinase II. Several FW2.2-like (FWL) genes have now been identified from a variety of plant species, but until recently only the tomato FW2.2 gene had been the subject of detailed investigation. Recently, soybean and maize FWL genes were identified and shown to have a role in plant organogenesis. It is now apparent that the FWL genes in plants are a large gene family, which is even larger given inclusion of genes for the various eukaryotic PLAC8-domain proteins. Although overall the protein sequence identity/similarity among the family members is relatively low, there is strong conservation of key domains, suggesting a conservation of the core biochemical function of these proteins. In this Addendum Article, we highlight the similarities and differences exiting between plant FWL genes and enlarge this comparison to the mammalian PLAC8 genes. These comparisons suggest the possible conservation of biological function for FWL proteins.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20855956">http://www.ncbi.nlm.nih.gov/pubmed/20855956</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Trends Plant Sci.(2010).15(11):641-50. doi:10.1016/J.Tplants.2010.08.010; PMID:20851035</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Root hair systems biology.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Laurent Brechenmacher, Jianlin Cheng, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; C.S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA. libaultm@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plant functional genomic studies have largely measured the response of whole plants, organs and tissues, resulting in the dilution of the signal from individual cells. Methods are needed where the full repertoire of functional genomic tools can be applied to a single plant cell. Root hair cells are an attractive model to study the biology of a single, differentiated cell type because of their ease of isolation, polar growth, and role in water and nutrient uptake, as well as being the site of infection by nitrogen-fixing bacteria. This review highlights the recent advances in our understanding of plant root hair biology and examines whether the root hair has potential as a model for plant cell systems biology.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20851035">http://www.ncbi.nlm.nih.gov/pubmed/20851035</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Phytopathology(2010).100(10):1111-7. doi:10.1094/Phyto-12-09-0365; PMID:20839946</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Combinatorially selected peptides for protection of soybean against Phakopsora pachyrhizi.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zhiwei D Fang, James J Marois, Gary Stacey, James E Schoelz, James T English, Francis J Schmidt</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Phakopsora pachyrhizi, the fungal pathogen that causes Asian soybean rust, has the potential to cause significant losses in soybean yield in many production regions of the United States. Germplasm with durable, single-gene resistance is lacking, and control of rust depends on timely application of fungicides. To assist the development of new modes of soybean resistance, we identified peptides from combinatorial phage-display peptide libraries that inhibit germ tube growth from urediniospores of P. pachyrhizi. Two peptides, Sp2 and Sp39, were identified that inhibit germ tube development when displayed as fusions with the coat protein of M13 phage or as fusions with maize cytokinin oxidase/dehydrogenase (ZmCKX1). In either display format, the inhibitory effect of the peptides on germ tube growth was concentration dependent. In addition, when peptides Sp2 or Sp39 in either format were mixed with urediniospores and inoculated to soybean leaves with an 8-h wetness period, rust lesion development was reduced. Peptides Sp2 and Sp39, displayed on ZmCKX1, were found to interact with a 20-kDa protein derived from germinated urediniospores. Incorporating peptides that inhibit pathogen development and pathogenesis into breeding programs may contribute to the development of soybean cultivars with improved, durable rust tolerance.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20839946">http://www.ncbi.nlm.nih.gov/pubmed/20839946</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Trends Cell Biol.(2010).20(10):601-8. doi:10.1016/J.Tcb.2010.07.005; PMID:20817461</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Extracellular ATP signaling in plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Simon Gilroy, Alan M Jones, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Extracellular adenosine-5'-triphosphate (ATP) induces a number of cellular responses in plants and animals. Some of the molecular components for purinergic signaling in animal cells appear to be lacking in plant cells, although some cellular responses are similar in both systems [e.g. increased levels of cytosolic free calcium, nitric oxide (NO), and reactive oxygen species (ROS)]. The purpose of this review is to compare and contrast purinergic signaling mechanisms in animal and plant cells. This comparison will aid our overall understanding of plant physiology and also provide details of the general fundamentals of extracellular ATP signaling in eukaryotes.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20817461">http://www.ncbi.nlm.nih.gov/pubmed/20817461</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2010).154(2):705-19. doi:10.1104/Pp.110.162503; PMID:20671112</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Extracellular nucleotides elicit cytosolic free calcium oscillations in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kiwamu Tanaka, Sarah J Swanson, Simon Gilroy, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Division of Plant Sciences; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Extracellular ATP induces a rise in the level of cytosolic free calcium ([Ca(2+)](cyt)) in plant cells. To expand our knowledge about the function of extracellular nucleotides in plants, the effects of several nucleotide analogs and pharmacological agents on [Ca(2+)](cyt) changes were studied using transgenic Arabidopsis (Arabidopsis thaliana) expressing aequorin or the fluorescence resonance energy transfer-based Ca(2+) sensor Yellow Cameleon 3.6. Exogenously applied CTP caused elevations in [Ca(2+)](cyt) that displayed distinct time- and dose-dependent kinetics compared with the purine nucleotides ATP and GTP. The inhibitory effects of antagonists of mammalian P2 receptors and calcium influx inhibitors on nucleotide-induced [Ca(2+)](cyt) elevations were distinct between CTP and purine nucleotides. These results suggest that distinct recognition systems may exist for the respective types of nucleotides. Interestingly, a mutant lacking the heterotrimeric G protein G&#206;&#178;-subunit exhibited a remarkably higher [Ca(2+)](cyt) elevation in response to all tested nucleotides in comparison with the wild type. These data suggest a role for G&#206;&#178; in negatively regulating extracellular nucleotide signaling and point to an important role for heterotrimeric G proteins in modulating the cellular effects of extracellular nucleotides. The addition of extracellular nucleotides induced multiple temporal [Ca(2+)](cyt) oscillations, which could be localized to specific root cells. The oscillations were attenuated by a vesicle-trafficking inhibitor, indicating that the oscillations likely require ATP release via exocytotic secretion. The results reveal new molecular details concerning extracellular nucleotide signaling in plants and the importance of fine control of extracellular nucleotide levels to mediate specific plant cell responses.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20671112">http://www.ncbi.nlm.nih.gov/pubmed/20671112</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Pathol.(2002).3(5):301-11. doi:10.1046/J.1364-3703.2002.00123.X; PMID:20569338</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Microarray analysis of chitin elicitation in Arabidopsis thaliana.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Katrina M Ramonell, Bing Zhang, Rob M Ewing, Yu Chen, Dong Xu, Gary Stacey, Shauna Somerville</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Biology; Carnegie Institution of Washington; 260 Panama Street; Stanford; CA 94305; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Summary Chitin oligomers, released from fungal cell walls by endochitinase, induce defence and related cellular responses in many plants. However, little is known about chitin responses in the model plant Arabidopsis. We describe here a large-scale characterization of gene expression patterns in Arabidopsis in response to chitin treatment using an Arabidopsis microarray consisting of 2375 EST clones representing putative defence-related and regulatory genes. Transcript levels for 71 ESTs, representing 61 genes, were altered three-fold or more in chitin-treated seedlings relative to control seedlings. A number of transcripts exhibited altered accumulation as early as 10 min after exposure to chitin, representing some of the earliest changes in gene expression observed in chitin-treated plants. Included among the 61 genes were those that have been reported to be elicited by various pathogen-related stimuli in other plants. Additional genes, including genes of unknown function, were also identified, broadening our understanding of chitin-elicited responses. Among transcripts with enhanced accumulation, one cluster was enriched in genes with both the W-box promoter element and a novel regulatory element. In addition, a number of transcripts had decreased abundance, encoding several proteins involved in cell wall strengthening and wall deposition. The chalcone synthase promoter element was identified in the upstream regions of these genes, suggesting that pathogen signals may suppress the expression of some genes. These data indicate that Arabidopsis should be an excellent model to elucidate the mechanisms of chitin elicitation in plant defence.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20569338">http://www.ncbi.nlm.nih.gov/pubmed/20569338</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Pathol.(2004).5(2):125-35. doi:10.1111/J.1364-3703.2004.00215.X; PMID:20565589</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Activation of a mitogen-activated protein kinase pathway in Arabidopsis by chitin.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jinrong Wan, Shuqun Zhang, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Microbiology and Pathology; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>SUMMARY Chitin, a polysaccharide composed of beta-1--&gt;4-linked N-acetyl-d-glucosamine, has been shown or implicated as a signal in plant defence and development. However, the key components of chitin perception and downstream signalling in non-leguminous plants are largely unknown. In recent years, mitogen-activated protein kinases (MAPKs) and their cascades were shown to transduce various extracellular stimuli into internal cellular responses. To investigate the possible involvement of MAPKs in chitin signalling in plants, the model plant Arabidopsis thaliana was treated with crab-shell chitin and also with the purified chitin oligomers (degree of polymerization, d.p. = 2-8). Both mRNA levels and kinase activity of two MAPK genes, AtMPK6 and AtMPK3, were monitored after treatment. The mRNA of AtMPK3 was strongly up-regulated by both chitin and its larger oligomers (d.p. = 6-8), but the mRNA of AtMPK6 did not appear to be regulated by these treatments. However, the kinase activity of both MAPKs was induced by chitin and the larger oligomers (d.p. = 6-8), with AtMPK6 much more strongly induced. In addition, WRKY22, WRKY29, WRKY33 and WRKY53, which encode four WRKY transcription factors that recognize TTGAC(C/T) W-box elements in promoters of numerous plant defence-related genes, were up-regulated by these treatments. WRKY33 and WRKY53 expression was induced by the transgenic expression of the tobacco MAPKK NtMEK2 active mutant NtMEK2(DD), suggesting a potential role for these WRKY transcription factors in relaying the signal generated from the MAPK cascade to downstream genes. These data suggest that AtMPK6/AtMPK3 and WRKY transcription factors (such as WRKY33 and WRKY53) may be important components of a pathway involved in chitin signalling in Arabidopsis plants.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20565589">http://www.ncbi.nlm.nih.gov/pubmed/20565589</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2010).153(4):1808-22. doi:10.1104/Pp.110.157800; PMID:20534735</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Soybean metabolites regulated in root hairs in response to the symbiotic bacterium Bradyrhizobium japonicum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Laurent Brechenmacher, Zhentian Lei, Marc Libault, Seth Findley, Masayuki Sugawara, Michael J Sadowsky, Lloyd W Sumner, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Division of Plant Sciences; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Nodulation of soybean (Glycine max) root hairs by the nitrogen-fixing symbiotic bacterium Bradyrhizobium japonicum is a complex process coordinated by the mutual exchange of diffusible signal molecules. A metabolomic study was performed to identify small molecules produced in roots and root hairs during the rhizobial infection process. Metabolites extracted from roots and root hairs mock inoculated or inoculated with B. japonicum were analyzed by gas chromatography-mass spectrometry and ultraperformance liquid chromatography-quadrupole time of flight-mass spectrometry. These combined approaches identified 2,610 metabolites in root hairs. Of these, 166 were significantly regulated in response to B. japonicum inoculation, including various (iso)flavonoids, amino acids, fatty acids, carboxylic acids, and various carbohydrates. Trehalose was among the most strongly induced metabolites produced following inoculation. Subsequent metabolomic analyses of root hairs inoculated with a B. japonicum mutant defective in the trehalose synthase, trehalose 6-phosphate synthase, and maltooligosyltrehalose synthase genes showed that the trehalose detected in the inoculated root hairs was primarily of bacterial origin. Since trehalose is generally considered an osmoprotectant, these data suggest that B. japonicum likely experiences osmotic stress during the infection process, either on the root hair surface or within the infection thread.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20534735">http://www.ncbi.nlm.nih.gov/pubmed/20534735</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Genetics(2010).185(3):727-44. doi:10.1534/Genetics.109.113753; PMID:20421607</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A fluorescence in situ hybridization system for karyotyping soybean.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Seth D Findley, Steven Cannon, Kranthi Varala, Jianchang Du, Jianxin Ma, Matthew E Hudson, James A Birchler, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The development of a universal soybean (Glycine max [L.] Merr.) cytogenetic map that associates classical genetic linkage groups, molecular linkage groups, and a sequence-based physical map with the karyotype has been impeded due to the soybean chromosomes themselves, which are small and morphologically homogeneous. To overcome this obstacle, we screened soybean repetitive DNA to develop a cocktail of fluorescent in situ hybridization (FISH) probes that could differentially label mitotic chromosomes in root tip preparations. We used genetically anchored BAC clones both to identify individual chromosomes in metaphase spreads and to complete a FISH-based karyotyping cocktail that permitted simultaneous identification of all 20 chromosome pairs. We applied these karyotyping tools to wild soybean, G. soja Sieb. and Zucc., which represents a large gene pool of potentially agronomically valuable traits. These studies led to the identification and characterization of a reciprocal chromosome translocation between chromosomes 11 and 13 in two accessions of wild soybean. The data confirm that this translocation is widespread in G. soja accessions and likely accounts for the semi-sterility found in some G. soja by G. max crosses.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20421607">http://www.ncbi.nlm.nih.gov/pubmed/20421607</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant J.(2010).63(1):86-99. doi:10.1111/J.1365-313X.2010.04222.X; PMID:20408999</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">An integrated transcriptome atlas of the crop model Glycine max, and its use in comparative analyses in plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Andrew Farmer, Trupti Joshi, Kaori Takahashi, Raymond J Langley, Levi D Franklin, Ji He, Dong Xu, Gregory May, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; C.S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA. libaultm@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean (Glycine max L.) is a major crop providing an important source of protein and oil, which can also be converted into biodiesel. A major milestone in soybean research was the recent sequencing of its genome. The sequence predicts 69,145 putative soybean genes, with 46,430 predicted with high confidence. In order to examine the expression of these genes, we utilized the Illumina Solexa platform to sequence cDNA derived from 14 conditions (tissues). The result is a searchable soybean gene expression atlas accessible through a browser (http://digbio.missouri.edu/soybean_atlas). The data provide experimental support for the transcription of 55,616 annotated genes and also demonstrate that 13,529 annotated soybean genes are putative pseudogenes, and 1736 currently unannotated sequences are transcribed. An analysis of this atlas reveals strong differences in gene expression patterns between different tissues, especially between root and aerial organs, but also reveals similarities between gene expression in other tissues, such as flower and leaf organs. In order to demonstrate the full utility of the atlas, we investigated the expression patterns of genes implicated in nodulation, and also transcription factors, using both the Solexa sequence data and large-scale qRT-PCR. The availability of the soybean gene expression atlas allowed a comparison with gene expression documented in the two model legume species, Medicago truncatula and Lotus japonicus, as well as data available for Arabidopsis thaliana, facilitating both basic and applied aspects of soybean research.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20408999">http://www.ncbi.nlm.nih.gov/pubmed/20408999</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Signal Behav(2010).5(4):419-21. doi:; PMID:20339317</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Soybean root hairs: a valuable system to investigate plant biology at the cellular level.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Andrew Farmer, Laurent Brechenmacher, Gregory D May, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; University of Missouri; Columbia; MO; USA. libaultm@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plant organs and tissues are composed of many differentiated cell types. Most functional genomic studies sample whole tissues, which dilutes the signals that may arise from individual cells within the population. The result is an averaging of the cellular response. In order to overcome these issues of "signal dilution", methods are needed to allow the full application of modern functional genomics tools to the study of a single differentiated plant cell type. In order to address this need, we developed a method for the isolation of soybean root hair cells, a single epidermal cell type, in sufficient quantities and purity to perform a variety of functional genomic analyses. As a first demonstration of the potential of soybean root hair cells to study plant systems biology, we compared the root hair transcriptome and proteome.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20339317">http://www.ncbi.nlm.nih.gov/pubmed/20339317</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant J.(2010).62(5):852-64. doi:10.1111/J.1365-313X.2010.04201.X; PMID:20230508</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A member of the highly conserved FWL (tomato FW2.2-like) gene family is essential for soybean nodule organogenesis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Xue-Cheng Zhang, Manjula Govindarajulu, Jing Qiu, Yee T Ong, Laurent Brechenmacher, R Howard Berg, Andrea Hurley-Sommer, Christopher G Taylor, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; Division of Biochemistry; C.S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>A soybean homolog of the tomato FW2.2 gene, here named GmFWL1 (Glycine max FW2.2-like 1), was found to respond strongly to inoculation with the nitrogen-fixing symbiotic bacterium Bradyrhizobium japonicum. In tomato, the FW2.2 gene is hypothesized to control 30% of the variance in fruit weight by negatively regulating cell division. In the present study, the induction of GmFWL1 expression in root hair cells and nodules in response to B. japonicum inoculation was documented using quantitative RT-PCR and transcriptional fusions to both beta-glucuronidase (GUS) and green fluorescent protein (GFP). RNAi-mediated silencing of GmFWL1 expression resulted in a significant reduction in nodule number, with a concomitant reduction in nuclear size and changes in chromatin structure. The reduction in nuclear size is probably due to a change in DNA heterochromatinization, as the ploidy level of wild-type and RNAi-silenced nodule cells was similar. GmFWL1 was localized to the plasma membrane. The data suggest that GmFWL1 probably acts indirectly, perhaps through a cellular cascade, to affect chromatin structure/nuclei architecture. As previously proposed in tomato, this function may be a result of effects on plant cell division.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20230508">http://www.ncbi.nlm.nih.gov/pubmed/20230508</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Bioinformatics(2010).11 Suppl 1():S14. doi:10.1186/1471-2105-11-S1-S14; PMID:20122185</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Prediction of novel miRNAs and associated target genes in Glycine max.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Trupti Joshi, Zhe Yan, Marc Libault, Dong-Hoon Jeong, Sunhee Park, Pamela J Green, D Janine Sherrier, Andrew Farmer, Greg May, Blake C Meyers, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Digital Biology Laboratory; Computer Science Department and Christopher S; Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA. joshitr@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Small non-coding RNAs (21 to 24 nucleotides) regulate a number of developmental processes in plants and animals by silencing genes using multiple mechanisms. Among these, the most conserved classes are microRNAs (miRNAs) and small interfering RNAs (siRNAs), both of which are produced by RNase III-like enzymes called Dicers. Many plant miRNAs play critical roles in nutrient homeostasis, developmental processes, abiotic stress and pathogen responses. Currently, only 70 miRNA have been identified in soybean.We utilized Illumina's SBS sequencing technology to generate high-quality small RNA (sRNA) data from four soybean (Glycine max) tissues, including root, seed, flower, and nodules, to expand the collection of currently known soybean miRNAs. We developed a bioinformatics pipeline using in-house scripts and publicly available structure prediction tools to differentiate the authentic mature miRNA sequences from other sRNAs and short RNA fragments represented in the public sequencing data.The combined sequencing and bioinformatics analyses identified 129 miRNAs based on hairpin secondary structure features in the predicted precursors. Out of these, 42 miRNAs matched known miRNAs in soybean or other species, while 87 novel miRNAs were identified. We also predicted the putative target genes of all identified miRNAs with computational methods and verified the predicted cleavage sites in vivo for a subset of these targets using the 5' RACE method. Finally, we also studied the relationship between the abundance of miRNA and that of the respective target genes by comparison to Solexa cDNA sequencing data.Our study significantly increased the number of miRNAs known to be expressed in soybean. The bioinformatics analysis provided insight on regulation patterns between the miRNAs and their predicted target genes expression. We also deposited the data in a soybean genome browser based on the UCSC Genome Browser architecture. Using the browser, we annotated the soybean data with miRNA sequences from four tissues and cDNA sequencing data. Overlaying these two datasets in the browser allows researchers to analyze the miRNA expression levels relative to that of the associated target genes. The browser can be accessed at http://digbio.missouri.edu/soybean_mirna/.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20122185">http://www.ncbi.nlm.nih.gov/pubmed/20122185</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Plant Biol.(2010).10():14. doi:10.1186/1471-2229-10-14; PMID:20082720</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">SoyDB: a knowledge database of soybean transcription factors.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zheng Wang, Marc Libault, Trupti Joshi, Babu Valliyodan, Henry T Nguyen, Dong Xu, Gary Stacey, Jianlin Cheng</b>
      </span>
      <br/>
      <span>Affiliation: Computer Science Department; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Transcription factors play the crucial rule of regulating gene expression and influence almost all biological processes. Systematically identifying and annotating transcription factors can greatly aid further understanding their functions and mechanisms. In this article, we present SoyDB, a user friendly database containing comprehensive knowledge of soybean transcription factors.The soybean genome was recently sequenced by the Department of Energy-Joint Genome Institute (DOE-JGI) and is publicly available. Mining of this sequence identified 5,671 soybean genes as putative transcription factors. These genes were comprehensively annotated as an aid to the soybean research community. We developed SoyDB - a knowledge database for all the transcription factors in the soybean genome. The database contains protein sequences, predicted tertiary structures, putative DNA binding sites, domains, homologous templates in the Protein Data Bank (PDB), protein family classifications, multiple sequence alignments, consensus protein sequence motifs, web logo of each family, and web links to the soybean transcription factor database PlantTFDB, known EST sequences, and other general protein databases including Swiss-Prot, Gene Ontology, KEGG, EMBL, TAIR, InterPro, SMART, PROSITE, NCBI, and Pfam. The database can be accessed via an interactive and convenient web server, which supports full-text search, PSI-BLAST sequence search, database browsing by protein family, and automatic classification of a new protein sequence into one of 64 annotated transcription factor families by hidden Markov models.A comprehensive soybean transcription factor database was constructed and made publicly accessible at http://casp.rnet.missouri.edu/soydb/.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/20082720">http://www.ncbi.nlm.nih.gov/pubmed/20082720</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Nature(2010).463(7278):178-83. doi:10.1038/Nature08670; PMID:20075913</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genome sequence of the palaeopolyploid soybean.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jeremy Schmutz, Steven B Cannon, Jessica Schlueter, Jianxin Ma, Therese Mitros, William Nelson, David L Hyten, Qijian Song, Jay J Thelen, Jianlin Cheng, Dong Xu, Uffe Hellsten, Gregory D May, Yeisoo Yu, Tetsuya Sakurai, Taishi Umezawa, Madan K Bhattacharyya, Devinder Sandhu, Babu Valliyodan, Erika Lindquist, Myron Peto, David Grant, Shengqiang Shu, David Goodstein, Kerrie Barry, Montona Futrell-Griggs, Brian Abernathy, Jianchang Du, Zhixi Tian, Liucun Zhu, Navdeep Gill, Trupti Joshi, Marc Libault, Anand Sethuraman, Xue-Cheng Zhang, Kazuo Shinozaki, Henry T Nguyen, Rod A Wing, Perry Cregan, James Specht, Jane Grimwood, Dan Rokhsar, Gary Stacey, Randy C Shoemaker, Scott A Jackson</b>
      </span>
      <br/>
      <span>Affiliation: HudsonAlpha Genome Sequencing Center; 601 Genome Way; Huntsville; Alabama 35806; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean (Glycine max) is one of the most important crop plants for seed protein and oil content, and for its capacity to fix atmospheric nitrogen through symbioses with soil-borne microorganisms. We sequenced the 1.1-gigabase genome by a whole-genome shotgun approach and integrated it with physical and high-density genetic maps to create a chromosome-scale draft sequence assembly. We predict 46,430 protein-coding genes, 70% more than Arabidopsis and similar to the poplar genome which, like soybean, is an ancient polyploid (palaeopolyploid). About 78% of the predicted genes occur in chromosome ends, which comprise less than one-half of the genome but account for nearly all of the genetic recombination. Genome duplications occurred at approximately 59 and 13 million years ago, resulting in a highly duplicated genome with nearly 75% of the genes present in multiple copies. The two duplication events were followed by gene diversification and loss, and numerous chromosome rearrangements. An accurate soybean genome sequence will facilitate the identification of the genetic basis of many soybean traits, and accelerate the creation of improved soybean varieties.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v463/n7278/Nature08670.html">http://www.nature.com/journal/v463/n7278/Nature08670.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2010).152(2):541-52. doi:10.1104/Pp.109.148379; PMID:19933387</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Complete transcriptome of the soybean root hair cell, a single-cell model, and its alteration in response to Bradyrhizobium japonicum infection.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Andrew Farmer, Laurent Brechenmacher, Jenny Drnevich, Raymond J Langley, Damla D Bilgin, Osman Radwan, David J Neece, Steven J Clough, Gregory D May, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; CS Bond Life Sciences Center; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Nodulation is the result of a mutualistic interaction between legumes and symbiotic soil bacteria (e.g. soybean [Glycine max] and Bradyrhizobium japonicum) initiated by the infection of plant root hair cells by the symbiont. Fewer than 20 plant genes involved in the nodulation process have been functionally characterized. Considering the complexity of the symbiosis, significantly more genes are likely involved. To identify genes involved in root hair cell infection, we performed a large-scale transcriptome analysis of B. japonicum-inoculated and mock-inoculated soybean root hairs using three different technologies: microarray hybridization, Illumina sequencing, and quantitative real-time reverse transcription-polymerase chain reaction. Together, a total of 1,973 soybean genes were differentially expressed with high significance during root hair infection, including orthologs of previously characterized root hair infection-related genes such as NFR5 and NIN. The regulation of 60 genes was confirmed by quantitative real-time reverse transcription-polymerase chain reaction. Our analysis also highlighted changes in the expression pattern of some homeologous and tandemly duplicated soybean genes, supporting their rapid specialization.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19933387">http://www.ncbi.nlm.nih.gov/pubmed/19933387</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Signal Behav(2009).4(9):818-23. doi:; PMID:19847106</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genetic analysis of ethylene regulation of legume nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Peter M Gresshoff, Dasharath Lohar, Pick-Kuen Chan, Bandana Biswas, Qunyi Jiang, Dugald Reid, Brett Ferguson, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Australian Research Council Centre of Excellence for Integrative Legume Research; The University of Queensland; St. Lucia; Brisbane; Australia. p.gresshoff@uq.edu.au</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The gaseous hormone ethylene has multiple roles in plant development and responses to external cues. Among these is the regulation of "Rhizobium"-induced nodulation in legumes. Extensive descriptive literature exists, but has been expanded to allow more mechanistic analysis through the application of genetics. Both mutants and transgenics displaying ethylene insensitivity have now been described, suggesting an intimate interplay of ethylene response, plant development and nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19847106">http://www.ncbi.nlm.nih.gov/pubmed/19847106</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Cell Physiol.(2009).50(11):1923-32. doi:10.1093/Pcp/Pcp136; PMID:19808809</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Arabidopsis OPT6 is an oligopeptide transporter with exceptionally broad substrate specificity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Sharon Pike, Ami Patel, Gary Stacey, Walter Gassmann</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211-7310; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Oligopeptide transporters (OPTs) are found in fungi, bacteria and plants. The nine Arabidopsis thaliana OPT genes are expressed mainly in the vasculature and are thought to transport tetra- and pentapeptides, and peptide-like substrates such as glutathione. Expression of AtOPT6 in Xenopus laevis oocytes demonstrated that AtOPT6 transports many tetra- and pentapeptides. In addition, AtOPT6 transported reduced glutathione (GSH), a tripeptide, but not oxidized glutathione (GSSG). Recent data showed that Candida albicans OPTs can transport peptides up to eight amino acids in length. AtOPT6 transported mammalian signaling peptides up to 10 amino acids in length and, in addition, known plant development- and nematode pathogenesis-associated peptides up to 13 amino acids long. AtOPT6 displayed high affinity for penta- and dodecapeptides, but low affinity for GSH. In comparison the Saccharomyces cerevisiae ScOPT1 was incapable of transporting any of the longer peptides tested. These data demonstrate the necessity of experimentally determining substrate specificity of individual OPTs, and lay a foundation for structure/function studies. Characterization of the AtOPT6 substrate range provides a basis for investigating the possible physiological function of AtOPT6 in peptide signaling and thiol transport in response to stress.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19808809">http://www.ncbi.nlm.nih.gov/pubmed/19808809</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2009).151(3):1207-20. doi:10.1104/Pp.109.144030; PMID:19755542</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Large-scale analysis of putative soybean regulatory gene expression identifies a Myb gene involved in soybean nodule development.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Trupti Joshi, Kaori Takahashi, Andrea Hurley-Sommer, Kari Puricelli, Sean Blake, Richard E Finger, Christopher G Taylor, Dong Xu, Henry T Nguyen, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; C.S. Bond Life Sciences Center; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Nodulation is the result of a symbiosis between legumes and rhizobial bacteria in soil. This symbiosis is mutually beneficial, with the bacteria providing a source of nitrogen to the host while the plant supplies carbon to the symbiont. Nodule development is a complex process that is tightly regulated in the host plant cell through networks of gene expression. In order to examine this regulation in detail, a library of quantitative reverse transcription-polymerase chain reaction primer sets was developed for a large number of soybean (Glycine max) putative regulatory genes available in the current expressed sequence tag collection. This library contained primers specific to soybean transcription factor genes as well as genes involved in chromatin modification and translational regulation. Using this library, we analyzed the expression of this gene set during nodule development. A large number of genes were found to be differentially expressed, especially at the later stages of nodule development when active nitrogen fixation was occurring. Expression of these putative regulatory genes was also analyzed in response to the addition of nitrate as a nitrogen source. This comparative analysis identified genes that may be specifically involved in nitrogen assimilation, metabolism, and the maintenance of active nodules. To address this possibility, the expression of one such candidate was studied in more detail by expressing in soybean roots promoter beta-glucuronidase and green fluorescent protein fusions. This gene, named Control of Nodule Development (CND), encoded a Myb transcription factor gene. When the CND gene was silenced, nodulation was reduced. These results, associated with a strong expression of the CND gene in the vascular tissues, suggest a role for CND in controlling soybean nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19755542">http://www.ncbi.nlm.nih.gov/pubmed/19755542</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2009).151(3):991-1001. doi:10.1104/Pp.109.144105; PMID:19726573</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Legume transcription factor genes: what makes legumes so special?</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Trupti Joshi, Vagner A Benedito, Dong Xu, Michael K Udvardi, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/></p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19726573">http://www.ncbi.nlm.nih.gov/pubmed/19726573</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Signal Behav(2008).3(10):831-3. doi:; PMID:19704513</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Chitin signaling and plant disease resistance.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jinrong Wan, Xue-Cheng Zhang, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; University of Missouri-Columbia; Columbia; Missouri USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin, a polymer of N-acetyl-D-glucosamine, is a component of the fungal cell wall and is not found in plants. Plant cells are equipped with chitin degrading enzymes to digest fungal cell walls and are capable of perceiving chitin fragments (chitooligosaccharides) released from fungal cell walls during fungal infection. Chitin recognition results in the activation of defense signaling pathways. Although chitin is a well recognized pathogen-associated molecular pattern (PAMP), little is known about the molecular mechanism of chitin signaling. Recent studies identified a number of critical components in the chitin-elicited signaling pathway including a potential receptor, MAPK cascade and transcription factor network. Interestingly, the chitin signaling pathway overlaps with the phytobacterial flagellin-and EF-Tu-elicited signaling pathways, suggesting that plant cells may perceive different PAMPs from various pathogens via specialized receptors and then utilize a conserved, common downstream pathway to mediate disease resistance. Given the fact that fungal pathogens are major problems in many agricultural systems, research on chitin signaling could have significance to sustainable agriculture and biofuel and biomaterial production.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19704513">http://www.ncbi.nlm.nih.gov/pubmed/19704513</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Evol. Biol.(2009).9():183. doi:10.1186/1471-2148-9-183; PMID:19650916</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Evolutionary genomics of LysM genes in land plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Xue-Cheng Zhang, Steven B Cannon, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and National Center for Soybean Biotechnology; University of Missouri; Columbia; MO 65211; USA. zhangxuec@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The ubiquitous LysM motif recognizes peptidoglycan, chitooligosaccharides (chitin) and, presumably, other structurally-related oligosaccharides. LysM-containing proteins were first shown to be involved in bacterial cell wall degradation and, more recently, were implicated in perceiving chitin (one of the established pathogen-associated molecular patterns) and lipo-chitin (nodulation factors) in flowering plants. However, the majority of LysM genes in plants remain functionally uncharacterized and the evolutionary history of complex LysM genes remains elusive.We show that LysM-containing proteins display a wide range of complex domain architectures. However, only a simple core architecture is conserved across kingdoms. Each individual kingdom appears to have evolved a distinct array of domain architectures. We show that early plant lineages acquired four characteristic architectures and progressively lost several primitive architectures. We report plant LysM phylogenies and associated gene, protein and genomic features, and infer the relative timing of duplications of LYK genes.We report a domain architecture catalogue of LysM proteins across all kingdoms. The unique pattern of LysM protein domain architectures indicates the presence of distinctive evolutionary paths in individual kingdoms. We describe a comparative and evolutionary genomics study of LysM genes in plant kingdom. One of the two groups of tandemly arrayed plant LYK genes likely resulted from an ancient genome duplication followed by local genomic rearrangement, while the origin of the other groups of tandemly arrayed LYK genes remains obscure. Given the fact that no animal LysM motif-containing genes have been functionally characterized, this study provides clues to functional characterization of plant LysM genes and is also informative with regard to evolutionary and functional studies of animal LysM genes.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19650916">http://www.ncbi.nlm.nih.gov/pubmed/19650916</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2009).151(3):1167-74. doi:10.1104/Pp.109.137935; PMID:19605552</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Molecular and chromosomal evidence for allopolyploidy in soybean.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Navdeep Gill, Seth Findley, Jason G Walling, Christian Hans, Jianxin Ma, Jeff Doyle, Gary Stacey, Scott A Jackson</b>
      </span>
      <br/>
      <span>Affiliation: Department of Agronomy; Purdue University; West Lafayette; Indiana 47907; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Recent studies have documented that the soybean (Glycine max) genome has undergone two rounds of large-scale genome and/or segmental duplication. To shed light on the timing and nature of these duplication events, we characterized and analyzed two subfamilies of high-copy centromeric satellite repeats, CentGm-1 and CentGm-2, using a combination of computational and molecular cytogenetic approaches. These two subfamilies of satellite repeats mark distinct subsets of soybean centromeres and, in at least one case, a pair of homologs, suggesting their origins from an allopolyploid event. The satellite monomers of each subfamily are arranged in large tandem arrays, and intermingled monomers of the two subfamilies were not detected by fluorescence in situ hybridization on extended DNA fibers nor at the sequence level. This indicates that there has been little recombination and homogenization of satellite DNA between these two sets of centromeres. These satellite repeats are also present in Glycine soja, the proposed wild progenitor of soybean, but could not be detected in any other relatives of soybean examined in this study, suggesting the rapid divergence of the centromeric satellite DNA within the Glycine genus. Together, these observations provide direct evidence, at molecular and chromosomal levels, in support of the hypothesis that the soybean genome has experienced a recent allopolyploidization event.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19605552">http://www.ncbi.nlm.nih.gov/pubmed/19605552</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>FEBS Lett.(2009).583(15):2521-6. doi:10.1016/J.Febslet.2009.07.007; PMID:19596005</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Touch induces ATP release in Arabidopsis roots that is modulated by the heterotrimeric G-protein complex.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Ravisha R Weerasinghe, Sarah J Swanson, Seiko F Okada, Michele B Garrett, Sung-Yong Kim, Gary Stacey, Richard C Boucher, Simon Gilroy, Alan M Jones</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biology; University of North Carolina; Chapel Hill; NC 27599; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Amongst the many stimuli orienting the growth of plant roots, of critical importance are the touch signals generated as roots explore the mechanically complex soil environment. However, the molecular mechanisms behind these sensory events remain poorly defined. We report an impaired obstacle-avoiding response of roots in Arabidopsis lacking a heterotrimeric G-protein. Obstacle avoidance may utilize a touch-induced release of ATP to the extracellular space. While sequential touch stimulation revealed a strong refractory period for ATP release in response to mechano-stimulation in wild-type plants, the refractory period in mutants was attenuated, resulting in extracellular ATP accumulation. We propose that ATP acts as an extracellular signal released by mechano-stimulation and that the G-protein complex is needed for fine-tuning this response.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19596005">http://www.ncbi.nlm.nih.gov/pubmed/19596005</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Ann. Bot.(2009).104(2):277-85. doi:10.1093/Aob/Mcp132; PMID:19505874</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Ethylene insensitivity conferred by a mutated Arabidopsis ethylene receptor gene alters nodulation in transgenic Lotus japonicus.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Dasharath Lohar, Jiri Stiller, Jason Kam, Gary Stacey, Peter M Gresshoff</b>
      </span>
      <br/>
      <span>Affiliation: ARC Centre of Excellence for Integrative Legume Research; The University of Queensland; St Lucia; Brisbane; QLD 4072; Australia.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Transgenics are used to demonstrate a causal relationship between ethylene insensitivity of a seedling legume plant, the level of ethylene receptor gene expression, lateral root growth and Mesorhizobium loti-induced nodule initiation.Lotus japonicus plants expressing the dominant etr1-1 allele of the Arabidopsis thaliana gene encoding a well-characterized mutated ethylene receptor were created by stable Agrobacterium tumefaciens transformation. Single insertion, homozygous lines were characterized for symbiotic properties.Transgenic plants were ethylene insensitive as judged by the lack of the 'Triple Response', and their continued ability to grow and nodulate in the presence of inhibitory concentrations of ACC (1-aminocyclopropane-1-carboxylic acid; an ethylene precursor). Transgenic plants with high insensitivity to ACC had significantly fewer lateral roots and exhibited increased nodulation while showing no altered nitrate sensitivity or lack of systemic autoregulation. Whereas ACC-insensitive shoot growth and nodulation were observed in transformants, root growth was inhibited similarly to the wild type. Increased nodulation was caused by increased infection and a seven-fold increase in nodules developing between xylem poles. Bacteroid numbers per symbiosome increased about 1.7-fold in ethylene-insensitive plants.The study further demonstrates multiple roles for ethylene in nodule initiation by influencing root cell infections and radial positioning, independent of autoregulation and nitrate inhibition of nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505874">http://www.ncbi.nlm.nih.gov/pubmed/19505874</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Plant Biol.(2009).9():46. doi:10.1186/1471-2229-9-46; PMID:19397807</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Generation of Phaseolus vulgaris ESTs and investigation of their regulation upon Uromyces appendiculatus infection.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Sandra Thibivilliers, Trupti Joshi, Kimberly B Campbell, Brian Scheffler, Dong Xu, Bret Cooper, Henry T Nguyen, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Center for Sustainable Energy; Divisions of Plant Sciences and Biochemistry; University of Missouri; Columbia; MO 65211; USA. st5y7@mizzou.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Phaseolus vulgaris (common bean) is the second most important legume crop in the world after soybean. Consequently, yield losses due to fungal infection, like Uromyces appendiculatus (bean rust), have strong consequences. Several resistant genes were identified that confer resistance to bean rust infection. However, the downstream genes and mechanisms involved in bean resistance to infection are poorly characterized.A subtractive bean cDNA library composed of 10,581 unisequences was constructed and enriched in sequences regulated by either bean rust race 41, a virulent strain, or race 49, an avirulent strain on cultivar Early Gallatin carrying the resistance gene Ur-4. The construction of this library allowed the identification of 6,202 new bean ESTs, significantly adding to the available sequences for this plant. Regulation of selected bean genes in response to bean rust infection was confirmed by qRT-PCR. Plant gene expression was similar for both race 41 and 49 during the first 48 hours of the infection process but varied significantly at the later time points (72-96 hours after inoculation) mainly due to the presence of the Avr4 gene in the race 49 leading to a hypersensitive response in the bean plants. A biphasic pattern of gene expression was observed for several genes regulated in response to fungal infection.The enrichment of the public database with over 6,000 bean ESTs significantly adds to the genomic resources available for this important crop plant. The analysis of these genes in response to bean rust infection provides a foundation for further studies of the mechanism of fungal disease resistance. The expression pattern of 90 bean genes upon rust infection shares several features with other legumes infected by biotrophic fungi. This finding suggests that the P. vulgaris-U. appendiculatus pathosystem could serve as a model to explore legume-rust interaction.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19397807">http://www.ncbi.nlm.nih.gov/pubmed/19397807</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Plant Physiol.(2009).166(13):1453-9. doi:10.1016/J.Jplph.2009.02.009; PMID:19342121</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Two MATE proteins play a role in iron efficiency in soybean.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Elizabeth E Rogers, Xiaolei Wu, Gary Stacey, Henry T Nguyen</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biochemistry; University of Missouri; Columbia; MO 65211; USA. elizabeth.rogers@ars.usda.gov</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Iron is a necessary but often limiting nutrient for plant growth and development. Soybeans grown on the high-pH calcareous soils are especially prone to developing iron deficiency chlorosis and suffering the resultant yield losses. Once iron is transported into the root, it must be translocated from the root to the shoot where it is needed for photosynthesis and other processes. Previous work has indicated that iron is likely to move through the xylem as ferric citrate. In Arabidopsis thaliana, citrate is effluxed into the xylem by the ferric reductase defective3 (FRD3) protein. Here, we present the identification and characterization of two soybean genes, GmFRD3a and GmFRD3b, with similar sequence and function to AtFRD3. The expression of both GmFRD3a and GmFRD3b is induced by iron deficiency in the iron-efficient reference cultivar Williams 82. GmFRD3b, but not GmFRD3a, is expressed at higher levels in the iron-efficient cultivar Clark than in its iron-inefficient near isogenic line iso-Clark (iso), likely accounting for the higher xylem citrate levels in Clark. Increased xylem citrate levels lead to increased solubility of ferric iron in Clark xylem exudate as compared to iso-Clark exudate. These results support the hypothesis that high xylem citrate levels are needed for efficient root to shoot translocation of iron. Along with efficient ferric chelate reductase activity and root iron uptake activity, high expression levels of FRD3 genes are also proposed as a target for future iron efficiency breeding projects.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19342121">http://www.ncbi.nlm.nih.gov/pubmed/19342121</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2009).149(2):994-1004. doi:10.1104/Pp.108.128728; PMID:19036836</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">GS52 ecto-apyrase plays a critical role during soybean nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Manjula Govindarajulu, Sung-Yong Kim, Marc Libault, R Howard Berg, Kiwamu Tanaka, Gary Stacey, Christopher G Taylor</b>
      </span>
      <br/>
      <span>Affiliation: Donald Danforth Plant Science Center; St. Louis; Missouri 63132; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Apyrases are non-energy-coupled nucleotide phosphohydrolases that hydrolyze nucleoside triphosphates and nucleoside diphosphates to nucleoside monophosphates and orthophosphates. GS52, a soybean (Glycine soja) ecto-apyrase, was previously shown to be induced very early in response to inoculation with the symbiotic bacterium Bradyrhizobium japonicum. Overexpression of the GS52 ecto-apyrase in Lotus japonicus increased the level of rhizobial infection and enhanced nodulation. These data suggest a critical role for the GS52 ecto-apyrase during nodulation. To further investigate the role of GS52 during nodulation, we used RNA interference to silence GS52 expression in soybean (Glycine max) roots using Agrobacterium rhizogenes-mediated root transformation. Transcript levels of GS52 were significantly reduced in GS52 silenced roots and these roots exhibited reduced numbers of mature nodules. Development of the nodule primordium and subsequent nodule maturation was significantly suppressed in GS52 silenced roots. Transmission electron micrographs of GS52 silenced root nodules showed that early senescence and infected cortical cells were devoid of symbiosome-containing bacteroids. Application of exogenous adenosine diphosphate to silenced GS52 roots restored nodule development. Restored nodules contained bacteroids, thus indicating that extracellular adenosine diphosphate is important during nodulation. These results clearly suggest that GS52 ecto-apyrase catalytic activity is critical for the early B. japonicum infection process, initiation of nodule primordium development, and subsequent nodule organogenesis in soybean.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19036836">http://www.ncbi.nlm.nih.gov/pubmed/19036836</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2009).149(2):670-82. doi:10.1104/Pp.108.131649; PMID:19036831</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Establishment of a protein reference map for soybean root hair cells.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Laurent Brechenmacher, Joohyun Lee, Sherri Sachdev, Zhao Song, Tran Hong Nha Nguyen, Trupti Joshi, Nathan Oehrle, Marc Libault, Brian Mooney, Dong Xu, Bret Cooper, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Division of Plant Sciences; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Root hairs are single tubular cells formed from the differentiation of epidermal cells on roots. They are involved in water and nutrient uptake and represent the infection site on leguminous roots by rhizobia, soil bacteria that establish a nitrogen-fixing symbiosis. Root hairs develop by polar cell expansion or tip growth, a unique mode of plant growth shared only with pollen tubes. A more complete characterization of root hair cell biology will lead to a better understanding of tip growth, the rhizobial infection process, and also lead to improvements in plant water and nutrient uptake. We analyzed the proteome of isolated soybean (Glycine max) root hair cells using two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) and shotgun proteomics (1D-PAGE-liquid chromatography and multidimensional protein identification technology) approaches. Soybean was selected for this study due to its agronomic importance and its root size. The resulting soybean root hair proteome reference map identified 1,492 different proteins. 2D-PAGE followed by mass spectrometry identified 527 proteins from total cell contents. A complementary shotgun analysis identified 1,134 total proteins, including 443 proteins that were specific to the microsomal fraction. Only 169 proteins were identified by the 2D-PAGE and shotgun methods, which highlights the advantage of using both methods. The proteins identified are involved not only in basic cell metabolism but also in functions more specific to the single root hair cell, including water and nutrient uptake, vesicle trafficking, and hormone and secondary metabolism. The data presented provide useful insight into the metabolic activities of a single, differentiated plant cell type.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/19036831">http://www.ncbi.nlm.nih.gov/pubmed/19036831</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Planta(2009).229(2):279-89. doi:10.1007/S00425-008-0827-9; PMID:18855007</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Establishment of a soybean (Glycine max Merr. L) transposon-based mutagenesis repository.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Melanie Mathieu, Elizabeth K Winters, Fanming Kong, Jinrong Wan, Shaoxing Wang, Helene Eckert, Diane Luth, Margie Paz, Christopher Donovan, Zhanyuan Zhang, David Somers, Kan Wang, Henry Nguyen, Randy C Shoemaker, Gary Stacey, Tom Clemente</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Division of Plant Sciences; Life Sciences Center; Department of Molecular Microbiology; University of Missouri; Rollins Road; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Soybean is a major crop species providing valuable feedstock for food, feed and biofuel. In recent years, considerable progress has been made in developing genomic resources for soybean, including on-going efforts to sequence the genome. These efforts have identified a large number of soybean genes, most with unknown function. Therefore, a major research priority is determining the function of these genes, especially those involved in agronomic performance and seed traits. One means to study gene function is through mutagenesis and the study of the resulting phenotypes. Transposon-tagging has been used successfully in both model and crop plants to support studies of gene function. In this report, we describe efforts to generate a transposon-based mutant collection of soybean. The Ds transposon system was used to create activation-tagging, gene and enhancer trap elements. Currently, the repository houses approximately 900 soybean events, with flanking sequence data derived from 200 of these events. Analysis of the insertions revealed approximately 70% disrupted known genes, with the majority matching sequences derived from either Glycine max or Medicago truncatula sequences. Among the mutants generated, one resulted in male-sterility and was shown to disrupt the strictosidine synthase gene. This example clearly demonstrates that it is possible to disrupt soybean gene function by insertional mutagenesis and to derive useful mutants by this approach in spite of the tetraploid nature of the soybean genome.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18855007">http://www.ncbi.nlm.nih.gov/pubmed/18855007</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant J.(2009).57(1):171-83. doi:10.1111/J.1365-313X.2008.03676.X; PMID:18786000</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Flavones and flavonols play distinct critical roles during nodulation of Medicago truncatula by Sinorhizobium meliloti.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Juan Zhang, Senthil Subramanian, Gary Stacey, Oliver Yu</b>
      </span>
      <br/>
      <span>Affiliation: Donald Danforth Plant Science Center; 975 N. Warson Road; Saint Louis; MO 63132; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Flavonoids play critical roles in legume-rhizobium symbiosis. However, the role of individual flavonoid compounds in this process has not yet been clearly established. We silenced different flavonoid-biosynthesis enzymes to generate transgenic Medicago truncatula roots with different flavonoid profiles. Silencing of chalcone synthase, the key entry-point enzyme for flavonoid biosynthesis led to flavonoid-deficient roots. Silencing of isoflavone synthase and flavone synthase led to roots deficient for a subset of flavonoids, isoflavonoids (formononetin and biochanin A) and flavones (7,4'-dihydroxyflavone), respectively. When tested for nodulation by Sinorhizobium meliloti, flavonoid-deficient roots had a near complete loss of nodulation, whereas flavone-deficient roots had reduced nodulation. Isoflavone-deficient roots nodulated normally, suggesting that isoflavones might not play a critical role in M. truncatula nodulation, even though they are the most abundant root flavonoids. Supplementation of flavone-deficient roots with 7, 4'-dihydroxyflavone, a major inducer of S. meliloti nod genes, completely restored nodulation. However, the same treatment did not restore nodulation in flavonoid-deficient roots, suggesting that other non-nod gene-inducing flavonoid compounds are also critical to nodulation. Supplementation of roots with the flavonol kaempferol (an inhibitor of auxin transport), in combination with the use of flavone pre-treated S. meliloti cells, completely restored nodulation in flavonoid-deficient roots. In addition, S. meliloti cells constitutively producing Nod factors were able to nodulate flavone-deficient roots, but not flavonoid-deficient roots. These observations indicated that flavones might act as internal inducers of rhizobial nod genes, and that flavonols might act as auxin transport regulators during nodulation. Both these roles of flavonoids appear critical for symbiosis in M. truncatula.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18786000">http://www.ncbi.nlm.nih.gov/pubmed/18786000</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Cell Proteomics(2009).8(1):19-31. doi:10.1074/Mcp.M800156-Mcp200; PMID:18755735</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Quantitative proteomic analysis of bean plants infected by a virulent and avirulent obligate rust fungus.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Joohyun Lee, Jian Feng, Kimberly B Campbell, Brian E Scheffler, Wesley M Garrett, Sandra Thibivilliers, Gary Stacey, Daniel Q Naiman, Mark L Tucker, M A Pastor-Corrales, Bret Cooper</b>
      </span>
      <br/>
      <span>Affiliation: Soybean Genomics and Improvement Laboratory; United States Department of Agriculture; Agricultural Research Service; Beltsville; Maryland 20705; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plants appear to have two types of active defenses, a broad-spectrum basal system and a system controlled by R-genes providing stronger resistance to some pathogens that break the basal defense. However, it is unknown if the systems are separate entities. Therefore, we analyzed proteins from leaves of the dry bean crop plant Phaseolus vulgaris using a high-throughput liquid chromatography tandem mass spectrometry method. By statistically comparing the amounts of proteins detected in a single plant variety that is susceptible or resistant to infection, depending on the strains of a rust fungus introduced, we defined basal and R-gene-mediated plant defenses at the proteomic level. The data reveal that some basal defense proteins are potential regulators of a strong defense weakened by the fungus and that the R-gene modulates proteins similar to those in the basal system. The results satisfy a new model whereby R-genes are part of the basal system and repair disabled defenses to reinstate strong resistance.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18755735">http://www.ncbi.nlm.nih.gov/pubmed/18755735</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Bacteriol.(2008).190(20):6697-705. doi:10.1128/Jb.00543-08; PMID:18689488</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Whole-genome transcriptional profiling of Bradyrhizobium japonicum during chemoautotrophic growth.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>William L Franck, Woo-Suk Chang, Jing Qiu, Masayuki Sugawara, Michael J Sadowsky, Stephanie A Smith, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Division of Plant Sciences; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Bradyrhizobium japonicum is a facultative chemoautotroph capable of utilizing hydrogen gas as an electron donor in a respiratory chain terminated by oxygen to provide energy for cellular processes and carbon dioxide assimilation via a reductive pentose phosphate pathway. A transcriptomic analysis of B. japonicum cultured chemoautotrophically identified 1,485 transcripts, representing 17.5% of the genome, as differentially expressed when compared to heterotrophic cultures. Genetic determinants required for hydrogen utilization and carbon fixation, including the uptake hydrogenase system and components of the Calvin-Benson-Bassham cycle, were strongly induced in chemoautotrophically cultured cells. A putative isocitrate lyase (aceA; blr2455) was among the most strongly upregulated genes, suggesting a role for the glyoxylate cycle during chemoautotrophic growth. Addition of arabinose to chemoautotrophic cultures of B. japonicum did not significantly alter transcript profiles. Furthermore, a subset of nitrogen fixation genes was moderately induced during chemoautotrophic growth. In order to specifically address the role of isocitrate lyase and nitrogenase in chemoautotrophic growth, we cultured aceA, nifD, and nifH mutants under chemoautotrophic conditions. Growth of each mutant was similar to that of the wild type, indicating that the glyoxylate bypass and nitrogenase activity are not essential components of chemoautotrophy in B. japonicum.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18689488">http://www.ncbi.nlm.nih.gov/pubmed/18689488</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2008).21(5):631-45. doi:10.1094/Mpmi-21-5-0631; PMID:18393623</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Transcription profiling of soybean nodulation by Bradyrhizobium japonicum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Laurent Brechenmacher, Moon-Young Kim, Marisol Benitez, Min Li, Trupti Joshi, Bernarda Calla, Mei Phing Lee, Marc Libault, Lila O Vodkin, Dong Xu, Suk-Ha Lee, Steven J Clough, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Divisions of Plant Sciences and Biochemistry; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Legumes interact with nodulating bacteria that convert atmospheric nitrogen into ammonia for plant use. This nitrogen fixation takes place within root nodules that form after infection of root hairs by compatible rhizobia. Using cDNA microarrays, we monitored gene expression in soybean (Glycine max) inoculated with the nodulating bacterium Bradyrhizobium japonicum 4, 8, and 16 days after inoculation, timepoints that coincide with nodule development and the onset of nitrogen fixation. This experiment identified several thousand genes that were differentially expressed in response to B. japonicum inoculation. Expression of 27 genes was analyzed by quantitative reverse transcriptase-polymerase chain reaction, and their expression patterns mimicked the microarray results, confirming integrity of analyses. The microarray results suggest that B. japonicum reduces plant defense responses during nodule development. In addition, the data revealed a high level of regulatory complexity (transcriptional, post-transcriptional, translational, post-translational) that is likely essential for development of the symbiosis and adjustment to an altered nutritional status.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18393623">http://www.ncbi.nlm.nih.gov/pubmed/18393623</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Mol. Biol.(2008).67(5):469-82. doi:10.1007/S11103-008-9332-6; PMID:18392777</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A lectin receptor-like kinase is required for pollen development in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jinrong Wan, Ami Patel, Melanie Mathieu, Sung-Yong Kim, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Divisions of Plant Sciences and Biochemistry; Department of Molecular Microbiology and Immunology; University of Missouri-Columbia; Columbia; MO 65211; USA. wanj@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Lectin receptor-like kinases (Lectin RLKs) are a large family of receptor-like kinases with an extracellular legume lectin-like domain. There are approximately 45 such receptor kinases in Arabidopsis thaliana. Surprisingly, although receptor-like kinases in general are well investigated in Arabidopsis, relatively little is known about the functions of members of the Lectin RLK family. A number of studies implicated members of this family in various functions, such as disease resistance, stress responses, hormone signaling, and legume-rhizobium symbiosis. Our current work demonstrated that mutation in one Lectin RLK gene led to male sterility in Arabidopsis. The sterility was due to defects in pollen development. Pollen development proceeded normally in the mutant until anther stage 8. After that, all pollen grains deformed and collapsed. Mature pollen grains were much smaller than wild-type pollen grains, glued together, and totally collapsed. Therefore, the mutant was named sgc, standing for small, glued-together, and collapsed pollen mutant. The mutant phenotype appeared to be caused by an unidentified sporophytic defect due to the mutation. As revealed by analysis of the promoter-GUS transgenic plants and the gene expression analysis using RT-PCR, the gene showed an interesting temporal and spatial expression pattern: it had no or a low expression in young flowers (roughly before anther stage 6), reached a maximum level around stages 6-7, and then declined gradually to a very low level in young siliques. No expression was detected in microspores or pollen. Together, our data demonstrated that SGC Lectin RLK plays a critical role in pollen development.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18392777">http://www.ncbi.nlm.nih.gov/pubmed/18392777</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Genome(2008).51(4):294-302. doi:10.1139/G08-010; PMID:18356965</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Microsatellite discovery from BAC end sequences and genetic mapping to anchor the soybean physical and genetic maps.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Randy C Shoemaker, David Grant, Terry Olson, Wesley C Warren, Rod Wing, Yeisoo Yu, HyeRan Kim, Perry Cregan, Bindu Joseph, Montona Futrell-Griggs, Will Nelson, Jon Davito, Jason Walker, John Wallis, Colin Kremitski, Debbie Scheer, Sandra W Clifton, Tina Graves, Henry Nguyen, Xiaolei Wu, Mingcheng Luo, Jan Dvorak, Rex Nelson, Steven Cannon, Jeff Tomkins, Jeremy Schmutz, Gary Stacey, Scott Jackson</b>
      </span>
      <br/>
      <span>Affiliation: USDA-ARS-CICGR Unit; Department of Agronomy; Ames; IA 50011-1010; USA. rcsshoe@iastate.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Whole-genome sequencing of the soybean (Glycine max (L.) Merr. 'Williams 82') has made it important to integrate its physical and genetic maps. To facilitate this integration of maps, we screened 3290 microsatellites (SSRs) identified from BAC end sequences of clones comprising the 'Williams 82' physical map. SSRs were screened against 3 mapping populations. We found the AAT and ACT motifs produced the greatest frequency of length polymorphisms, ranging from 17.2% to 32.3% and from 11.8% to 33.3%, respectively. Other useful motifs include the dinucleotide repeats AG, AT, and AG, with frequency of length polymorphisms ranging from 11.2% to 18.4% (AT), 12.4% to 20.6% (AG), and 11.3% to 16.4% (GT). Repeat lengths less than 16 bp were generally less useful than repeat lengths of 40-60 bp. Two hundred and sixty-five SSRs were genetically mapped in at least one population. Of the 265 mapped SSRs, 60 came from BAC singletons not yet placed into contigs of the physical map. One hundred and ten originated in BACs located in contigs for which no genetic map location was previously known. Ninety-five SSRs came from BACs within contigs for which one or more other BACs had already been mapped. For these fingerprinted contigs (FPC) a high percentage of the mapped markers showed inconsistent map locations. A strategy is introduced by which physical and genetic map inconsistencies can be resolved using the preliminary 4x assembly of the whole genome sequence of soybean.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18356965">http://www.ncbi.nlm.nih.gov/pubmed/18356965</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Cell(2008).20(2):471-81. doi:10.1105/Tpc.107.056754; PMID:18263776</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A LysM receptor-like kinase plays a critical role in chitin signaling and fungal resistance in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jinrong Wan, Xue-Cheng Zhang, David Neece, Katrina M Ramonell, Steve Clough, Sung-Yong Kim, Minviluz G Stacey, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; National Center for Soybean Biotechnology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin, a polymer of N-acetyl-d-glucosamine, is found in fungal cell walls but not in plants. Plant cells can perceive chitin fragments (chitooligosaccharides) leading to gene induction and defense responses. We identified a LysM receptor-like protein (LysM RLK1) required for chitin signaling in Arabidopsis thaliana. The mutation in this gene blocked the induction of almost all chitooligosaccharide-responsive genes and led to more susceptibility to fungal pathogens but had no effect on infection by a bacterial pathogen. Additionally, exogenously applied chitooligosaccharides enhanced resistance against both fungal and bacterial pathogens in the wild-type plants but not in the mutant. Together, our data indicate that LysM RLK1 is essential for chitin signaling in plants (likely as part of the receptor complex) and is involved in chitin-mediated plant innate immunity. The LysM RLK1-mediated chitin signaling pathway is unique, but it may share a conserved downstream pathway with the FLS2/flagellin- and EFR/EF-Tu-mediated signaling pathways. Additionally, our work suggests a possible evolutionary relationship between the chitin and Nod factor perception mechanisms due to the similarities between their potential receptors and between the signal molecules perceived by them.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18263776">http://www.ncbi.nlm.nih.gov/pubmed/18263776</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>BMC Genomics(2008).9():28. doi:10.1186/1471-2164-9-28; PMID:18211698</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genetic marker anchoring by six-dimensional pools for development of a soybean physical map.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Xiaolei Wu, Guohua Zhong, Seth D Findley, Perry Cregan, Gary Stacey, Henry T Nguyen</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and National Center for Soybean Biotechnology; University of Missouri-Columbia; Columbia; MO 65211; USA. wuxia@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Integrated genetic and physical maps are extremely valuable for genomic studies and as important references for assembling whole genome shotgun sequences. Screening of a BAC library using molecular markers is an indispensable procedure for integration of both physical and genetic maps of a genome. Molecular markers provide anchor points for integration of genetic and physical maps and also validate BAC contigs assembled based solely on BAC fingerprints. We employed a six-dimensional BAC pooling strategy and an in silico approach to anchor molecular markers onto the soybean physical map.A total of 1,470 markers (580 SSRs and 890 STSs) were anchored by PCR on a subset of a Williams 82 BstY I BAC library pooled into 208 pools in six dimensions. This resulted in 7,463 clones (approximately 1x genome equivalent) associated with 1470 markers, of which the majority of clones (6,157, 82.5%) were anchored by one marker and 1106 (17.5%) individual clones contained two or more markers. This contributed to 1184 contigs having anchor points through this 6-D pool screening effort. In parallel, the 21,700 soybean Unigene set from NCBI was used to perform in silico mapping on 80,700 Williams 82 BAC end sequences (BES). This in silico analysis yielded 9,835 positive results anchored by 4152 unigenes that contributed to 1305 contigs and 1624 singletons. Among the 1305 contigs, 305 have not been previously anchored by PCR. Therefore, 1489 (78.8%) of 1893 contigs are anchored with molecular markers. These results are being integrated with BAC fingerprints to assemble the BAC contigs. Ultimately, these efforts will lead to an integrated physical and genetic map resource.We demonstrated that the six-dimensional soybean BAC pools can be efficiently used to anchor markers to soybean BACs despite the complexity of the soybean genome. In addition to anchoring markers, the 6-D pooling method was also effective for targeting BAC clones for investigating gene families and duplicated regions in the genome, as well as for extending physical map contigs.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18211698">http://www.ncbi.nlm.nih.gov/pubmed/18211698</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2008).146(2):589-601. doi:10.1104/Pp.107.108183; PMID:18083798</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The Arabidopsis AtOPT3 protein functions in metal homeostasis and movement of iron to developing seeds.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Minviluz G Stacey, Ami Patel, William E McClain, Melanie Mathieu, Melissa Remley, Elizabeth E Rogers, Walter Gassmann, Dale G Blevins, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The Arabidopsis thaliana AtOPT3 belongs to the oligopeptide transporter (OPT) family, a relatively poorly characterized family of peptide/modified peptide transporters found in archebacteria, bacteria, fungi, and plants. A null mutation in AtOPT3 resulted in embryo lethality, indicating an essential role for AtOPT3 in embryo development. In this article, we report on the isolation and phenotypic characterization of a second AtOPT3 mutant line, opt3-2, harboring a T-DNA insertion in the 5' untranslated region of AtOPT3. The T-DNA insertion in the AtOPT3 promoter resulted in reduced but sufficient AtOPT3 expression to allow embryo formation in opt3-2 homozygous seeds. Phenotypic analyses of opt3-2 plants revealed three interesting loss-of-function phenotypes associated with iron metabolism. First, reduced AtOPT3 expression in opt3-2 plants resulted in the constitutive expression of root iron deficiency responses regardless of exogenous iron supply. Second, deregulation of root iron uptake processes in opt3-2 roots resulted in the accumulation of very high levels of iron in opt3-2 tissues. Hyperaccumulation of iron in opt3-2 resulted in the formation of brown necrotic areas in opt3-2 leaves and was more pronounced during the seed-filling stage. Third, reduced AtOPT3 expression resulted in decreased accumulation of iron in opt3-2 seeds. The reduced accumulation of iron in opt3-2 seeds is especially noteworthy considering the excessively high levels of accumulated iron in other opt3-2 tissues. AtOPT3, therefore, plays a critical role in two important aspects of iron metabolism, namely, maintenance of whole-plant iron homeostasis and iron nutrition of developing seeds.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/18083798">http://www.ncbi.nlm.nih.gov/pubmed/18083798</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2007).20(10):1298-307. doi:10.1094/Mpmi-20-10-1298; PMID:17918631</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">An oligonucleotide microarray resource for transcriptional profiling of Bradyrhizobium japonicum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Woo-Suk Chang, William L Franck, Eddie Cytryn, Sooyoung Jeong, Trupti Joshi, David W Emerich, Michael J Sadowsky, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; University of Missouri; Columbia 65211; USA. changwo@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>A DNA microarray, comprising 70-mer oligonucleotides, representing 8,453 open reading frames (ORFs), was constructed based on the Bradyrhizobium japonicum strain USDA110 genomic sequence. New annotation predicted 199 additional genes, which were added to the microarray and were shown to be transcribed. These arrays were used to profile transcription in cells under a variety of conditions, including growth in minimal versus rich medium, osmotic stress, and free-living cells versus bacteroids. Increased expression was seen for genes involved in translation, motility, and cell envelope synthesis in rich medium whereas expression increased in minimal medium for genes involved in vitamin biosynthesis and stress responses. Treatment with 50 mM NaCl activated stress-inducible genes but repressed genes involved in chemotaxis and motility. Strikingly, no known transport systems for accumulation of compatible solutes or osmoprotectants were induced in response to osmotic stress. A number of nif, fix, and hup genes, but not all, were upregulated in bacteroids. The B. japonicum type III secretion system, known to be important in early nodulation, was downregulated in bacteroids. The availability of a reliable, low-cost B. japonicum microarray provides a useful tool for functional genomic studies of one of the most agriculturally important bacteria.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17918631">http://www.ncbi.nlm.nih.gov/pubmed/17918631</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2007).20(8):900-11. doi:10.1094/Mpmi-20-8-0900; PMID:17722694</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Identification of 118 Arabidopsis transcription factor and 30 ubiquitin-ligase genes responding to chitin, a plant-defense elicitor.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Marc Libault, Jinrong Wan, Tomasz Czechowski, Michael Udvardi, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Division of Plant Science; Department of Molecular Microbiology and Immunology; Christopher S. Bond Life Sciences Center; University of Missouri; Columbia; MO 65211; USA. libaultm@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin, found in the cell walls of true fungi and the exoskeleton of insects and nematodes, is a well-established elicitor of plant defense responses. In this study, we analyzed the expression patterns of Arabidopsis thaliana transcription factor (TF) and ubiquitin-ligase genes in response to purified chitooctaose at different treatment times (15, 30, 60, 90, and 120 min after treatment), using both quantitative polymerase chain reaction and the Affymetrix Arabidopsis whole-genome array. A total of 118 TF genes and 30 ubiquitin-ligase genes were responsive to the chitin treatment. Among these genes, members from the following four TF families were overrepresented: APETALA2/ethylene-reponsive element binding proteins (27), C2H2 zinc finger proteins (14), MYB domain-containing proteins (11), and WRKY domain transcription factors (14). Transcript variants from a few of these genes were found to respond differentially to chitin, suggesting transcript-specific regulation of these TF genes.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17722694">http://www.ncbi.nlm.nih.gov/pubmed/17722694</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Bacteriol.(2007).189(19):6751-62. doi:10.1128/Jb.00533-07; PMID:17660288</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Transcriptional and physiological responses of Bradyrhizobium japonicum to desiccation-induced stress.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Eddie J Cytryn, Dipen P Sangurdekar, John G Streeter, William L Franck, Woo-Suk Chang, Gary Stacey, David W Emerich, Trupti Joshi, Dong Xu, Michael J Sadowsky</b>
      </span>
      <br/>
      <span>Affiliation: Department of Soil; Water; and Climate; BioTechnology Institute; University of Minnesota; St. Paul; Minnesota; USA [corrected]</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The growth and persistence of rhizobia and bradyrhizobia in soils are negatively impacted by drought conditions. In this study, we used genome-wide transcriptional analyses to obtain a comprehensive understanding of the response of Bradyrhizobium japonicum to drought. Desiccation of cells resulted in the differential expression of 15 to 20% of the 8,453 [corrected] B. japonicum open reading frames, with considerable differentiation between early (after 4 h) and late (after 24 and 72 h) expressed genes. While 225 genes were universally up-regulated at all three incubation times in response to desiccation, an additional 43 and 403 up-regulated genes were common to the 4/24- and 24/72-h incubation times, respectively. Desiccating conditions resulted in the significant induction (&gt;2.0-fold) of the trehalose-6-phosphate synthetase (otsA), trehalose-6-phosphate phosphatase (otsB), and trehalose synthase (treS) genes, which encode two of the three trehalose synthesis pathways found in B. japonicum. Gene induction was correlated with an elevated intracellular concentration of trehalose and increased activity of trehalose-6-phosphate synthetase, collectively supporting the hypothesis that this disaccharide plays a prominent and important role in promoting desiccation tolerance in B. japonicum. Microarray data also indicated that sigma(54)- and sigma(24)-associated transcriptional regulators and genes encoding isocitrate lyase, oxidative stress responses, the synthesis and transport of exopolysaccharides, heat shock response proteins, enzymes for the modification and repair of nucleic acids, and the synthesis of pili and flagella are also involved in the response of B. japonicum to desiccation. Polyethylene glycol-generated osmotic stress induced significantly fewer genes than those transcriptionally activated by desiccation. However, 67 genes were commonly induced under both conditions. Taken together, these results suggest that B. japonicum directly responds to desiccation by adapting to changes imparted by reduced water activity, such as the synthesis of trehalose and polysaccharides and, secondarily, by the induction of a wide variety of proteins involved in protection of the cell membrane, repair of DNA damage, stability and integrity of proteins, and oxidative stress responses.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17660288">http://www.ncbi.nlm.nih.gov/pubmed/17660288</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Trends Plant Sci.(2007).12(7):282-5. doi:10.1016/J.Tplants.2007.06.006; PMID:17591456</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Distinct, crucial roles of flavonoids during legume nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Senthil Subramanian, Gary Stacey, Oliver Yu</b>
      </span>
      <br/>
      <span>Affiliation: Donald Danforth Plant Science Center; 975 North Warson Road; Saint Louis; MO 63132; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>RNA interference-mediated silencing of the key flavonoid and isoflavone biosynthesis enzyme, respectively, by two different research groups has provided direct genetic evidence for the essential roles that these compounds play in nodulation. Anton Wasson et al. have shown that flavonoids are essential for localized auxin transport inhibition during nodulation in the indeterminate legume Medicago truncatula. By contrast, Senthil Subramanian et al. have shown that isoflavones are essential for endogenous nod gene induction in the determinate legume soybean.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17591456">http://www.ncbi.nlm.nih.gov/pubmed/17591456</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Science(2007).316(5829):1307-12. doi:10.1126/Science.1139548; PMID:17540897</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Eric Giraud, Lionel Moulin, David Vallenet, Val&#195;&#169;rie Barbe, Eddie Cytryn, Jean-Christophe Avarre, Marianne Jaubert, Damien Simon, Fabienne Cartieaux, Yves Prin, Gilles Bena, Laure Hannibal, Joel Fardoux, Mila Kojadinovic, Laurie Vuillet, Aur&#195;&#169;lie Lajus, St&#195;&#169;phane Cruveiller, Zoe Rouy, Sophie Mangenot, B&#195;&#169;atrice Segurens, Carole Dossat, William L Franck, Woo-Suk Chang, Elizabeth Saunders, David Bruce, Paul Richardson, Philippe Normand, Bernard Dreyfus, David Pignol, Gary Stacey, David Emerich, Andr&#195;&#169; Verm&#195;&#169;glio, Claudine M&#195;&#169;digue, Michael Sadowsky</b>
      </span>
      <br/>
      <span>Affiliation: Institut de Recherche pour le D&#195;&#169;veloppement; Centre de Coop&#195;&#169;ration International en Recherche Agronomique pour le D&#195;&#169;veloppement; Institut National de la Recherche Agronomique; Universit&#195;&#169; Montpellier 2; France. giraud@mpl.ird.fr</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Leguminous plants (such as peas and soybeans) and rhizobial soil bacteria are symbiotic partners that communicate through molecular signaling pathways, resulting in the formation of nodules on legume roots and occasionally stems that house nitrogen-fixing bacteria. Nodule formation has been assumed to be exclusively initiated by the binding of bacterial, host-specific lipochito-oligosaccharidic Nod factors, encoded by the nodABC genes, to kinase-like receptors of the plant. Here we show by complete genome sequencing of two symbiotic, photosynthetic, Bradyrhizobium strains, BTAi1 and ORS278, that canonical nodABC genes and typical lipochito-oligosaccharidic Nod factors are not required for symbiosis in some legumes. Mutational analyses indicated that these unique rhizobia use an alternative pathway to initiate symbioses, where a purine derivative may play a key role in triggering nodule formation.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/316/5829/1307.full">http://science.sciencemag.org/content/316/5829/1307.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2007).144(2):623-36. doi:10.1104/Pp.107.097097; PMID:17449649</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Molecular evolution of lysin motif-type receptor-like kinases in plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Xue-Cheng Zhang, Xiaolei Wu, Seth Findley, Jinrong Wan, Marc Libault, Henry T Nguyen, Steven B Cannon, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences and National Center for Soybean Biotechnology ; University of Missouri; Columbia; Missouri 65211; USA. zhangxuec@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The lysin motif (LysM) domain is an ancient and ubiquitous protein module that binds peptidoglycan and structurally related molecules. A genomic survey in a large number of species spanning all kingdoms reveals that the combination of LysM and receptor kinase domains is present exclusively in plants. However, the particular biological functions and molecular evolution of this gene family remain largely unknown. We show that LysM domains in plant LysM proteins are highly diversified and that a minimum of six distinct types of LysM motifs exist in plant LysM kinase proteins and five additional types of LysM motifs exist in nonkinase plant LysM proteins. Further, motif similarities suggest that plant LysM motifs are ancient. Although phylogenetic signals are not sufficient to resolve the earliest relationships, plant LysM motifs may have arisen through common ancestry with LysM motifs in other kingdoms. Within plants, the gene family has evolved through local and segmental duplications. The family has undergone further duplication and diversification in legumes, where some LysM kinase genes function as receptors for bacterial nodulation factor. Two pairs of homeologous regions were identified in soybean (Glycine max) based on microsynteny and fluorescence in situ hybridization. Expression data show that most plant LysM kinase genes are expressed predominantly in the root and that orthologous LysM kinase genes share similar tissue expression patterns. We also examined synteny around plant LysM kinase genes to help reconstruct scenarios for the evolution of this important gene family.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17449649">http://www.ncbi.nlm.nih.gov/pubmed/17449649</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Conf Proc IEEE Eng Med Biol Soc(2004).4():3051-4. doi:10.1109/Iembs.2004.1403863; PMID:17270922</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Statistical assessment for mass-spec protein identification using peptide fingerprinting approach.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Ashwin Ganapathy, Xiu-Feng Wan, Jinrong Wan, Jay Thelen, David W Emerich, Gary Stacey, Dong Xu</b>
      </span>
      <br/>
      <span>Affiliation: Dept. of Comput. Sci.; Missouri Univ.; Columbia; MO; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>We derive and validate a novel statistical model for confidence assessment of protein identification results using peptide mass fingerprint data. We simulate the digestion of the proteins and compare each peptide mass with the input mass. We compute scores from this matching of peptide and compute the distribution of scores for all the proteins in the database. Based on the distribution, we can provide the expectation value for a protein match in the database. We conclude that, given the complexity and noise of the data, the best method for effective confidence matching is using one scoring scheme for matching and another scoring scheme for confidence assessment.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17270922">http://www.ncbi.nlm.nih.gov/pubmed/17270922</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Electrophoresis(2007).28(5):864-70. doi:10.1002/Elps.200600305; PMID:17265538</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Development and assessment of scoring functions for protein identification using PMF data.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Zhao Song, Luonan Chen, Ashwin Ganapathy, Xiu-Feng Wan, Laurent Brechenmacher, Nengbing Tao, David Emerich, Gary Stacey, Dong Xu</b>
      </span>
      <br/>
      <span>Affiliation: Computer Science Department and Christopher S. Bond Life Sciences Center; University of Missouri-Columbia; Columbia; MO 65211-2060; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>PMF is one of the major methods for protein identification using the MS technology. It is faster and cheaper than MS/MS. Although PMF does not differentiate trypsin-digested peptides of identical mass, which makes it less informative than MS/MS, current computational methods for PMF have the potential to improve its detection accuracy by better use of the information content in PMF spectra. We developed a number of new probability-based scoring functions for PMF protein identification based on the MOWSE algorithm. We considered a detailed distribution of matching masses in a protein database and peak intensity, as well as the likelihood of peptide matches to be close to each other in a protein sequence. Our computational methods are assessed and compared with other methods using PMF data of 52 gel spots of known protein standards. The comparison shows that our new scoring schemes have higher or comparable accuracies for protein identification in comparison to the existing methods. Our software is freely available upon request. The scoring functions can be easily incorporated into other proteomics software packages.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17265538">http://www.ncbi.nlm.nih.gov/pubmed/17265538</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant J.(2006).48(2):261-73. doi:10.1111/J.1365-313X.2006.02874.X; PMID:17018035</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Endogenous isoflavones are essential for the establishment of symbiosis between soybean and Bradyrhizobium japonicum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Senthil Subramanian, Gary Stacey, Oliver Yu</b>
      </span>
      <br/>
      <span>Affiliation: Donald Danforth Plant Science Center; St Louis; MO 63132; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Legume iso/flavonoids have been implicated in the nodulation process, but questions remain as to their specific role(s), and no unequivocal evidence exists showing that these compounds are essential for nodulation. Two hypotheses suggest that the primary role of iso/flavonoids is their ability to induce rhizobial nod gene expression and/or their ability to modulate internal root auxin concentrations. The present work provides direct, genetic evidence that isoflavones are essential for nodulation of soybean roots because of their ability to induce the nodulation genes of Bradyrhizobium japonicum. Expression of isoflavone synthase (IFS), a key enzyme in the biosynthesis of isoflavones, is specifically induced by B. japonicum. When IFS was silenced using RNA interference in soybean hairy root composite plants, these plants had severely reduced nodulation. Surprisingly, pre-treatment of B. japonicum or exogenous application to the root system of either of the major soybean isoflavones, daidzein or genistein, failed to restore normal nodulation. Silencing of chalcone reductase led to very low levels of daidzein and increased levels of genistein, but did not affect nodulation, suggesting that the endogenous production of genistein was sufficient to support nodulation. Consistent with a role for isoflavones as endogenous regulators of auxin transport in soybean roots, silencing of IFS resulted in altered auxin-inducible gene expression and auxin transport. However, use of a genistein-hypersensitive B. japonicum strain or purified B. japonicum Nod signals rescued normal nodulation in IFS-silenced roots, indicating that the ability of isoflavones to modulate auxin transport is not essential to nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/17018035">http://www.ncbi.nlm.nih.gov/pubmed/17018035</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2006).142(3):984-92. doi:10.1104/Pp.106.085670; PMID:16963521</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Extracellular ATP in plants. Visualization, localization, and analysis of physiological significance in growth and signaling.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Sung-Yong Kim, Mayandi Sivaguru, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology and Division of Plant Sciences ; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Extracellular ATP (eATP) in animals is well documented and known to play an important role in cellular signaling (e.g. at the nerve synapse). The existence of eATP has been postulated in plants; however, there is no definitive experimental evidence for its presence or an explanation as to how such a polar molecule could exit the plant cell and what physiological role it may play in plant growth and development. The presence of eATP in plants (Medicago truncatula) was detected by constructing a novel reporter; i.e. fusing a cellulose-binding domain peptide to the ATP-requiring enzyme luciferase. Application of this reporter to plant roots allowed visualization of eATP in the presence of the substrate luciferin. Luciferase activity could be detected in the interstitial spaces between plant epidermal cells and predominantly at the regions of actively growing cells. The levels of eATP were closely correlated with regions of active growth and cell expansion. Pharmacological compounds known to alter cytoplasmic calcium levels revealed that ATP release is a calcium-dependent process and may occur through vesicular fusion, an important step in the polar growth of actively growing root hairs. Reactive oxygen species (ROS) activity at the root hair tip is not only essential for root hair growth, but also dependent on the cytoplasmic calcium levels. Whereas application of exogenous ATP and a chitin mixture increased ROS activity in root hairs, no changes were observed in response to adenosine, AMP, ADP, and nonhydrolyzable ATP (betagammameATP). However, application of exogenous potato (Solanum tuberosum) apyrase (ATPase) decreased ROS activity, suggesting that cytoplasmic calcium gradients and ROS activity are closely associated with eATP release.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/16963521">http://www.ncbi.nlm.nih.gov/pubmed/16963521</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2006).141(4):1473-81. doi:10.1104/Pp.106.080986; PMID:16798946</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Effects of endogenous salicylic acid on nodulation in the model legumes Lotus japonicus and Medicago truncatula.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Gary Stacey, Crystal Bickley McAlvin, Sung-Yong Kim, Jos&#195;&#169; Olivares, Mar&#195;&#173;a Jos&#195;&#169; Soto</b>
      </span>
      <br/>
      <span>Affiliation: Department of Microbiology; University of Tennessee; Knoxville; Tennessee 37996; USA. staceyg@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The exogenous addition of salicylic acid (SA) was previously shown to inhibit indeterminate but not determinate-type nodulation. We sought to extend these results by modulating endogenous levels of SA through the transgenic expression of salicylate hydroxylase (NahG) in both stably transformed Lotus japonicus and composite Medicago truncatula plants. NahG expression in L. japonicus resulted in a marked reduction of SA levels. This reduction correlated with an increase in the number of infections and mean nodule number when compared to controls. However, a complicating factor was that NahG-expressing plants had greater root growth. Spot inoculations of NahG-expressing L. japonicus plants confirmed increased nodulation in these plants. Consistent with the reported inhibitory effects of exogenous SA on indeterminate-type nodulation, NahG expression in M. truncatula plants led to enhanced nodulation and infection. These data point to an important role for SA-mediated plant defense pathways in controlling nodule formation on both determinate and indeterminate nodule-forming hosts.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/16798946">http://www.ncbi.nlm.nih.gov/pubmed/16798946</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Curr. Opin. Plant Biol.(2006).9(2):110-21. doi:10.1016/J.Pbi.2006.01.005; PMID:16458572</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genetics and functional genomics of legume nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Gary Stacey, Marc Libault, Laurent Brechenmacher, Jinrong Wan, Gregory D May</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Division of Plant Science; University of Missouri; Columbia; Missouri 65211; USA. staceyg@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Gram-negative soil bacteria (rhizobia) within the Rhizobiaceae phylogenetic family (alpha-proteobacteria) have the unique ability to infect and establish a nitrogen-fixing symbiosis on the roots of leguminous plants. This symbiosis is of agronomic importance, reducing the need for nitrogen fertilizer for agriculturally important plants (e.g. soybean and alfalfa). The establishment of the symbiosis involves a complex interplay between host and symbiont, resulting in the formation of a novel organ, the nodule, which the bacteria colonize as intracellular symbionts. This review focuses on the most recent discoveries relating to how this symbiosis is established. Two general developments have contributed to the recent explosion of research progress in this area: first, the adoption of two genetic model legumes, Medicago truncatula and Lotus japonicus, and second, the application of modern methods in functional genomics (e.g. transcriptomic, proteomic and metabolomic analyses).</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/16458572">http://www.ncbi.nlm.nih.gov/pubmed/16458572</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Planta(2006).223(2):291-305. doi:10.1007/S00425-005-0087-X; PMID:16151844</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Expression analyses of Arabidopsis oligopeptide transporters during seed germination, vegetative growth and reproduction.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Minviluz G Stacey, Hiroki Osawa, Ami Patel, Walter Gassmann, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; The University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>AtOPT promoter-GUS fusions were constructed for six of the nine known, putative oligopeptide transporters (OPTs) in Arabidopsis thaliana and used to examine AtOPT expression at various stages of plant development. AtOPT1, AtOPT3, AtOPT4, AtOPT6 and AtOPT7 were expressed in the embryonic cotyledons prior to root radicle emergence. Except for AtOPT8, which gave weak expression, all AtOPTs were strongly expressed in post-germinative seedlings with strongest expression in vascular tissues of cotyledons and hypocotyls. Preferential expression of AtOPTs in vascular tissues was also observed in cotyledons, leaves, hypocotyls, roots, flowers, siliques, and seed funiculi of seedlings and adult plants. Differential tissue-specific expression was observed for specific AtOPTs. For example, AtOPT1, AtOPT3 and AtOPT8 were uniquely expressed in pollen. Only AtOPT1 was expressed in growing pollen tubes, while only AtOPT6 was observed in ovules. AtOPT8 was transiently expressed in seeds during early stages of embryogenesis. Iron limitation was found to enhance expression of AtOPT3. These data suggest distinct cellular roles for specific AtOPTs including nitrogen mobilization during germination and senescence, pollen tube growth, pollen and ovule development, seed formation and metal transport.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/16151844">http://www.ncbi.nlm.nih.gov/pubmed/16151844</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Biochem. J.(2006).393(Pt 1):267-75. doi:10.1042/Bj20050920; PMID:16149917</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">ScOPT1 and AtOPT4 function as proton-coupled oligopeptide transporters with broad but distinct substrate specificities.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Hiroki Osawa, Gary Stacey, Walter Gassmann</b>
      </span>
      <br/>
      <span>Affiliation: Division of Plant Sciences; Life Sciences Center; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>A group of OPTs (oligopeptide transporters) exclusively identified in plants and fungi are proposed to transport oligopeptides and derivatives of three to six amino acids in length, but their transport mechanisms and biological functions are poorly understood. We expressed the Saccharomyces cerevisiae (yeast) OPT ScOPT1 and five Arabidopsis thaliana AtOPTs in Xenopus laevis oocytes for two-electrode voltage-clamp studies. ScOPT1 produced inward currents in response to GSH or GSSG, the phytochelatin (PC) PC2 and oligopeptides including the tetrapeptide GGFL, but not KLGL. Inward currents were dependent on the external proton and substrate concentrations, with high affinity for both. This and the inward currents evoked by substrates with net negative charges showed that ScOPT1 is a proton-coupled transporter. ScOPT1 displayed highest apparent affinity for PC2, with small differences in the maximal current among substrates. Glutathione transport by any of the tested AtOPTs, including AtOPT6, was not detected in yeast growth complementation assays. With AtOPT4, initially only small KLGL-dependent currents were recorded in batches of oocytes showing high ScOPT1 expression. AtOPT4 expression was optimized by swapping the 5'-untranslated region with that of ScOPT1. AtOPT4 displayed a higher affinity for KLGL than ScOPT1 did for any peptide. AtOPT4-mediated KLGL transport was detectable at pH 5.0, but not at pH 6.0 or 7.0. Taken together, our results demonstrate that ScOPT1 and AtOPT4 are proton-coupled OPTs with broad but distinct substrate specificities and affinities.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/16149917">http://www.ncbi.nlm.nih.gov/pubmed/16149917</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2005).138(2):1027-36. doi:10.1104/Pp.105.060947; PMID:15923325</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Loss-of-function mutations in chitin responsive genes show increased susceptibility to the powdery mildew pathogen Erysiphe cichoracearum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Katrina Ramonell, Marta Berrocal-Lobo, Serry Koh, Jinrong Wan, Herb Edwards, Gary Stacey, Shauna Somerville</b>
      </span>
      <br/>
      <span>Affiliation: Department of Biological Sciences; University of Alabama; Tuscaloosa; Alabama 35487-0344; USA. kramonel@bama.ua.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Chitin is a major component of fungal walls and insect exoskeletons. Plants produce chitinases upon pathogen attack and chito-oligomers induce defense responses in plants, though the exact mechanism behind this response is unknown. Using the ATH1 Affymetrix microarrays consisting of about 23,000 genes, we examined the response of Arabidopsis (Arabidopsis thaliana) seedlings to chito-octamers and hydrolyzed chitin after 30 min of treatment. The expression patterns elicited by the chito-octamer and hydrolyzed chitin were similar. Microarray expression profiles for several genes were verified via northern analysis or quantitative reverse transcription-PCR. We characterized T-DNA insertion mutants for nine chito-oligomer responsive genes. Three of the mutants were more susceptible to the fungal pathogen, powdery mildew, than wild type as measured by conidiophore production. These three mutants included mutants of genes for two disease resistance-like proteins and a putative E3 ligase. The isolation of loss-of-function mutants with enhanced disease susceptibility provides direct evidence that the chito-octamer is an important oligosaccharide elicitor of plant defenses. Also, this study demonstrates the value of microarray data for identifying new components of uncharacterized signaling pathways.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/15923325">http://www.ncbi.nlm.nih.gov/pubmed/15923325</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2005).18(5):458-67. doi:10.1094/Mpmi-18-0458; PMID:15915644</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Proteomic analysis of soybean root hairs after infection by Bradyrhizobium japonicum.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jinrong Wan, Michael Torres, Ashwin Ganapathy, Jay Thelen, Beverly B DaGue, Brian Mooney, Dong Xu, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Department of Plant Microbiology and Pathology; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Infection of soybean root hairs by Bradyrhizobium japonicum is the first of several complex events leading to nodulation. In the current proteomic study, soybean root hairs after inoculation with B. japonicum were separated from roots. Total proteins were analyzed by two-dimensional (2-D) polyacrylamide gel electrophoresis. In one experiment, 96 protein spots were analyzed by matrix-assisted laser desorption ionization time-of-flight (MALDI-TOF) mass spectrometry (MS) to compare protein profiles between uninoculated roots and root hairs. Another 37 spots, derived from inoculated root hairs over different timepoints, were also analyzed by tandem MS (MS/MS). As expected, some proteins were differentially expressed in root hairs compared with roots (e.g., a chitinase and phosphoenolpyruvate carboxylase). Out of 37 spots analyzed by MS/MS, 27 candidate proteins were identified by database comparisons. These included several proteins known to respond to rhizobial inoculation (e.g., peroxidase and phenylalanine-ammonia lyase). However, novel proteins were also identified (e.g., phospholipase D and phosphoglucomutase). This research establishes an excellent system for the study of root-hair infection by rhizobia and, in a more general sense, the functional genomics of a single, plant cell type. The results obtained also indicate that proteomic studies with soybean, lacking a complete genome sequence, are practical.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/15915644">http://www.ncbi.nlm.nih.gov/pubmed/15915644</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Editorial</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2005).137(4):1173. doi:10.1104/Pp.104.900141; PMID:15824278</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">"Translational" legume biology. Models to crops.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Gary Stacey, Kate VandenBosch</b>
      </span>
      <br/>
      <span>Affiliation: </span>
    </p>
    <p><span><b>Abstract:</b></span><br/></p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/15824278">http://www.ncbi.nlm.nih.gov/pubmed/15824278</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2005).137(4):1456-62. doi:10.1104/Pp.104.055939; PMID:15793071</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Transgenic expression of the soybean apyrase in Lotus japonicus enhances nodulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Crystal B McAlvin, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Microbiology; University of Tennessee; Knoxville; Tennessee 37996; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The soybean apyrase, GS52, was previously characterized as an early nodulin that is expressed in roots and localized to the plasma membrane. Transgenic Lotus japonicus plants were constructed constitutively expressing the GS52 apyrase. Segregation and Southern-blot analysis identified four single-copy sense lines, several double-copy sense lines, and one double-copy antisense line for further analysis. The single- and double-copy sense gs52 L. japonicus lines had enhanced nodulation that correlated with expression of the transgene. The sense transgenic lines were also found to have increased infection thread formation and enhanced infection zone length when infected by Mesorhizobium loti, the natural symbiont of L. japonicus. The data presented show that expression of the GS52 apyrase can enhance nodulation in L. japonicus and points to an important role for this group of enzymes in nodulation.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/15793071">http://www.ncbi.nlm.nih.gov/pubmed/15793071</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Congresses</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2004).135(1):59-70. doi:10.1104/Pp.103.037903; PMID:15141067</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">National Science Foundation-sponsored workshop report. Draft plan for soybean genomics.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Gary Stacey, Lila Vodkin, Wayne A Parrott, Randy C Shoemaker</b>
      </span>
      <br/>
      <span>Affiliation: National Center for Soybean Biotechnology; Department of Plant Microbiology and Pathology; University of Missouri; Columbia; Missouri 65203; USA. staceyg@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Recent efforts to coordinate and define a research strategy for soybean (Glycine max) genomics began with the establishment of a Soybean Genetics Executive Committee, which will serve as a communication focal point between the soybean research community and granting agencies. Secondly, a workshop was held to define a strategy to incorporate existing tools into a framework for advancing soybean genomics research. This workshop identified and ranked research priorities essential to making more informed decisions as to how to proceed with large scale sequencing and other genomics efforts. Most critical among these was the need to finalize a physical map and to obtain a better understanding of genome microstructure. Addressing these research needs will require pilot work on new technologies to demonstrate an ability to discriminate between recently duplicated regions in the soybean genome and pilot projects to analyze an adequate amount of random genome sequence to identify and catalog common repeats. The development of additional markers, reverse genetics tools, and bioinformatics is also necessary. Successful implementation of these goals will require close coordination among various working groups.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/15141067">http://www.ncbi.nlm.nih.gov/pubmed/15141067</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Biol. Chem.(2004).279(29):30150-7. doi:10.1074/Jbc.M405192200; PMID:15138259</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Mechanisms and functional properties of two peptide transporters, AtPTR2 and fPTR2.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Chien-Sung Chiang, Gary Stacey, Yi-Fang Tsay</b>
      </span>
      <br/>
      <span>Affiliation: Institute of Molecular Biology; Academia Sinica; Taipei 11529; Taiwan.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The Arabidopsis AtPTR2 and fungal fPTR2 genes, which encode H+/dipeptide cotransporters, belong to two different subgroups of the peptide transporter (PTR) (NRT1) family. In this study, the kinetics, substrate specificity, stoichiometry, and voltage dependence of these two transporters expressed in Xenopus oocytes were investigated using the two-microelectrode voltage-clamp method. The results showed that: 1) although AtPTR2 belongs to the same PTR family subgroup as certain H+/nitrate cotransporters, neither AtPTR2 nor fPTR2 exhibited any nitrate transporting activity; 2) AtPTR2 and fPTR2 transported a wide spectrum of dipeptides with apparent affinity constants in the range of 30 microM to 3 mM, the affinity being dependent on the side chain structure of both the N- and C-terminal amino acids; 3) larger maximal currents (Imax) were evoked by positively charged dipeptides in AtPTR2- or fPTR2-injected oocytes; 4) a major difference between AtPTR2 and fPTR2 was that, whereas fPTR2 exhibited low Ala-Asp- transporting activity, AtPTR2 transported Ala-Asp- as efficiently as some of the positively charged dipeptides; 5) kinetic analysis suggested that both fPTR2 and AtPTR2 transported by a random binding, simultaneous transport mechanism. The results also showed that AtPTR2 and fPTR2 were quite distinct from PepT1 and PepT2, two well characterized animal PTR transporters in terms of order of binding of substrate and proton(s), pH sensitivity, and voltage dependence.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/15138259">http://www.ncbi.nlm.nih.gov/pubmed/15138259</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2004).134(2):871-9. doi:10.1104/Pp.103.031740; PMID:14966249</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">NopL, an effector protein of Rhizobium sp. NGR234, thwarts activation of plant defense reactions.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Alexander V Bartsev, William J Deakin, Nawal M Boukli, Crystal B McAlvin, Gary Stacey, Pia Malno&#195;&#171;, William J Broughton, Christian Staehelin</b>
      </span>
      <br/>
      <span>Affiliation: Laboratoire de Biologie Mol&#195;&#169;culaire des Plantes Sup&#195;&#169;rieures; Sciences III; Universit&#195;&#169; de Gen&#195;&#168;ve; 1211 Gen&#195;&#168;ve 4; Switzerland.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Bacterial effector proteins delivered into eukaryotic cells via bacterial type III secretion systems are important virulence factors in plant-pathogen interactions. Type III secretion systems have been found in Rhizobium species that form symbiotic, nitrogen-fixing associations with legumes. One such bacterium, Rhizobium sp. NGR234, secretes a number of type III effectors, including nodulation outer protein L (NopL, formerly y4xL). Here, we show that expression of nopL in tobacco (Nicotiana tabacum) prevents full induction of pathogenesis-related (PR) defense proteins. Transgenic tobacco plants that express nopL and were infected with potato virus Y (necrotic strain 605) exhibited only very low levels of chitinase (class I) and beta-1,3-glucanase (classes I and III) proteins. Northern-blot analysis indicated that expression of nopL in plant cells suppresses transcription of PR genes. Treatment with ethylene counteracted the effect of NopL on chitinase (class I). Transgenic Lotus japonicus plants that expressed nopL exhibited delayed development and low chitinase levels. In vitro experiments showed that NopL is a substrate for plant protein kinases. Together, these data suggest that NopL, when delivered into the plant cell, modulates the activity of signal transduction pathways that culminate in activation of PR proteins.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/14966249">http://www.ncbi.nlm.nih.gov/pubmed/14966249</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Appl. Environ. Microbiol.(2003).69(1):10-7. doi:; PMID:12513971</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Nodulation gene regulation in Bradyrhizobium japonicum: a unique integration of global regulatory circuits.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>John Loh, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Microbiology and Pathology; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/></p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/12513971">http://www.ncbi.nlm.nih.gov/pubmed/12513971</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Cell(2002).14(11):2799-811. doi:; PMID:12417702</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">AtOPT3, a member of the oligopeptide transporter family, is essential for embryo development in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Minviluz G Stacey, Serry Koh, Jeffrey Becker, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Microbiology and Pathology; University of Missouri; Columbia; Missouri 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>A T-DNA-tagged population of Arabidopsis was screened for mutations in AtOPT3, which encodes a member of the oligopeptide (OPT) family of peptide transporters, and a recessive mutant allele, opt3, was identified. Phenotypic analysis of opt3 showed that most homozygous embryos were arrested at or before the octant stage of embryo development and that none showed the usual periclinal division leading to the formation of the protoderm. This defective phenotype could be reversed by complementation with the full-length, wild-type AtOPT3 gene. A beta-glucuronidase (GUS) fusion to DNA sequences upstream of the putative AtOPT3 ATG start codon was constructed, and the expression pattern was assayed in transgenic plants. AtOPT3 was expressed in the vascular tissues of seedlings and mature plants as well as in pollen. Consistent with the function of AtOPT3 in embryogenesis, AtOPT3::GUS expression also was detected in developing embryos and in the maternal tissues of seeds. These data suggest a critical role for peptide transport in early embryo development.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/12417702">http://www.ncbi.nlm.nih.gov/pubmed/12417702</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Proc. Natl. Acad. Sci. U.S.A.(2002).99(22):14446-51. doi:10.1073/Pnas.222336799; PMID:12393811</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Bradyoxetin, a unique chemical signal involved in symbiotic gene regulation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>John Loh, Russell W Carlson, William S York, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Microbiology and Pathology; University of Missouri; Columbia; MO 65211; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Bradyrhizobium japonicum is a symbiotic bacterium that nodulates soybean. Critical for the infection and establishment of this symbiosis are the bacterial nodulation genes (nod, nol, noe), which are induced in the presence of plant produced isoflavones. Transcription of the nodulation genes is also controlled in a population density-dependent fashion. Expression of the nod genes is maximal at low population densities, and decreases significantly at higher culture densities. Population density control of the nodulation genes involves NolA and NodD2, both of which function in tandem to repress nod gene expression. An extracellular secreted factor (CDF) is known to mediate this repression. Here, we report that CDF is a novel signaling molecule, designated bradyoxetin, different from other Gram-negative quorum signals. The proposed structure of bradyoxetin is 2-[4-[[4-(3-aminooxetan-2-yl)phenyl](imino)methyl]phenyl]oxetan-3-ylamine. Interestingly, expression of bradyoxetin is iron-regulated, and is maximally produced under iron-starved conditions. Consistent with this, expression of the nodulation genes occurred in an iron-dependent fashion. Addition of iron to B. japonicum cultures at high optical densities resulted in decreased bradyoxetin production, and a concomitant reduction in nolA expression. A corresponding increase in nodY-lacZ expression was observed with iron treatment.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/12393811">http://www.ncbi.nlm.nih.gov/pubmed/12393811</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Mol. Plant Microbe Interact.(2002).15(9):963-70. doi:10.1094/Mpmi.2002.15.9.963; PMID:12236603</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Characterization of early, chitin-induced gene expression in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Bing Zhang, Katrina Ramonell, Shauna Somerville, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Center for Legume Research; Department of Microbiology; University of Tennessee; Knoxville 37996-0845; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Three genes (i.e., a zinc finger protein, a lectin-like protein, and AtMPK3), previously shown to respond to chitin elicitation in microarray experiments, were used to examine the response of Arabidopsis spp. to chitin addition. Maximum induction for all three genes was found upon addition of crab-shell chitin at 100 mg per liter. Threefold induction was found with a chitin concentration as low as 10(-4) mg per liter. The specificity of this response was examined using purified chitin oligomers (degree of polymerization = 2 to 8). The larger chitin oligomers (hexamer to octamer), were most effective in inducing expression of the three genes assayed. Gene induction was observed after the addition of 1 nM chitin octamer. The protein kinase inhibitors staurosporine and K252a effectively suppressed chitin-induced gene expression, while the protein phosphatase inhibitors calyculin A and okadaic acid induced the accumulation of mRNA in the absence of chitin. The phosphorylation event necessary for transmission of the chitin signal was completed within the first 20 min of chitin addition. The level of chitin-induced gene expression of the lectin-like protein and AtMPK3 was not significantly changed in mutants blocked in the jasmonic acid (JA, jar1)-, ethylene (ein2)-, or salicylic acid (SA, pad4, npr1, and eds5)-dependent pathway. In contrast, expression of mRNA for the zinc finger protein was reduced in the mutants affected in the JA- or SA-dependent pathway.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/12236603">http://www.ncbi.nlm.nih.gov/pubmed/12236603</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Curr. Opin. Plant Biol.(2002).5(4):285-90. doi:; PMID:12179960</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Quorum sensing in plant-associated bacteria.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>John Loh, Elizabeth A Pierson, Leland S Pierson, Gary Stacey, Arun Chatterjee</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Microbiology and Pathology; University of Missouri; Columbia; Missouri; USA. lohjt@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>N-acyl homoserine lactone (AHL)-mediated quorum sensing by bacteria regulates traits that are involved in symbiotic, pathogenic and surface-associated relationships between microbial populations and their plant hosts. Recent advances demonstrate deviations from the classic LuxR/LuxI paradigm, which was first developed in Vibrio. For example, LuxR homologs can repress as well as activate gene expression, and non-AHL signals and signal mimics can affect the expression of genes that are controlled by quorum sensing. Many bacteria utilize multiple quorum-sensing systems, and these may be modulated via post-transcriptional and other global regulatory mechanisms. Microbes inhabiting plant surfaces also produce and respond to a diverse mixture of AHL signals. The production of AHL mimics by plants and the identification of AHL degradative pathways suggest that bacteria and plants utilize this method of bacterial communication as a key control point for influencing the outcome of their interactions.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/12179960">http://www.ncbi.nlm.nih.gov/pubmed/12179960</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Review</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Trends Plant Sci.(2002).7(6):257-63. doi:; PMID:12049922</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Peptide transport in plants.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Gary Stacey, Serry Koh, Cheryl Granger, Jeffrey M Becker</b>
      </span>
      <br/>
      <span>Affiliation: Dept of Plant Microbiology and Pathology; University of Missouri; Columbia; MO 65211; USA. staceyg@missouri.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Recent completion of the Arabidopsis genome revealed that this organism has ten times more peptide transporters than any other sequenced organism (prokaryote or eukaryote). These transporters are found in three protein families: the ABC-type transporters; the di- and tripeptide transporters; and the newly described tetra- and pentapeptide oligopetide transporters. The abundance of these transporters suggests that they play diverse and important roles in plant growth and development. Possible substrates for these transporters include glutathione, gamma-glutamyl peptides, hormone-amino acid conjugates, phytosulfokine, peptide-like compounds and peptide phytotoxins. However, the exact role of peptide transport in plants is still undefined.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/12049922">http://www.ncbi.nlm.nih.gov/pubmed/12049922</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>J. Bacteriol.(2002).184(6):1759-66. doi:; PMID:11872728</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A two-component regulator mediates population-density-dependent expression of the Bradyrhizobium japonicum nodulation genes.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>John Loh, Dasharath P Lohar, Brett Andersen, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Center for Legume Research and Department of Microbiology; The University of Tennessee; Knoxville; Tennessee 37996; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Bradyrhizobium japonicum nod gene expression was previously shown to be population density dependent. Induction of the nod genes is highest at low culture density and repressed at high population densities. This repression involves both NolA and NodD2 and is mediated by an extracellular factor found in B. japonicum conditioned medium. NolA and NodD2 expression is maximal at high population densities. We demonstrate here that a response regulator, encoded by nwsB, is required for the full expression of the B. japonicum nodYABC operon. In addition, NwsB is also required for the population-density-dependent expression of both nolA and nodD2. Expression of nolA and nodD2 in the nwsB mutant remained at a basal level, even at high culture densities. The nwsB defect could be complemented by overexpression of a second response regulator, NodW. Consistent with the fact that NolA and NodD2 repress nod gene expression, the expression of a nodY-lacZ fusion in the nwsB mutant was unaffected by culture density. In plant assays with GUS fusions, nodules infected with the wild type showed no nodY-GUS expression. In contrast, nodY-GUS expression was not repressed in nodules infected with the nwsB mutant. Nodule competition assays between the wild type and the nwsB mutant revealed that the addition of conditioned medium resulted in a competitive advantage for the nwsB mutant.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/11872728">http://www.ncbi.nlm.nih.gov/pubmed/11872728</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font style="bold" size="4" color="#6666FF"/>Journal Article</span>
    <br/>
    <font size="3" color="#000000"/>
    <span>Plant Physiol.(2002).128(1):21-9. doi:; PMID:11788749</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">An oligopeptide transporter gene family in Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Serry Koh, Amy M Wiles, Joshua S Sharp, Fred R Naider, Jeffrey M Becker, Gary Stacey</b>
      </span>
      <br/>
      <span>Affiliation: Department of Microbiology; University of Tennessee; Knoxville; Tennessee 37996-0845; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>We have identified nine oligopeptide transporter (OPT) orthologs (AtOPT1 to AtOPT9) in Arabidopsis. These proteins show significant sequence similarity to OPTs of Candida albicans (CaOpt1p), Schizosaccharomyces pombe (Isp4p), and Saccharomyces cerevisiae (Opt1p and Opt2p). Hydrophilicity plots of the OPTs suggest that they are integral membrane proteins with 12 to 14 transmembrane domains. Sequence comparisons showed that the AtOPTs form a distinct subfamily when compared with the fungal OPTs. Two highly conserved motifs (NPG and KIPPR) were found among all OPT members. The identification of multiple OPTs in Arabidopsis suggests that they may play different functional roles. This idea is supported by the fact that AtOPTs have a distinct, tissue-specific expression pattern. The cDNAs encoding seven of the AtOPTs were cloned into a yeast vector under the control of a constitutive promoter. AtOPT4 expressed in S. cerevisiae mediated the uptake of KLG-[3H]L. Similarly, expression of five of the seven AtOPT proteins expressed in yeast conferred the ability to uptake tetra- and pentapeptides as measured by growth. This study provides new evidence for multiple peptide transporter systems in Arabidopsis, suggesting an important physiological role for small peptides in plants.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/11788749">http://www.ncbi.nlm.nih.gov/pubmed/11788749</a>
    </p>
  </body>
  <hr/>
</html>
