<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(1996).274(5292):1535-7. doi:; PMID:8929415</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Self-sterility in Arabidopsis due to defective pollen tube guidance.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>L K Wilhelmi, D Preuss</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Genetics and Cell Biology; University of Chicago; Chicago; IL 60637; USA. dpreuss@midway.uchicago.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>In flowering plants, a series of cell-cell interactions govern the delivery of sperm to the ovules through precise guidance of pollen tubes. Two Arabidopsis genes, POP2 and POP3, were found that mediate pollen tube guidance and are critical for self-fertility in diploid reproductive cells. The pop2 and pop3 mutations exhibited genetic redundancy: Self-sterility occurred only when male and female tissues were defective in both genes. This phenotype resembles that found in many self-incompatible species.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/274/5292/1535.full">http://science.sciencemag.org/content/274/5292/1535.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(1994).372(6505):475-8. doi:10.1038/372475A0; PMID:7984243</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Protein disaggregation mediated by heat-shock protein Hsp104.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>D A Parsell, A S Kowal, M A Singer, S Lindquist</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Genetics and Cell Biology; Howard Hughes Medical Institute; University of Chicago; Illinois 60637.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The heat-inducible members of the Hsp100 (or Clp) family of proteins share a common function in helping organisms to survive extreme stress, but the basic mechanism through which these proteins function is not understood. Hsp104 protects cells against a variety of stresses, under many physiological conditions, and its function has been evolutionarily conserved, at least from Saccharomyces cerevisiae to Arabidopsis thaliana. Homology with the Escherichia coli ClpA protein suggests that Hsp104 may provide stress tolerance by helping to rid the cell of heat-denatured proteins through proteolysis. But genetic analysis indicates that Hsp104 may function like Hsp70 as a molecular chaperone. Here we investigate the role of Hsp104 in vivo using a temperature-sensitive Vibrio harveyi luciferase-fusion protein as a test substrate. We find that Hsp104 does not protect luciferase from thermal denaturation, nor does it promote proteolysis of luciferase. Rather, Hsp104 functions in a manner not previously described for other heat-shock proteins: it mediates the resolubilization of heat-inactivated luciferase from insoluble aggregates.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v372/n6505/.html">http://www.nature.com/journal/v372/n6505/.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2014).512(7515):436-40. doi:10.1038/Nature13439; PMID:25043057</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The long-term maintenance of a resistance polymorphism through diffuse interactions.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Talia L Karasov, Joel M Kniskern, Liping Gao, Brody J DeYoung, Jing Ding, Ullrich Dubiella, Ruben O Lastra, Sumitha Nallu, Fabrice Roux, Roger W Innes, Luke G Barrett, Richard R Hudson, Joy Bergelson</b>
      </span>
      <br/>
      <span>Affiliation: Department of Ecology and Evolution; University of Chicago; Chicago; Illinois 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plant resistance (R) genes are a crucial component in plant defence against pathogens. Although R genes often fail to provide durable resistance in an agricultural context, they frequently persist as long-lived balanced polymorphisms in nature. Standard theory explains the maintenance of such polymorphisms through a balance of the costs and benefits of resistance and virulence in a tightly coevolving host-pathogen pair. However, many plant-pathogen interactions lack such specificity. Whether, and how, balanced polymorphisms are maintained in diffusely interacting species is unknown. Here we identify a naturally interacting R gene and effector pair in Arabidopsis thaliana and its facultative plant pathogen, Pseudomonas syringae. The protein encoded by the R gene RPS5 recognizes an AvrPphB homologue (AvrPphB2) and exhibits a balanced polymorphism that has been maintained for over 2&#194;&#160;million years (ref. 3). Consistent with the presence of an ancient balanced polymorphism, the R gene confers a benefit when plants are infected with P. syringae carrying avrPphB2 but also incurs a large cost in the absence of infection. RPS5 alleles are maintained at intermediate frequencies in populations globally, suggesting ubiquitous selection for resistance. However, the presence of P. syringae carrying avrPphB is probably insufficient to explain the RPS5 polymorphism. First, avrPphB homologues occur at very low frequencies in P. syringae populations on A. thaliana. Second, AvrPphB only rarely confers a virulence benefit to P. syringae on A. thaliana. Instead, we find evidence that selection for RPS5 involves multiple non-homologous effectors and multiple pathogen species. These results and an associated model suggest that the R gene polymorphism in A. thaliana may not be maintained through a tightly coupled interaction involving a single coevolved R gene and effector pair. More likely, the stable polymorphism is maintained through complex and diffuse community-wide interactions.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v512/n7515/Nature13439.html">http://www.nature.com/journal/v512/n7515/Nature13439.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2011).334(6052):83-6. doi:10.1126/Science.1209244; PMID:21980108</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Adaptation to climate across the Arabidopsis thaliana genome.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Angela M Hancock, Benjamin Brachi, Nathalie Faure, Matthew W Horton, Lucien B Jarymowycz, F Gianluca Sperone, Chris Toomajian, Fabrice Roux, Joy Bergelson</b>
      </span>
      <br/>
      <span>Affiliation: Department of Ecology and Evolution; University of Chicago; 1101 East 57th Street; Chicago; IL 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Understanding the genetic bases and modes of adaptation to current climatic conditions is essential to accurately predict responses to future environmental change. We conducted a genome-wide scan to identify climate-adaptive genetic loci and pathways in the plant Arabidopsis thaliana. Amino acid-changing variants were significantly enriched among the loci strongly correlated with climate, suggesting that our scan effectively detects adaptive alleles. Moreover, from our results, we successfully predicted relative fitness among a set of geographically diverse A. thaliana accessions when grown together in a common environment. Our results provide a set of candidates for dissecting the molecular bases of climate adaptations, as well as insights about the prevalence of selective sweeps, which has implications for predicting the rate of adaptation.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/334/6052/83.full">http://science.sciencemag.org/content/334/6052/83.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2009).324(5923):89-91. doi:10.1126/Science.1170025; PMID:19342588</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Priming in systemic plant immunity.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Ho Won Jung, Timothy J Tschaplinski, Lin Wang, Jane Glazebrook, Jean T Greenberg</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Genetics and Cell Biology; University of Chicago; 1103 East 57th Street EBC410; Chicago; IL 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plants possess inducible systemic defense responses when locally infected by pathogens. Bacterial infection results in the increased accumulation of the mobile metabolite azelaic acid, a nine-carbon dicarboxylic acid, in the vascular sap of Arabidopsis that confers local and systemic resistance against the pathogen Pseudomonas syringae. Azelaic acid primes plants to accumulate salicylic acid (SA), a known defense signal, upon infection. Mutation of the AZELAIC ACID INDUCED 1 (AZI1) gene, which is induced by azelaic acid, results in the specific loss of systemic immunity triggered by pathogen or azelaic acid and of the priming of SA induction in plants. Furthermore, the predicted secreted protein AZI1 is also important for generating vascular sap that confers disease resistance. Thus, azelaic acid and AZI1 are components of plant systemic immunity involved in priming defenses.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/324/5923/89.full">http://science.sciencemag.org/content/324/5923/89.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Cell(2003).114(1):47-59. doi:; PMID:12859897</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Pollen tube growth and guidance is regulated by POP2, an Arabidopsis gene that controls GABA levels.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Ravishankar Palanivelu, Laura Brass, Anna F Edlund, Daphne Preuss</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Genetics and Cell Biology; The University of Chicago; Chicago; IL 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>During angiosperm reproduction, pollen grains form a tube that navigates through female tissues to the micropyle, delivering sperm to the egg; the signals that mediate this process are poorly understood. Here, we describe a role for gamma-amino butyric acid (GABA) in pollen tube growth and guidance. In vitro, GABA stimulates pollen tube growth, although vast excesses are inhibitory. The Arabidopsis POP2 gene encodes a transaminase that degrades GABA and contributes to the formation of a gradient leading up to the micropyle. pop2 flowers accumulate GABA, and the growth of many pop2 pollen tubes is arrested, consistent with their in vitro GABA hypersensitivity. Some pop2 tubes continue to grow toward ovules, yet they are misguided, presumably because they target ectopic GABA on the ovule surface. Interestingly, wild-type tubes exhibit normal growth and guidance in pop2 pistils, perhaps by degrading excess GABA and sharpening the gradient leading to the micropyle.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.cell.com/cell/full/S0092867403004793.html">http://www.cell.com/cell/full/S0092867403004793.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2003).423(6935):74-7. doi:10.1038/Nature01588; PMID:12721627</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Fitness costs of R-gene-mediated resistance in Arabidopsis thaliana.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>D Tian, M B Traw, J Q Chen, M Kreitman, J Bergelson</b>
      </span>
      <br/>
      <span>Affiliation: Department of Ecology and Evolution; University of Chicago; 1101 E. 57th Street; Chicago; Illinois 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Resistance genes (R-genes) act as an immune system in plants by recognizing pathogens and inducing defensive pathways. Many R-gene loci are present in plant genomes, presumably reflecting the need to maintain a large repertoire of resistance alleles. These loci also often segregate for resistance and susceptibility alleles that natural selection has maintained as polymorphisms within a species for millions of years. Given the obvious advantage to an individual of being disease resistant, what prevents these resistance alleles from being driven to fixation by natural selection? A cost of resistance is one potential explanation; most models require a lower fitness of resistant individuals in the absence of pathogens for long-term persistence of susceptibility alleles. Here we test for the presence of a cost of resistance at the RPM1 locus of Arabidopsis thaliana. Results of a field experiment comparing the fitness of isogenic strains that differ in the presence or absence of RPM1 and its natural promoter reveal a large cost of RPM1, providing the first evidence that costs contribute to the maintenance of an ancient R-gene polymorphism.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v423/n6935/Nature01588.html">http://www.nature.com/journal/v423/n6935/Nature01588.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(2002).417(6889):618-24. doi:10.1038/Nature749; PMID:12050657</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Hsp90 as a capacitor of phenotypic variation.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Christine Queitsch, Todd A Sangster, Susan Lindquist</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Genetics and Cell Biology; University of Chicago; Chicago; Illinois 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Heat-shock protein 90 (Hsp90) chaperones the maturation of many regulatory proteins and, in the fruitfly Drosophila melanogaster, buffers genetic variation in morphogenetic pathways. Levels and patterns of genetic variation differ greatly between obligatorily outbreeding species such as fruitflies and self-fertilizing species such as the plant Arabidopsis thaliana. Also, plant development is more plastic, being coupled to environmental cues. Here we report that, in Arabidopsis accessions and recombinant inbred lines, reducing Hsp90 function produces an array of morphological phenotypes, which are dependent on underlying genetic variation. The strength and breadth of Hsp90's effects on the buffering and release of genetic variation suggests it may have an impact on evolutionary processes. We also show that Hsp90 influences morphogenetic responses to environmental cues and buffers normal development from destabilizing effects of stochastic processes. Manipulating Hsp90's buffering capacity offers a tool for harnessing cryptic genetic variation and for elucidating the interplay between genotypes, environments and stochastic events in the determination of phenotype.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v417/n6889/Nature749.html">http://www.nature.com/journal/v417/n6889/Nature749.html</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2001).292(5526):2482-5. doi:10.1126/Science.1060972; PMID:11431566</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Gene families from the Arabidopsis thaliana pollen coat proteome.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>J A Mayfield, A Fiebig, S E Johnstone, D Preuss</b>
      </span>
      <br/>
      <span>Affiliation: Department of Molecular Genetics and Cell Biology; Howard Hughes Medical Institute; The University of Chicago; Chicago; IL 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The pollen extracellular matrix contains proteins mediating species specificity and components needed for efficient pollination. We identified all proteins &gt;10 kilodaltons in the Arabidopsis pollen coating and showed that most of the corresponding genes reside in two genomic clusters. One cluster encodes six lipases, whereas the other contains six lipid-binding oleosin genes, including GRP17, a gene that promotes efficient pollination. Individual oleosins exhibit extensive divergence between ecotypes, but the entire cluster remains intact. Analysis of the syntenic region in Brassica oleracea revealed even greater divergence, but a similar clustering of the genes. Such allelic flexibility may promote speciation in plants.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/292/5526/2482.full">http://science.sciencemag.org/content/292/5526/2482.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Review</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(2001).292(5525):2281-5. doi:10.1126/Science.1061337; PMID:11423651</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Evolutionary dynamics of plant R-genes.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>J Bergelson, M Kreitman, E A Stahl, D Tian</b>
      </span>
      <br/>
      <span>Affiliation: Department of Ecology and Evolution; University of Chicago; 1101 East 57th Street; Chicago; IL 60637; USA. jbergels@midway.uchicago.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Plant R-genes involved in gene-for-gene interactions with pathogens are expected to undergo coevolutionary arms races in which plant specificity and pathogen virulence continually adapt in response to each other. Lending support to this idea, the solvent-exposed amino acid residues of leucine-rich repeats, a region of R-genes involved in recognizing pathogens, often evolve at unusually fast rates. But within-species polymorphism is also common in R-genes, implying that the adaptive substitution process is not simply one of successive selective sweeps. Here we document these features in available data and discuss them in light of the evolutionary dynamics they likely reflect.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/292/5525/2281.full">http://science.sciencemag.org/content/292/5525/2281.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Science(1999).286(5449):2468-74. doi:; PMID:10617454</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genetic definition and sequence analysis of Arabidopsis centromeres.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>G P Copenhaver, K Nickel, T Kuromori, M I Benito, S Kaul, X Lin, M Bevan, G Murphy, B Harris, L D Parnell, W R McCombie, R A Martienssen, M Marra, D Preuss</b>
      </span>
      <br/>
      <span>Affiliation: University of Chicago; Department of Molecular Genetics and Cell Biology; 1103 East 57 Street; Chicago; IL 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>High-precision genetic mapping was used to define the regions that contain centromere functions on each natural chromosome in Arabidopsis thaliana. These regions exhibited dramatic recombinational repression and contained complex DNA surrounding large arrays of 180-base pair repeats. Unexpectedly, the DNA within the centromeres was not merely structural but also encoded several expressed genes. The regions flanking the centromeres were densely populated by repetitive elements yet experienced normal levels of recombination. The genetically defined centromeres were well conserved among Arabidopsis ecotypes but displayed limited sequence homology between different chromosomes, excluding repetitive DNA. This investigation provides a platform for dissecting the role of individual sequences in centromeres in higher eukaryotes.</p>
    <p>
      <span>URL:</span>
      <a href="http://science.sciencemag.org/content/286/5449/2468.full">http://science.sciencemag.org/content/286/5449/2468.full</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Nature(1999).400(6745):667-71. doi:10.1038/23260; PMID:10458161</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Dynamics of disease resistance polymorphism at the Rpm1 locus of Arabidopsis.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>E A Stahl, G Dwyer, R Mauricio, M Kreitman, J Bergelson</b>
      </span>
      <br/>
      <span>Affiliation: Committee on Genetics; University of Chicago; Illinois 60637; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The co-evolutionary 'arms race' is a widely accepted model for the evolution of host-pathogen interactions. This model predicts that variation for disease resistance will be transient, and that host populations generally will be monomorphic at disease-resistance (R-gene) loci. However, plant populations show considerable polymorphism at R-gene loci involved in pathogen recognition. Here we have tested the arms-race model in Arabidopsis thaliana by analysing sequences flanking Rpm1, a gene conferring the ability to recognize Pseudomonas pathogens carrying AvrRpm1 or AvrB. We reject the arms-race hypothesis: resistance and susceptibility alleles at this locus have co-existed for millions of years. To account for the age of alleles and the relative levels of polymorphism within allelic classes, we use coalescence theory to model the long-term accumulation of nucleotide polymorphism in the context of the short-term ecological dynamics of disease resistance. This analysis supports a 'trench warfare' hypothesis, in which advances and retreats of resistance-allele frequency maintain variation for disease resistance as a dynamic polymorphism.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.nature.com/journal/v400/n6745/.html">http://www.nature.com/journal/v400/n6745/.html</a>
    </p>
  </body>
  <hr/>
</html>
