<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2016).():. doi:10.1105/Tpc.15.01057; PMID:27169463</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The Effects of High Steady State Auxin Levels on Root Cell Elongation in Brachypodium.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>David Pacheco-Villalobos, Sara M D&#195;&#173;az-Moreno, Alja van der Schuren, Takayuki Tamaki, Yeon Hee Kang, Bojan Gujas, Ondrej Novak, Nina Jaspert, Zhenni Li, Sebastian Wolf, Claudia Oecking, Karin Ljung, Vincent Bulone, Christian S Hardtke</b>
      </span>
      <br/>
      <span>Affiliation: University of Lausanne CITY: Lausanne POSTAL_CODE: CH-1015 Switzerland [CH] christian.hardtke@unil.ch.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The long-standing Acid Growth Theory of Plant Cell Elongation posits that auxin promotes cell elongation by stimulating cell wall acidification and thus expansin action. To date, the paucity of pertinent genetic materials has precluded thorough analysis of the importance of this concept in roots. The recent isolation of mutants of the model grass species Brachypodium distachyon with dramatically enhanced root cell elongation due to increased cellular auxin levels has allowed us to address this question. We found that the primary transcriptomic effect associated with elevated steady state auxin concentration in elongating root cells is up-regulation of cell wall remodeling factors, notably expansins, while plant hormone signaling pathways maintain remarkable homeostasis. These changes are specifically accompanied by reduced cell wall arabinogalactan complexity but not by increased proton excretion. On the contrary, we observed a tendency for decreased rather than increased proton extrusion from root elongation zones with higher cellular auxin levels. Moreover, similar to Brachypodium distachyon, root cell elongation is, in general, robustly buffered against external pH fluctuation in Arabidopsis thaliana. However, forced acidification through artificial proton pump activation inhibits root cell elongation. Thus, the interplay between auxin, proton pump activation and expansin action may be more flexible in roots than in shoots.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/27169463">http://www.ncbi.nlm.nih.gov/pubmed/27169463</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2016).():. doi:10.1105/Tpc.16.01057; PMID:27152020</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">The Effects of High Steady State Auxin Levels on Root Cell Elongation in Brachypodium.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>David Pacheco-Villalobos, Sara M D&#195;&#173;az-Moreno, Alja van der Schuren, Takayuki Tamaki, Yeon Hee Kang, Bojan Gujas, Ondrej Novak, Nina Jaspert, Zhenni Li, Sebastian Wolf, Claudia Oecking, Karin Ljung, Vincent Bulone, Christian S Hardtke</b>
      </span>
      <br/>
      <span>Affiliation: University of Lausanne CITY: Lausanne POSTAL_CODE: CH-1015 Switzerland [CH] christian.hardtke@unil.ch.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The long-standing Acid Growth Theory of Plant Cell Elongation posits that auxin promotes cell elongation by stimulating cell wall acidification and thus expansin action. To date, the paucity of pertinent genetic materials has precluded thorough analysis of the importance of this concept in roots. The recent isolation of mutants of the model grass species Brachypodium distachyon with dramatically enhanced root cell elongation due to increased cellular auxin levels has allowed us to address this question. We found that the primary transcriptomic effect associated with elevated steady state auxin concentration in elongating root cells is up-regulation of cell wall remodeling factors, notably expansins, while plant hormone signaling pathways maintain remarkable homeostasis. These changes are specifically accompanied by reduced cell wall arabinogalactan complexity but not by increased proton excretion. On the contrary, we observed a tendency for decreased rather than increased proton extrusion from root elongation zones with higher cellular auxin levels. Moreover, similar to Brachypodium distachyon, root cell elongation is, in general, robustly buffered against external pH fluctuation in Arabidopsis thaliana. However, forced acidification through artificial proton pump activation inhibits root cell elongation. Thus, the interplay between auxin, proton pump activation and expansin action may be more flexible in roots than in shoots.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/27152020">http://www.ncbi.nlm.nih.gov/pubmed/27152020</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2015).27(1):71-85. doi:10.1105/Tpc.114.133991; PMID:25634987</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Genome-wide analysis of alternative splicing landscapes modulated during plant-virus interactions in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Kranthi K Mandadi, Karen-Beth G Scholthof</b>
      </span>
      <br/>
      <span>Affiliation: Department of Plant Pathology and Microbiology; Texas A&amp;M University; College Station; Texas 77843 kbgs@tamu.edu.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>In eukaryotes, alternative splicing (AS) promotes transcriptome and proteome diversity. The extent of genome-wide AS changes occurring during a plant-microbe interaction is largely unknown. Here, using high-throughput, paired-end RNA sequencing, we generated an isoform-level spliceome map of Brachypodium distachyon infected with Panicum mosaic virus and its satellite virus. Overall, we detected &#226;&#136;&#188;44,443 transcripts in B. distachyon, &#226;&#136;&#188;30% more than those annotated in the reference genome. Expression of &#226;&#136;&#188;28,900 transcripts was &#226;&#137;&#165;2 fragments per kilobase of transcript per million mapped fragments, and &#226;&#136;&#188;42% of multi-exonic genes were alternatively spliced. Comparative analysis of AS patterns in B. distachyon, rice (Oryza sativa), maize (Zea mays), sorghum (Sorghum bicolor), Arabidopsis thaliana, potato (Solanum tuberosum), Medicago truncatula, and poplar (Populus trichocarpa) revealed conserved ratios of the AS types between monocots and dicots. Virus infection quantitatively altered AS events in Brachypodium with little effect on the AS ratios. We discovered AS events for &gt;100 immune-related genes encoding receptor-like kinases, NB-LRR resistance proteins, transcription factors, RNA silencing, and splicing-associated proteins. Cloning and molecular characterization of SCL33, a serine/arginine-rich splicing factor, identified multiple novel intron-retaining splice variants that are developmentally regulated and modulated during virus infection. B. distachyon SCL33 splicing patterns are also strikingly conserved compared with a distant Arabidopsis SCL33 ortholog. This analysis provides new insights into AS landscapes conserved among monocots and dicots and uncovered AS events in plant defense-related genes.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25634987">http://www.ncbi.nlm.nih.gov/pubmed/25634987</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2015).27(1):7. doi:10.1105/Tpc.115.00039; PMID:25634986</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Sick as a&#226;&#128;&#166; grass? Viral infection causes massive changes in alternative splicing in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jennifer Mach</b>
      </span>
      <br/>
      <span>Affiliation: Science Editor jmach@aspb.org.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/></p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/25634986">http://www.ncbi.nlm.nih.gov/pubmed/25634986</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2014).26(3):1094-104. doi:10.1105/Tpc.113.121830; PMID:24642944</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">A role for barley CRYPTOCHROME1 in light regulation of grain dormancy and germination.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jose M Barrero, A Bruce Downie, Qian Xu, Frank Gubler</b>
      </span>
      <br/>
      <span>Affiliation: CSIRO Plant Industry; Canberra ACT 2601; Australia.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>It is well known that abscisic acid (ABA) plays a central role in the regulation of seed dormancy and that transcriptional regulation of genes encoding ABA biosynthetic and degradation enzymes is responsible for determining ABA content. However, little is known about the upstream signaling pathways impinging on transcription to ultimately regulate ABA content or how environmental signals (e.g., light and cold) might direct such expression in grains. Our previous studies indicated that light is a key environmental signal inhibiting germination in dormant grains of barley (Hordeum vulgare), wheat (Triticum aestivum), and Brachypodium distachyon and that this effect attenuates as after-ripening progresses further. We found that the blue component of the light spectrum inhibits completion of germination in barley by inducing the expression of the ABA biosynthetic gene 9-cis-epoxycarotenoid dioxygenase and dampening expression of ABA 8'-hydroxylase, thus increasing ABA content in the grain. We have now created barley transgenic lines downregulating the genes encoding the blue light receptors CRYTOCHROME (CRY1) and CRY2. Our results demonstrate that CRY1 is the key receptor perceiving and transducing the blue light signal in dormant grains.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24642944">http://www.ncbi.nlm.nih.gov/pubmed/24642944</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2013).25(11):4283. doi:10.1105/Tpc.113.251112; PMID:24285793</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Small RNAs and the big decisions: MicroRNA regulation of photoperiodic flowering in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Jennifer Mach</b>
      </span>
      <br/>
      <span>Affiliation: Science Editor.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/></p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24285793">http://www.ncbi.nlm.nih.gov/pubmed/24285793</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2013).25(11):4363-77. doi:10.1105/Tpc.113.118620; PMID:24285787</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Regulation of FLOWERING LOCUS T by a microRNA in Brachypodium distachyon.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Liang Wu, Dongfeng Liu, Jiajie Wu, Rongzhi Zhang, Zhengrui Qin, Danmei Liu, Aili Li, Daolin Fu, Wenxue Zhai, Long Mao</b>
      </span>
      <br/>
      <span>Affiliation: National Key Facility for Crop Gene Resources and Genetic Improvement; Institute of Crop Science; Chinese Academy of Agricultural Sciences; Beijing 100081; China.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>The highly conserved florigen gene FLOWERING LOCUS T (FT) functions at the core of the flowering pathways. Extensive studies have examined the transcriptional regulation of FT; however, other layers of FT regulation remain unclear. Here, we identified miR5200 a Pooideae-specific microRNA that is expressed in leaves and targets Brachypodium distachyon FT orthologs for mRNA cleavage. miR5200 was abundantly expressed in plants grown under short-day (SD) conditions but was dramatically repressed in plants transferred to long-day (LD) conditions. We also found that the epigenetic chromatin status, specifically the levels of histone methylation marks, at miR5200 precursor loci changed in response to daylength. Moreover, artificial interruption of miR5200 activity by target mimicry in B. distachyon altered flowering time in SD but not in LD conditions, suggesting that miR5200 functions in photoperiod-mediated flowering time regulation. Together, these findings illustrate a posttranscriptional regulation mechanism of FT and provide insights into understanding of the multiple concerted pathways for flowering time control in plants.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24285787">http://www.ncbi.nlm.nih.gov/pubmed/24285787</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2013).25(10):3685-98. doi:10.1105/Tpc.113.114553; PMID:24104565</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Reticulate evolution of the rye genome.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Mihaela M Martis, Ruonan Zhou, Grit Haseneyer, Thomas Schmutzer, Jan Vr&#195;&#161;na, Marie Kubal&#195;&#161;kov&#195;&#161;, Susanne K&#195;&#182;nig, Karl G Kugler, Uwe Scholz, Bernd Hackauf, Viktor Korzun, Chris-Carolin Sch&#195;&#182;n, Jaroslav Dolezel, Eva Bauer, Klaus F X Mayer, Nils Stein</b>
      </span>
      <br/>
      <span>Affiliation: Helmholtz Center Munich; German Research Centre for Environmental Health; Munich Information Center for Protein Sequences/IBIS; Institute of Bioinformatics and Systems Biology; 85764 Neuherberg; Germany.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Rye (Secale cereale) is closely related to wheat (Triticum aestivum) and barley (Hordeum vulgare). Due to its large genome (~8 Gb) and its regional importance, genome analysis of rye has lagged behind other cereals. Here, we established a virtual linear gene order model (genome zipper) comprising 22,426 or 72% of the detected set of 31,008 rye genes. This was achieved by high-throughput transcript mapping, chromosome survey sequencing, and integration of conserved synteny information of three sequenced model grass genomes (Brachypodium distachyon, rice [Oryza sativa], and sorghum [Sorghum bicolor]). This enabled a genome-wide high-density comparative analysis of rye/barley/model grass genome synteny. Seventeen conserved syntenic linkage blocks making up the rye and barley genomes were defined in comparison to model grass genomes. Six major translocations shaped the modern rye genome in comparison to a putative Triticeae ancestral genome. Strikingly dissimilar conserved syntenic gene content, gene sequence diversity signatures, and phylogenetic networks were found for individual rye syntenic blocks. This indicates that introgressive hybridizations (diploid or polyploidy hybrid speciation) and/or a series of whole-genome or chromosome duplications played a role in rye speciation and genome evolution.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/24104565">http://www.ncbi.nlm.nih.gov/pubmed/24104565</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2011).23(5):1756-71. doi:10.1105/Tpc.111.084590; PMID:21540434</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">BARREN STALK FASTIGIATE1 is an AT-hook protein required for the formation of maize ears.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Andrea Gallavotti, Simon Malcomber, Craig Gaines, Sharon Stanfield, Clinton Whipple, Elizabeth Kellogg, Robert J Schmidt</b>
      </span>
      <br/>
      <span>Affiliation: Section of Cell and Developmental Biology; University of California-San Diego; La Jolla; CA 92093-0116; USA. agallavotti@ucsd.edu</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Ears are the seed-bearing inflorescences of maize (Zea mays) plants and represent a crucial component of maize yield. The first step in the formation of ears is the initiation of axillary meristems in the axils of developing leaves. In the classic maize mutant barren stalk fastigiate1 (baf1), first discovered in the 1950s, ears either do not form or, if they do, are partially fused to the main stalk. We positionally cloned Baf1 and found that it encodes a transcriptional regulator containing an AT-hook DNA binding motif. Single coorthologs of Baf1 are found in syntenic regions of brachypodium (Brachypodium distachyon), rice (Oryza sativa), and sorghum (Sorghum bicolor), suggesting that the gene is likely present in all cereal species. Protein-protein interaction assays suggest that BAF1 is capable of forming homodimers and heterodimers with other members of the AT-hook family. Another transcriptional regulator required for ear initiation is the basic helix-loop-helix protein BARREN STALK1 (BA1). Genetic and expression analyses suggest that Baf1 is required to reach a threshold level of Ba1 expression for the initiation of maize ears. We propose that Baf1 functions in the demarcation of a boundary region essential for the specification of a stem cell niche.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21540434">http://www.ncbi.nlm.nih.gov/pubmed/21540434</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2011).23(4):1249-63. doi:10.1105/Tpc.110.082537; PMID:21467582</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Unlocking the barley genome by chromosomal and comparative genomics.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Klaus F X Mayer, Mihaela Martis, Pete E Hedley, Hana Simkov&#195;&#161;, Hui Liu, Jenny A Morris, Burkhard Steuernagel, Stefan Taudien, Stephan Roessner, Heidrun Gundlach, Marie Kubal&#195;&#161;kov&#195;&#161;, Pavla Such&#195;&#161;nkov&#195;&#161;, Florent Murat, Marius Felder, Thomas Nussbaumer, Andreas Graner, Jerome Salse, Takashi Endo, Hiroaki Sakai, Tsuyoshi Tanaka, Takeshi Itoh, Kazuhiro Sato, Matthias Platzer, Takashi Matsumoto, Uwe Scholz, Jaroslav Dolezel, Robbie Waugh, Nils Stein</b>
      </span>
      <br/>
      <span>Affiliation: Munich Information Center for Protein Sequences/Institute of Bioinformatics and Systems Biology; Institute for Bioinformatics and Systems Biology; Helmholtz Center Munich; 85764 Neuherberg; Germany.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>We used a novel approach that incorporated chromosome sorting, next-generation sequencing, array hybridization, and systematic exploitation of conserved synteny with model grasses to assign ~86% of the estimated ~32,000 barley (Hordeum vulgare) genes to individual chromosome arms. Using a series of bioinformatically constructed genome zippers that integrate gene indices of rice (Oryza sativa), sorghum (Sorghum bicolor), and Brachypodium distachyon in a conserved synteny model, we were able to assemble 21,766 barley genes in a putative linear order. We show that the barley (H) genome displays a mosaic of structural similarity to hexaploid bread wheat (Triticum aestivum) A, B, and D subgenomes and that orthologous genes in different grasses exhibit signatures of positive selection in different lineages. We present an ordered, information-rich scaffold of the barley genome that provides a valuable and robust framework for the development of novel strategies in cereal breeding.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21467582">http://www.ncbi.nlm.nih.gov/pubmed/21467582</a>
    </p>
  </body>
  <hr/>
</html>
<html>
  <head>
    <title>Search result</title>
  </head>
  <body>
    <span><font color="#6666FF" style="bold" size="4"/>Journal Article</span>
    <br/>
    <font color="#000000" size="3"/>
    <span>Plant Cell(2011).23(1):27-37. doi:10.1105/Tpc.110.080622; PMID:21266659</span>
    <br/>
    <span>
      <font weight="10" size="5"/>
      <b CLASS="title">Seventy million years of concerted evolution of a homoeologous chromosome pair, in parallel, in major Poaceae lineages.</b>
    </span>
    <br/>
    <font size="4"/>
    <p>
      <span>
        <b>Xiyin Wang, Haibao Tang, Andrew H Paterson</b>
      </span>
      <br/>
      <span>Affiliation: Plant Genome Mapping Laboratory; University of Georgia; Athens; Georgia 30602; USA.</span>
    </p>
    <p><span><b>Abstract:</b></span><br/>Whole genome duplication ~70 million years ago provided raw material for Poaceae (grass) diversification. Comparison of rice (Oryza sativa), sorghum (Sorghum bicolor), maize (Zea mays), and Brachypodium distachyon genomes revealed that one paleo-duplicated chromosome pair has experienced very different evolution than all the others. For tens of millions of years, the two chromosomes have experienced illegitimate recombination that has been temporally restricted in a stepwise manner, producing structural stratification in the chromosomes. These strata formed independently in different grass lineages, with their similarities (low sequence divergence between paleo-duplicated genes) preserved in parallel for millions of years since the divergence of these lineages. The pericentromeric region of this homeologous chromosome pair accounts for two-thirds of the gene content differences between the modern chromosomes. Both intriguing and perplexing is a distal chromosomal region with the greatest DNA similarity between surviving duplicated genes but also with the highest concentration of lineage-specific gene pairs found anywhere in these genomes and with a significantly elevated gene evolutionary rate. Intragenomic similarity near this chromosomal terminus may be important in hom(e)ologous chromosome pairing. Chromosome structural stratification, together with enrichment of autoimmune response-related (nucleotide binding site-leucine-rich repeat) genes and accelerated DNA rearrangement and gene loss, confer a striking resemblance of this grass chromosome pair to the sex chromosomes of other taxa.</p>
    <p>
      <span>URL:</span>
      <a href="http://www.ncbi.nlm.nih.gov/pubmed/21266659">http://www.ncbi.nlm.nih.gov/pubmed/21266659</a>
    </p>
  </body>
  <hr/>
</html>
